 <!-- FlatFy Theme - Andrea Galanti /-->
<!doctype html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->
<!--[if IE 9]>    <html class="no-js ie9" lang="en"> <![endif]-->
<!--[if gt IE 9]><!--> <html> <!--<![endif]-->
<script>
function showhide(id) {
  var x = document.getElementById(id);
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
    <meta name="description" content="Workshop at the 35th Conference on Neural Information Processing Systems">
    <meta name="author" content="">

    <title>Machine Learning in Structural Biology</title>

    <!-- Bootstrap core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom Google Web Font -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
    <link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,100italic,300italic,400italic,700italic,900italic' rel='stylesheet' type='text/css'>
	<link href='http://fonts.googleapis.com/css?family=Arvo:400,700' rel='stylesheet' type='text/css'>

    <!-- Custom CSS-->
    <link href="css/general.css" rel="stylesheet">

	 <!-- Owl-Carousel -->
    <link href="css/custom.css" rel="stylesheet">
	<link href="css/owl.carousel.css" rel="stylesheet">
    <link href="css/owl.theme.css" rel="stylesheet">
	<link href="css/style.css" rel="stylesheet">
	<link href="css/animate.css" rel="stylesheet">

	<!-- Magnific Popup core CSS file -->
	<link rel="stylesheet" href="css/magnific-popup.css">

	<script src="js/modernizr-2.8.3.min.js"></script>  <!-- Modernizr /-->
	<!--[if IE 9]>
		<script src="js/PIE_IE9.js"></script>
	<![endif]-->
	<!--[if lt IE 9]>
		<script src="js/PIE_IE678.js"></script>
	<![endif]-->

	<!--[if lt IE 9]>
		<script src="js/html5shiv.js"></script>
	<![endif]-->

</head>

<body id="home">

	<!-- Preloader -->
	<div id="preloader">
		<div id="status"></div>
	</div>

	<!-- FullScreen -->
    <div class="intro-header">
		<div class="col-xs-12 text-center abcen1">
			<h1 class="h1_home wow fadeIn" data-wow-delay="0.4s">Machine Learning in Structural Biology</h1>
			<h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Workshop at the 35th Conference on Neural Information Processing Systems</h3>
			<h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Monday, December 13, 2021</h3>
			<ul class="list-inline intro-social-buttons">
                <li><a href="https://neurips.cc/virtual/2021/workshop/21869" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">Livestream</span></a>
				</li>
                <li><a href="index_2020.html" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">2020 Workshop</span></a>
                </li>
                <!--
				<li id="download" ><a href="" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">Gather</span></a>
                -->
				</li>
			</ul>
		</div>
        <!-- /.container -->
		<div class="col-xs-12 text-center abcen wow fadeIn">
			<div class="button_down ">
				<a class="imgcircle wow bounceInUp" data-wow-duration="1.5s"  href="#about"> <img class="img_scroll" src="img/icon/circle.png" alt="Go down to content"> </a>
			</div>
		</div>
    </div>

	<!-- NavBar-->
	<nav class="navbar-default" role="navigation">
		<div class="container">
			<div class="navbar-header">
				<button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse">
					<span class="sr-only">Toggle navigation</span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
					<span class="icon-bar"></span>
				</button>
				<a class="navbar-brand" href="#home">MLSB</a>
			</div>

			<div class="collapse navbar-collapse navbar-right navbar-ex1-collapse">
				<ul class="nav navbar-nav">

					<li class="menuItem"><a href="#about">About</a></li>
                    <li class="menuItem"><a href="#callforpapers">Call for Papers</a></li>
					<li class="menuItem"><a href="#invitedspeakers">Invited Speakers</a></li>
                    <!-- <li class="menuItem"><a href="#preregistration">Pre-Registration</a></li> -->
                    <li class="menuItem"><a href="#schedule">Schedule</a></li>
                    <li class="menuItem"><a href="#accepted">Accepted Papers</a></li>
					<li class="menuItem"><a href="#organizers">Organizers</a></li>
                    <li class="menuItem"><a href="#related">Related Workshops</a></li>
                    <li class="menuItem"><a href="#sponsors">Sponsors</a></li>
                    <!-- <li class="menuItem"><a href="#contact">Contact</a></li> -->
				</ul>
			</div>

		</div>
	</nav>

	<!-- About -->
	<div id="about" class="content-section-a" style="border-top: 0">
        <div class="container">
            <div class="row">
                <div class="col-md-12 wow fadeInLeftBig"e data-animation-delay="200">
                    <h2 class="section-heading">About</h2>

                    <p class="lead">
                        Structural biology, the study of proteins and other biomolecules through their 3D structures, is a field on the cusp of transformation. While measuring and interpreting biomolecular structures has traditionally been an expensive and difficult endeavor, recent machine-learning based modeling approaches have shown that it will become routine to  predict and reason about structure at proteome scales with unprecedented atomic resolution. This broad liberation of 3D structure within bioscience and biomedicine will likely have transformative impacts on our ability to create effective medicines, to understand and engineer biology, and to design new molecular materials and machinery.  Machine learning also shows great promise to continue to revolutionize many core technical problems in structural biology, including protein design, modelling protein dynamics, predicting higher order complexes, and integrating learning with experimental structure determination.
                    </p>
                    <p class="lead">
                        At this inflection point, we hope that the Machine Learning in Structural Biology (MLSB) workshop will help bring community and direction to this rising field. To achieve these goals, this workshop will bring together researchers from a unique and diverse set of domains, including core machine learning, computational biology, experimental structural biology, geometric deep learning, and natural language processing.
                    </p>

                    <p class="lead">
                    To attend the MLSB workshop talks, poster sessions, and virtual hangouts, <strong> register for the main NeurIPS conference</strong> at <a href=https://neurips.cc>neurips.cc</a>.
                    </p>
				</div>
            </div>
        </div>
        <!-- /.container -->
        <!--
		<div class="container">

			<div class="col-md-6 col-md-offset-3 text-center wrap_title">
				<h2>About</h2>
				<p class="lead" style="margin-top:0">Description of MLSB</p>

			</div>
		</div>
        -->
	</div>

	<!-- Use it -->
    <div id ="callforpapers" class="content-section-a">
        <div class="container">

            <div class="row">
                <div class="col-md-12 wow fadeInRightBig"e data-animation-delay="200">
                    <h2 class="section-heading">Call For Papers</h2>
                    <p>We welcome submissions of short papers leveraging machine learning to address problems in structural biology, including but not limited to:</p>
                    <ul class="bulleted">
                        <li>Prediction of biomolecular structures, complexes, and interactions</li>
                        <li>Geometric and symmetry-aware deep learning</li>
                        <li>Protein and RNA design</li>
                        <li>Generative models for structure and sequence</li>
                        <li>Experimental structure determination</li>
                        <li>Conformational change, ensembles, and dynamics</li>
                        <li>Function and property prediction</li>
                        <li>Structural systems biology</li>
                        <li>Language models and other representations of proteins</li>
                    </ul>
					<p>We request anonymized PDF submissions by
                    </span><s>Friday, September 17th</s> <strong>Friday, October 1st, 2021, 11:59PM</strong> in the timezone of your choice through our submission website on <a href="https://cmt3.research.microsoft.com/MLSB2021">CMT</a>.</p>
					<p>Papers should present novel work that has not been previously accepted at an archival venue at the time of submission.  Submissions should be a maximum of 4 pages (excluding references and appendices) in PDF format and fully anonymized as per the requirements of NeurIPS. We request use of the <a href="https://neurips.cc/Conferences/2020/PaperInformation/StyleFiles">NeurIPS style files</a>. Submissions meeting these criteria will go through a light, double-blind review process. Reviewer comments will be returned to the authors as feedback.</p>

                    <p>Accepted papers will be invited to present a poster at the virtual workshop, with nominations of spotlight talks at the discretion of the organizers.  This workshop is considered non-archival and does not publish proceedings, however authors of accepted contributions will have the option to make their work available through the workshop website. Presentation of work that is concurrently in submission is welcome.</p>

                    <h3 class="section-heading">Important Dates</h3>
                    <p class="">Submission Deadline: <s>Friday, September 17th</s> <b>Friday, October 1st, 2021, 11:59 PM, Anywhere on Earth.</b></p>
                    <p class="">Notification of Acceptance: Friday, October 22nd, 2021.</p>
                    <p class="">Workshop Date: Monday, December 13th, 2021.</p>
				</div>
            </div>
        </div>
        <!-- /.container -->
    </div>


	<div id="invitedspeakers" class="content-section-b" style="border-top: 0">
		<div class="container">

			<div class="col-md-6 col-md-offset-3 text-center wrap_title">
				<h2>Invited Speakers</h2>
			</div>

            <div class="row">
				<div class="col-sm-6 wow fadeInDown text-center">
				  <img class="magnify img-rounded" src="img/speakers/john_jumper.jpg" alt="John Jumper">
				  <h3>John Jumper</h3>
				  <p class="lead3">Keynote</p>
                  <p class="lead" style="margin-bottom:0;" >Senior Staff Research Scientist at DeepMind.</p>
		  <a onclick="showhide('john_jumper')">Show/Hide Bio</a>
		  <div style="display: none; text-align: left; text-indent: 30px;" id="john_jumper">
			  <p>
John Jumper is the AlphaFold team lead at DeepMind. John received his PhD in Chemistry from the University of Chicago for developing machine learning methods to simulate protein dynamics.
              Before that,
				  he worked on molecular dynamics simulations of proteins and supercooled liquids at D. E. Shaw Research.
He earned a B.S. in Physics and Mathematics from Vanderbilt University and an MPhil in Physics from the University of Cambridge. At DeepMind, John is leading the development of new methods to apply machine learning to protein biology.
            </p>
		  </div>
				  <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-md-6 -->
				<div class="col-sm-6 wow fadeInDown text-center">
				  <img  class="magnify img-rounded" src="img/speakers/jane_richardson.jpg" alt="Jane Richardson">
				   <h3>Jane Richardson</h3>
				 <p class="lead3">Keynote</p>
                  <p class="lead" style="margin-bottom:0;">Professor of Biochemistry at Duke University.</p>
		  <a onclick="showhide('jane_richardson')">Show/Hide Bio</a>
		  <div style="display: none; text-align: left; text-indent: 30px;" id="jane_richardson">
			  <p>Jane S. Richardson and her husband David have worked together for over 50 years on
				  research to understand the 3D structure of protein and RNA molecules, including their description, determinants,
				  folding, evolution, and control.  They were among the early groups doing protein crystallography, helped to
				  start the field of protein de novo design, and developed the molecular graphics system of "kinemages".
				  Around 2000 they developed a method that calculates hydrogen-atom contacts to visualize and quantify the
				  details of packing interactions inside and between molecules.  Its principal application is to improving
				  the accuracy of experimental structures of protein and RNA molecules, as implemented on their MolProbity
				  structure-validation web service and within the Phenix software system for crystallography & cryoEM, and
				  used to help depositors re-version early-release SARS-COV-2 structures.  They are now working on validation
				  methods that can still find errors when previous validation criteria are being refined or restrained.</p>
			  <p>Prof. Richardson pioneered ribbon drawings for representing protein structures, first
				  described many of the common features of overall protein folds and their local motifs (such as Greek key beta
				  barrels, righthanded crossovers, helix caps, cis Pro touch-turns, vicinal disulfides) and has worked to spread
				  molecular 3D literacy to students and colleagues at Duke and around the world.  From a Swarthmore B.A. in math,
				  physics, astronomy and philosophy, she has become a biophysicist, a MacArthur Fellow, a member of the National
				  Academies of Sciences & of Medicine, and a James B. Duke Professor of Biochemistry at Duke, and has three honorary
				  degrees to fill in for her lack of a PhD.  For more details, see Ann. Rev. Biophysics (2013) 4:1-28 or J. Biol.
				  Chem. (2021) 296:100742.</p>
		   </div>
				  <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-md-6 -->
            </div> <!-- row -->

			<div class="row tworow">
				<div class="col-sm-4 wow fadeInDown text-center">
				  <img  class="magnify img-rounded" src="img/speakers/regina_barzilay.jpg" alt="Regina Barzilay">
				   <h3>Regina Barzilay</h3>
					<p class="lead" style="margin-bottom:0;">Professor of Computer Science at Massachusetts Institute of Technology.</p>
				        <a onclick="showhide('regina_barzilay')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="regina_barzilay">
						<p>
Regina Barzilay is a Delta Electronics Professor in MIT’s Department of Electrical Engineering and Computer Science and a faculty lead for the MIT Abdul Latif Jameel Clinic for Machine Learning in Health (J-Clinic). She is also a principal investigator at the Computer Science and Artificial Intelligence Laboratory. Barzilay’s research interests are in natural language processing, and applications of deep learning to chemistry and oncology. Her research focuses on developing deep learning models that can transfer annotations across domains and languages, that can learn from a few annotated examples by utilizing supplementary data sources, and that can take advantage of human-provided rationales to constrain model structure. Barzilay has received an NSF Career Award, MIT Technology Review’s TR-35 Award, and Microsoft Faculty Fellowship. She has also received fellowships from the MacArthur Foundation, Association for Computational Linguistics and Association for the Advancement of Artificial Intelligence. She received a PhD in computer science from Columbia University, and was a postdoc at Cornell University.
							</p>
					</div>
				  <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

				<div class="col-sm-4  wow fadeInDown text-center">
				  <img class="magnify img-rounded" src="img/speakers/michael_bronstein.jpg" alt="Michael Bronstein">
				  <h3>Michael Bronstein</h3>
					<p class="lead" style="margin-bottom:0;">Professor of Machine Learning at Imperial College London and Head of Graph Learning at Twitter.</p>
				        <a onclick="showhide('michael_bronstein')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="michael_bronstein">
						<p>
Michael Bronstein joined the Department of Computing as Professor in 2018. He has served as a professor at USI Lugano, Switzerland since 2010 and held visiting positions at Stanford, Harvard, MIT, TUM, and Tel Aviv University. Michael received his PhD with distinction from the Technion (Israel Institute of Technology) in 2007. His main expertise is in theoretical and computational geometric methods for machine learning and data science, and his research encompasses a broad spectrum of applications ranging from computer vision and pattern recognition to geometry processing, computer graphics, and biomedicine. Michael has authored over 200 papers, a book, and holds over 35 granted patents. He was awarded five ERC grants, two Google Faculty Research awards, two Amazon ML Research awards, Facebook Computational Social Science award, Dalle Molle prize, Royal Society Wolfson Merit award, and Royal Academy of Engineering Silver Medal. He is a PI and ML Lead in Project CETI, a TED Audacious Prize winning collaboration aimed at understanding the communication of sperm whales. During 2017-2018 he was a fellow at the Radcliffe Institute for Advanced Study at Harvard University and since 2017, he is a Rudolf Diesel fellow at TU Munich. He was invited as a Young Scientist to the World Economic Forum, an honour bestowed on forty world’s leading scientists under the age of forty. Michael is a Member of Academia Europaea, Fellow of IEEE, IAPR, ELLIS, and BCS, alumnus of the Technion Excellence Program and the Academy of Achievement, and ACM Distinguished Speaker. In addition to academic work, Michael's industrial experience includes technological leadership in multiple startup companies, including Novafora, Videocites, Invision (acquired by Intel in 2012), and Fabula AI (acquired by Twitter in 2019). Following the acquisition of Fabula, he joined Twitter as Head of Graph Learning Research. He previously served as Principal Engineer at Intel Perceptual Computing (2012-2019) and was one of the key developers of the Intel RealSense 3D camera technology. He is also an angel investor and supporter of multiple early-stage startups. 
                        </p>
					</div>
				 <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

				<div class="col-sm-4 wow fadeInDown text-center">
				  <img  class="magnify img-rounded" src="img/speakers/cecilia_clementi.jpg" alt="Cecilia Clementi">
				   <h3>Cecilia Clementi</h3>
					<p class="lead" style="margin-bottom:0;">Professor of Computational Biophysics at Freie Universität Berlin.</p>
				        <a onclick="showhide('cecilia_clementi')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="cecilia_clementi">
						<p>Cecilia Clementi is an Italian-American scientist who specialises in the simulation of biomolecules.
							She is a Professor of Computational Biophysics at the Free University of Berlin. She was previously
							a Professor of Chemistry at the Rice University and co-director of the National Science Foundation
							Molecular Sciences Software Institute. From 2017 to 2019 she held an Einstein Foundation fellowship.</p>
					</div>
				   <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

			</div><!-- /.row -->

			<div class="row tworow">
				<div class="col-sm-4 wow fadeInDown text-center">
				  <img  class="magnify img-rounded" src="img/speakers/lucy_colwell.jpg" alt="Lucy Colwell">
				   <h3>Lucy Colwell</h3>
				   <p class="lead" style="margin-bottom:0;">Assistant Professor of Chemistry at University of Cambridge and Research Scientist at Google.</p>
				        <a onclick="showhide('lucy_colwell')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="lucy_colwell">
						<p>Lucy is a research scientist at Google Research who works closely with colleagues from GAS and
							Brain to better understand the relationship between the sequence and function of biological
							macromolecules. Her broader research interests involve understanding how Google's strengths
							in experimental design and machine learning can be applied to the discovery and production
							of proteins for use in a diverse range of applications.</p>
					</div>
				   <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

                <div class="col-sm-4  wow fadeInDown text-center">
				  <img class="magnify img-rounded" src="img/speakers/amy_keating.jpg" alt="Amy Keating">
				  <h3>Amy Keating</h3>
				  <p class="lead" style="margin-bottom:0;">Professor of Biology at Massachusetts Institute of Technology.</p>
				        <a onclick="showhide('amy_keating')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="amy_keating">
						<p>A Professor of Biology and Biological Engineering at the Massachusetts Institute of Technology (MIT),
							Keating’s research involves working to understand how protein sequences and structures encode
							interaction specificity. Her lab uses computational and experimental methods to measure protein
							interactions in high throughput, predict new interactions, and design proteins and peptides with
							novel interaction properties.</p>
					</div>
				 <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

                <div class="col-sm-4  wow fadeInDown text-center">
				  <img class="magnify img-rounded" src="img/speakers/derek_lowe.jpg" alt="Derek Lowe">
				  <h3>Derek Lowe</h3>
				  <p class="lead" style="margin-bottom:0;">Director in Chemical Biology at Novartis Institutes for BioMedical Research.</p>
				        <a onclick="showhide('derek_lowe')">Show/Hide Bio</a>
					<div style="display: none; text-align: left; text-indent: 30px;" id="derek_lowe">
						<p>Derek Lowe is a medicinal chemist working on preclinical drug discovery in the pharmaceutical industry.
							Lowe has published a blog about this field, "In the Pipeline", since 2002 and is a columnist for
							the Royal Society of Chemistry's Chemistry World.</p>
					</div>
				 <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
				</div><!-- /.col-lg-4 -->

			</div><!-- /.row -->
		</div>
	</div>

    <!-- Schedule -->
    <div id ="schedule" class="content-section-a">
        <div class="container">
			<h2 class="section-heading">Schedule (EST)</h2>
            <div class="row"><div class="col-md-6"><table class="schedule"><tr><th>09:00</th><td colspan="4" rowspan=1 class="stage-saturn">Opening Remarks</td></tr><tr><th>09:10</th><td colspan="4" rowspan=2 class="stage-earth">Invited Talk - Michael Bronstein</td></tr><tr><th>09:20</th></tr><tr><th>09:30</th><td colspan="4" rowspan=2 class="stage-saturn">Invited Talk - Cecilia Clementi</td></tr><tr><th>09:40</th></tr><tr><th>09:50</th><td colspan="4" rowspan=2 class="stage-earth">Invited Talk - Lucy Colwell</td></tr><tr><th>10:00</th></tr><tr><th>10:10</th><td colspan="4" rowspan=1 class="stage-saturn">Contributed Talk:<p style="font-weight:normal">Structure-aware generation of drug-like molecules (Pavol Drotar, Arian R. Jamasb, Ben J. Day, Catalina Cangea, Pietro Li&oacute)</p></td></tr><tr><th>10:20</th><td colspan="4" rowspan=1 class="stage-earth">Contributed Talk:<p style="font-weight:normal">Learning physics confers pose-sensitivity in structure-based virtual screening (Pawel Gniewek, Bradley Worley, Kate Stafford, Brandon Anderson, Henry van den Bedem)</p></td></tr><tr><th>10:30</th><td colspan="4" rowspan=1 class="stage-saturn">Contributed Talk:<p style="font-weight:normal">Inferring a Continuous Distribution of Atom Coordinates from Cryo-EM Images using VAEs (Dan Rosenbaum, Marta Garnelo, Michal Zielinski, Charlie Beattie, Ellen Clancy, Andrea Huber, Pushmeet Kohli, Andrew Senior, John Jumper, Carl Doersch, S. M. Ali Eslami, Olaf Ronneberger, Jonas Adler)</p></td></tr><tr><th>10:40</th><td colspan="4" rowspan=1 class="stage-earth">Contributed Talk:<p style="font-weight:normal">Weakly Supervised Learning for Joint Image Denoising and Protein Localization in Cryo-EM (Qinwen Huang, Alberto Bartesaghi, Ye Zhou, Hsuan-fu Liu)</p></td></tr><tr><th>10:50</th><td colspan="4" rowspan=4 class="stage-saturn">Keynote - John Jumper:<p style="font-weight:normal">Highly accurate protein structure prediction with AlphaFold</p></td></tr><tr><th>11:00</th></tr><tr><th>11:10</th></tr><tr><th>11:20</th></tr><tr><th>11:30</th><td colspan="4" rowspan=6 class="stage-earth">Poster Session<p style="font-weight:normal">Held on gather.town</p></td></tr><tr><th>11:40</th></tr><tr><th>11:50</th></tr><tr><th>12:00</th></tr><tr><th>12:10</th></tr><tr><th>12:20</th></tr><tr><th>12:30</th><td colspan="4" rowspan=6 class="stage-saturn">Panel Discussion</td></tr><tr><th>12:40</th></tr><tr><th>12:50</th></tr><tr><th>01:00</th></tr><tr><th>01:10</th></tr><tr><th>01:20</th></tr></table></div><div class="col-md-6"><table class="schedule"><tr><th>01:30</th><td colspan="4" rowspan=4 class="stage-earth">Keynote - Jane Richardson:<p style="font-weight:normal">The Very Early Days of Structural Biology before ML</p></td></tr><tr><th>01:40</th></tr><tr><th>01:50</th></tr><tr><th>02:00</th></tr><tr><th>02:10</th><td colspan="4" rowspan=1 class="stage-saturn">Break</td></tr><tr><th>02:20</th><td colspan="4" rowspan=1 class="stage-earth">Contributed Talk:<p style="font-weight:normal">Predicting cryptic pocket opening from protein structures using graph neural networks (Artur Meller, Michael Ward, Meghana Kshirsagar, Felipe Oviedo Perhavec, Jonathan Borowsky, Juan M Lavista Ferres, Greg Bowman)</p></td></tr><tr><th>02:30</th><td colspan="4" rowspan=1 class="stage-saturn">Contributed Talk:<p style="font-weight:normal">End-to-end learning of multiple sequence alignments with differentiable Smith-Waterman (Samantha Petti, Nicholas Bhattacharya, Roshan Rao, Justas Dauparas, Neil Thomas, Juannan Zhou, Alexander Rush, Peter K. Koo, Sergey Ovchinnikov)</p></td></tr><tr><th>02:40</th><td colspan="4" rowspan=1 class="stage-earth">Contributed Talk:<p style="font-weight:normal">Function-guided protein design by deep manifold sampling (Vladimir Gligorijevic, Stephen Ra, Daniel Berenberg, Richard Bonneau, Kyunghyun Cho)</p></td></tr><tr><th>02:50</th><td colspan="4" rowspan=1 class="stage-saturn">Contributed Talk:<p style="font-weight:normal">Deciphering antibody affinity maturation with language models and weakly supervised learning (Jeffrey A. Ruffolo, Jeffrey Gray, Jeremias Sulam)</p></td></tr><tr><th>03:00</th><td colspan="4" rowspan=1 class="stage-earth">Contributed Talk:<p style="font-weight:normal">Deep generative models create new and diverse protein structures (Zeming Lin, Tom Sercu, Alexander Rives)</p></td></tr><tr><th>03:10</th><td colspan="4" rowspan=6 class="stage-saturn">Poster Session<p style="font-weight:normal">Held on gather.town</p></td></tr><tr><th>03:20</th></tr><tr><th>03:30</th></tr><tr><th>03:40</th></tr><tr><th>03:50</th></tr><tr><th>04:00</th></tr><tr><th>04:10</th><td colspan="4" rowspan=2 class="stage-earth">Invited Talk - Derek Lowe</td></tr><tr><th>04:20</th></tr><tr><th>04:30</th><td colspan="4" rowspan=2 class="stage-saturn">Invited Talk - Regina Barzilay</td></tr><tr><th>04:40</th></tr><tr><th>04:50</th><td colspan="4" rowspan=2 class="stage-earth">Invited Talk - Amy Keating</td></tr><tr><th>05:00</th></tr><tr><th>05:10</th><td colspan="4" rowspan=1 class="stage-saturn">Closing Remarks</td></tr><tr><th>05:20</th><td colspan="4" rowspan=1 class="stage-earth">Social Hour<p style="font-weight:normal">Held on gather.town</p></td></tr></table></div></row>
			<!--
            <div class="col-md-6 col-md-offset-3 text-center wrap_title">
                <h2 class="section-heading">Schedule</h2>
                <h4 class="section-heading">Coming soon!</h4>
            </div>
            <div class="row">
                <div class="col-md-6">
                    <table class="schedule">
                        <tr><th>08:00</th>
                            <td colspan="4" rowspan="1" class="stage-saturn"><a href="https://neurips.cc/virtual/2020/protected/workshop_16137.html">Opening Remarks</a></td>
                        </tr>
                        <tr><th>08:10</th>
                            <td colspan="4" rowspan="4" class="stage-earth"><a href="https://neurips.cc/virtual/2020/protected/workshop_16137.html">Keynote - Michael Levitt:</a><p style="font-weight:normal">Is Basic Science Needed for Significant and Fundamental Discoveries</p></td>
                        </tr>
                        <tr><th>08:20</th></tr>
                        <tr><th>08:30</th></tr>
                        <tr><th>08:40</th></tr>
                        <tr><th>08:50</th>
                            <td colspan="4" rowspan="2" class="stage-saturn"><a href="https://slideslive.com/38938191">Invited Talk - Charlotte Deane:</a> <p style="font-weight:normal">Predicting the conformational ensembles of proteins</p></td>
                        </tr>
                        <tr><th>9:00</th></tr>
                        <tr><th>9:10</th>
                            <td colspan="4" rowspan="2" class="stage-earth"><a href="https://slideslive.com/38938201">Invited Talk - Frank Noe:</a> <p style="font-weight:normal">Deep Markov State Models versus Covid-19</p></td>
                        </tr>
                        <tr><th>9:20</th></tr>
                        <tr><th>9:30</th>
                            <td colspan="4" rowspan="2" class="stage-saturn"><a href="https://slideslive.com/38938193">Invited Talk - Andrea Thorn:</a> <p style="font-weight:normal">Finding Secondary Structure in Cryo-EM maps: HARUSPEX</p></td>
                        </tr>
                        <tr><th>9:40</th></tr>
                        <tr><th>9:50</th>
                            <td colspan="4" rowspan="3" class="stage-earth">Break</td>
                        </tr>
                        <tr><th>10:00</th></tr>
                        <tr><th>10:10</th></tr>
                        <tr><th>10:20</th>
                            <td colspan="4" rowspan="4" class="stage-saturn"><a href="https://slideslive.com/38938195">Keynote - David Baker:</a> <p style="font-weight:normal">Rosetta design of COVID antivirals and diagnostics</p></td>
                        </tr>
                        <tr><th>10:30</th></tr>
                        <tr><th>10:40</th></tr>
                        <tr><th>10:50</th></tr>
                        <tr><th>11:00</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://slideslive.com/38942411">Contributed Talk:</a> <span style="font-weight:normal">Predicting Chemical Shifts with Graph Neural Networks (Ziyue Yang, Maghesree Chakraborty, Andrew White)</span></td>
                        </tr>
                        <tr><th>11:10</th>
                            <td colspan="4" rowspan="1" class="stage-saturn"><a href="https://slideslive.com/38938199">Contributed Talk:</a> <span style="font-weight:normal">Cryo-ZSSR: multiple-image super-resolution based on deep internal learning (Qinwen Huang, Ye Zhou, Xiaochen Du, Reed Chen, Jianyou Wang, Cynthia Rudin, Alberto Bartesaghi)</span></td>
                        </tr>
                        <tr><th>11:20</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://slideslive.com/38938198">Contributed Talk:</a> <span style="font-weight:normal">Wasserstein K-Means for Clustering Tomographic Projections (Rohan Rao, Amit Moscovich, Amit Singer)</span></td>
                        </tr>
                        <tr><th>11:30</th>
                            <td colspan="4" rowspan="6" class="stage-saturn"><a href="https://neurips.gather.town/app/1UBzVZ8TnX03EOsa/mlsb">Poster Session</a><span style="font-weight:normal">Held on gather.town</span></td>
                        </tr>
                        <tr><th>11:40</th></tr>
                        <tr><th>11:50</th></tr>
                        <tr><th>12:00</th></tr>
                        <tr><th>12:10</th></tr>
                        <tr><th>12:20</th></tr>
                        <tr><th>12:30</th>
                            <td colspan="4" rowspan="3" class="stage-earth">Lunch</td>
                        </tr>
                        <tr><th>12:40</th></tr>
                        <tr><th>12:50</th></tr>
                    </table>
                </div>
                <div class="col-md-6">
                    <table class="schedule">
                        <tr><th>01:00</th>
                            <td colspan="4" rowspan="6" class="stage-earth"><a href="https://videos.neurips.cc/search/MLSB/video/slideslive-38951540">Panel Discussion:</a> <span style="font-weight:normal">Future of ML for Structural Biology</span></td>
                        </tr>
                        <tr><th>01:10</th></tr>
                        <tr><th>01:20</th></tr>
                        <tr><th>01:30</th></tr>
                        <tr><th>01:40</th></tr>
                        <tr><th>01:50</th></tr>
                        <tr><th>2:00</th>
                            <td colspan="4" rowspan="2" class="stage-saturn"><a href="https://neurips.cc/virtual/2020/protected/workshop_16137.html">Invited Talk - Possu Huang:</a><span style="font-weight:normal">Jump starting an evolution by protein design through deep learning of protein structures</span></td>
                        </tr>
                        <tr><th>2:10</th></tr>
                        <tr><th>2:20</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://slideslive.com/38938399">Contributed Talk:</a> <span style="font-weight:normal">ProGen: Language Modeling for Protein Generation (Ali Madani, Bryan McCann, Nikhil Naik, Nitish Shirish Keskar, Namrata Anand, Alexander E Chu, Raphael R Eguchi, Po-Ssu Huang, Richard Socher)</span></td>
                        </tr>
                        <tr><th>2:30</th>
                            <td colspan="4" rowspan="1" class="stage-saturn"><a href="https://slideslive.com/38942412">Contributed Talk:</a> <span style="font-weight:normal">Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences (Alexander Rives, Joshua Meier, Tom Sercu, Siddharth Goyal, Zeming Lin, Demi Guo, Myle Ott, Larry Zitnick, Jerry Ma, Rob Fergus)</span></td>
                        </tr>
                        <tr><th>2:40</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://slideslive.com/38938398">Contributed Talk:</a> <span style="font-weight:normal">SidechainNet: An All-Atom Protein Structure Dataset for Machine Learning (Jonathan King, David Koes)</span></td>
                        </tr>
                        <tr><th>2:50</th>
                            <td colspan="4" rowspan="1" class="stage-saturn"><a href="https://slideslive.com/38938200">Contributed Talk:</a> <span style="font-weight:normal">Generating 3D Molecular Structures Conditional on a Receptor Binding Site with Deep Generative Models (Tomohide Masuda, Matthew Ragoza, David Koes)</span></td>
                        </tr>
                        <tr><th>3:00</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://slideslive.com/38942410">Contributed Talk:</a> <span style="font-weight:normal">Learning from Protein Structure with Geometric Vector Perceptrons (Bowen Jing, Stephan Eismann, Patricia Suriana, Raphael Townshend, Ron Dror)</span></td>
                        </tr>
                        <tr><th>3:10</th>
                            <td colspan="4" rowspan="6" class="stage-saturn"><a href="https://neurips.gather.town/app/1UBzVZ8TnX03EOsa/mlsb">Poster Session</a> <span style="font-weight:normal">Held on gather.town</span></td>
                        </tr>
                        <tr><th>3:20</th></tr>
                        <tr><th>3:30</th></tr>
                        <tr><th>3:40</th></tr>
                        <tr><th>3:50</th></tr>
                        <tr><th>4:00</th></tr>
                        <tr><th>4:10</th>
                            <td colspan="4" rowspan="2" class="stage-earth"><a href="https://slideslive.com/38938196">Invited Talk - Mohammed AlQuraishi: </a><p style="font-weight:normal">(Nearly) end-to-end differentiable learning of protein structure</p></td>
                        </tr>
                        <tr><th>4:20</th></tr>
                        <tr><th>4:30</th>
                            <td colspan="4" rowspan="2" class="stage-saturn"><a href="https://slideslive.com/38938203">Invited Talk - Chaok Seok:</a> <span style="font-weight:normal">Ab initio protein structure prediction by global optimization of neural network energy: Can AI learn physics?</span></td>

                        </tr>
                        <tr><th>4:40</th></tr>
                        <tr><th>4:50</th>
                            <td colspan="4" rowspan="1" class="stage-earth"><a href="https://neurips.cc/virtual/2020/protected/workshop_16137.html">Closing Remarks</a></td>
                        </tr>
                        <tr><th>5:00</th>
                            <td colspan="4" rowspan="1" class="stage-saturn"><a href="https://neurips.gather.town/app/1UBzVZ8TnX03EOsa/mlsb">Happy Hour</a><span style="font-weight:normal">Held on gather.town</span></td>
                        </tr>
                    </table>
                </div> <!--col -->
            </div> <!-- row -->
        </div> <!-- container -->
    </div> <!-- schedule -->

	<div id="accepted" class="content-section-b" style="border-top: 0">
            <div class="container" id="accepted">
		    <h2 class="section-heading">Accepted Papers</h2>
		    <div class="row wow fadeInLeftBig"  data-animation-delay="200">
				<div class="col-md-12 wow fadeInRightBig"  data-animation-delay="200">
                    <ul class="paperlist">
						<li><p class="title">Simple End-to-end Deep Learning Model for CDR-H3 Loop Structure Prediction</p>
							<p class="authors">NataliaZenkova, Ekaterina Sedykh, Timofei Ermak, Tatiana Shugaeva, Vladislav Strashko, Aleksei  Shpilman</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Simple_End-to-end_Deep_Learning.pdf">[paper]</a>
								<a href="https://arxiv.org/abs/2111.10656">[arXiv]</a>								
							</p>
						</li>
						<li><p class="title">MOLUCINATE: A Generative Model for Molecules in 3D Space</p>
							<p class="authors">Michael Arcidiacono, David Koes</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_MOLUCINATE:_A_Generative_Model.pdf">[paper]</a>
								<a href="https://arxiv.org/abs/2109.15308">[arXiv]</a>								
							</p>
						</li>
						<li><p class="title">Predicting cryptic pocket opening from protein structures using graph neural networks</p>
							<p class="authors">Artur Meller, Michael Ward, Meghana Kshirsagar, Felipe Oviedo Perhavec, Jonathan Borowsky, Juan M Lavista Ferres, Greg Bowman</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Predicting_cryptic_pocket_opening.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Active site sequence representation of human kinases outperforms full sequence for affinity prediction</p>
							<p class="authors">Jannis Born, Tien Huynh, Astrid Stroobants, Wendy Cornell, Matteo Manica</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Active_site_sequence_representation.pdf">[paper]</a>
								<a href="https://chemrxiv.org/engage/chemrxiv/article-details/60ed9f6eaf9cdb0a4ddb803f">[ChemRxiv]</a>	
							</p>
						</li>
						<li><p class="title">Interpretable Pairwise Distillations for Generative Protein Sequence Models</p>
							<p class="authors">Christoph Feinauer, Barthélémy Meynard, Carlo Lucibello</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Interpretable_Pairwise_Distillations_for.pdf">[paper]</a>
								<a href="https://www.biorxiv.org/content/10.1101/2021.10.14.464358v1">[bioRxiv]</a>	
							</p>
						</li>
						<li><p class="title">Dock2D: Toy datasets for the molecular recognition problem</p>
							<p class="authors">Georgy Derevyanko, Siddharth Bhadra-Lobo, Guillaume Lamoureux</p>
							<p class="preprints">
							</p>
						</li>
						<li><p class="title">Inferring a Continuous Distribution of Atom Coordinates from Cryo-EM Images using VAEs</p>
							<p class="authors">Dan Rosenbaum, Marta Garnelo, Michal Zielinski, Charlie Beattie, Ellen Clancy, Andrea Huber, Pushmeet Kohli, Andrew Senior, John Jumper, Carl Doersch, S. M. Ali Eslami, Olaf Ronneberger, Jonas Adler</p>
							<p class="preprints">
								<a href="https://arxiv.org/abs/2106.14108">[arXiv]</a>
							</p>
						</li>
						<li><p class="title">Studying signal peptides with attention neural networks informs cleavage site predictions</p>
							<p class="authors">Patrick Bryant, Arne  Elofsson</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Studying_signal_peptides_with.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Protein sequence sampling and prediction from structural data</p>
							<p class="authors">Gabriel A Orellana, Javier Caceres-Delpiano, Roberto Ibanez, Michael Dunne, Leonardo Alvarez</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Protein_sequence_sampling_and.pdf">[paper]</a>
								<a href="https://www.biorxiv.org/content/10.1101/2021.09.06.459171v3">[bioRxiv]</a>	
							</p>
						</li>
						<li><p class="title">Structure-aware generation of drug-like molecules</p>
							<p class="authors">Pavol Drotar, Arian R. Jamasb, Ben J Day, Catalina Cangea, Pietro Lió</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Structure-aware_generation_of_drug-like.pdf">[paper]</a>
								<a href="https://arxiv.org/abs/2111.04107">[arXiv]</a>
							</p>
						</li>
						<li><p class="title">Turning high-throughput structural biology into predictive drug design</p>
							<p class="authors">Kadi L Saar, Daren Fearon, John Chodera, Frank von Delft, Alpha Lee</p>
							<p class="preprints">
								<a href="https://www.biorxiv.org/content/10.1101/2021.10.15.464568v1">[bioRxiv]</a>
							</p>
						</li>
						<li><p class="title">DLA-Ranker: Evaluating protein docking conformations with many locally oriented cubes</p>
							<p class="authors">Yasser  Mohseni Behbahani, Élodie Laine, Alessandra Carbone</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_DLA-Ranker:_Evaluating_protein_docking.pdf">[paper]</a>
								<a href="https://www.biorxiv.org/content/10.1101/2021.10.26.465898v1">[bioRxiv]</a>
							</p>
						</li>
						<li><p class="title">Predicting single-point mutational effect on protein stability</p>
							<p class="authors">Kit Sang Chu, Justin Siegel</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Predicting_single-point_mutational_effect.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Exploring ∆∆G prediction with Siamese Networks</p>
							<p class="authors">Andrew McNutt, David Koes</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Exploring_∆∆G_prediction_with.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Residue characterization on AlphaFold2 protein structures using graph neural networks</p>
							<p class="authors">Nasim Abdollahi</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Residue_characterization_on_AlphaFold2.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">A kernel for continuously relaxed, discrete Bayesian optimization of protein sequences</p>
							<p class="authors">Yevgen Zainchkovskyy, Simon Bartels, Soren Hauberg, Jes Frellsen, Wouter Krogh Boomsma</p>
							<p class="preprints">
							</p>
						</li>
						<li><p class="title">Deciphering antibody affinity maturation with language models and weakly supervised learning</p>
							<p class="authors">Jeffrey A Ruffolo, Jeffrey Gray, Jeremias Sulam</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Deciphering_antibody_affinity_maturation.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">HelixGAN: A bidirectional Generative Adversarial Network with search in latent space for generation under constraints</p>
							<p class="authors">Xuezhi Xie, Philip Kim</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_HelixGAN:_A_bidirectional_Generative.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Weakly Supervised Learning for Joint Image Denoising and Protein Localization in Cryo-EM</p>
							<p class="authors">QinwenHuang, Alberto  Bartesaghi, Ye Zhou, Hsuan-fu Liu</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Weakly_Supervised_Learning_for.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Deep generative models create new and diverse protein structures</p>
							<p class="authors">Zeming Lin, Tom Sercu, yann lecun, Alex Rives</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Deep_generative_models_create.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Adapting protein language models for rapid DTI prediction</p>
							<p class="authors">Samuel Sledzieski, Rohit Singh, Lenore Cowen, Bonnie Berger</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Adapting_protein_language_models.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Generative Language Modeling for Antibody Design</p>
							<p class="authors">Richard W. Shuai, Jeffrey A Ruffolo, Jeffrey Gray</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Generative_Language_Modeling_for.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">End-to-end learning of multiple sequence alignments with differentiable Smith-Waterman</p>
							<p class="authors">Samantha Petti, Nicholas Bhattacharya, Roshan M Rao, Justas Dauparas, Neil Thomas, Juannan Zhou, Alexander Rush, Peter K. Koo, Sergey Ovchinnikov</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_End-to-end_learning_of_multiple.pdf">[paper]</a>
								<a href="https://www.biorxiv.org/content/10.1101/2021.10.23.465204v1">[bioRxiv]</a>
							</p>
						</li>
						<li><p class="title">TERMinator: A Neural Framework for Structure-Based Protein Design using Tertiary Repeating Motifs</p>
							<p class="authors">AlexLi, Vikram Sundar, Gevorg Grigoryan, Amy  Keating</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_TERMinator:_A_Neural_Framework.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Learning physics confers pose-sensitivity in structure-based virtual screening</p>
							<p class="authors">Pawel Gniewek, Bradley Worley, Kate Stafford, Brandon Anderson, Henry van den Bedem</p>
							<p class="preprints">
								<a href="https://arxiv.org/abs/2110.15459">[arXiv]</a>
							</p>
						</li>
						<li><p class="title">MSA-Conditioned Generative Protein Language Models for Fitness Landscape Modelling and Design</p>
							<p class="authors">Alex Hawkins-Hooker, David T. Jones, Brooks Paige</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_MSA-Conditioned_Generative_Protein_Language.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">Real-valued Sidechain Dihedrals Prediction Using Relation-Shape Convolution</p>
							<p class="authors">Xiyao Long, Roland Dunbrack, Maxim  Shapovalov</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Real-valued_Sidechain_Dihedrals_Prediction.pdf">[paper]</a>
							</p>
						</li>
						<li><p class="title">AutoFoldFinder: An Automated Adaptive Optimization Toolkit for De Novo Protein Fold Design</p>
							<p class="authors">Shuhao Zhang, Xu Youjun, Jianfeng Pei, Luhua Lai</p>
							<p class="preprints">
							</p>
						</li>
						<li><p class="title">Function-guided protein design by deep manifold sampling</p>
							<p class="authors">Vladimir Gligorijevic, Stephen Ra, Daniel Berenberg, Richard Bonneau, Kyunghyun Cho</p>
							<p class="preprints">
								<a href="papers_2021/MLSB2021_Function-guided_protein_design_by.pdf">[paper]</a>
							</p>
						</li>
		    </ul>
                </div>
            </div>
            </div>
        </div>

        <div id="organizers" class="content-section-a" style="border-top: 0">
            <div class="container" id="organizers">
		    <h2 class="section-heading">Organizers</h2>
		    <div class="row wow fadeInRightBig"  data-animation-delay="200">
				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Namrata <br> Anand</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/namrata_anand.png" alt="Photo of Namrata Anand"></center>
					<h5 class="section-heading"><center><i>Stanford University</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Bonnie <br> Berger</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/bonnie_berger.png" alt="Photo of Bonnie Berger"></center>
					<h5 class="section-heading"><center><i>MIT</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Wouter <br> Boomsma</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/wouter_boomsma.png" alt="Photo of Wouter Boomsma"></center>
					<h5 class="section-heading"><center><i>University of Copenhagen</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Erika <br> DeBenedictis</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="./img/organizer/erika_debenedictis.jpg" alt="Photo of Erika DeBenedictis"></center>
					<h5 class="section-heading"><center><i>University of Washington</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
                    <h4 class="section-heading"><center>Stephan <br> Eismann </center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/stephan_eismann.png" alt="Photo of Stephan Eismann"></center>
					<h5 class="section-heading"><center><i>Stanford University</i></center></h5>
				</div>
			</div>
			<div class="row wow fadeInRightBig"  data-animation-delay="200">

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>John <br> Ingraham</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/john_ingraham.jpg" alt="Photo of John Ingraham"></center>
					<h5 class="section-heading"><center><i>Generate Biomedicines</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
                    <h4 class="section-heading"><center>Sergey <br> Ovchinnikov</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/sergey_ovchinnikov.jpg" alt="Photo of Sergey Ovchinnikov"></center>
					<h5 class="section-heading"><center><i>Harvard University</i></center></h5>
				</div>

                <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
                    <h4 class="section-heading"><center>Roshan <br> Rao</center></h4>
                    <center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/roshan_rao.jpg" alt="Photo of Roshan Rao"></center>
                    <h5 class="section-heading"><center><i>UC Berkeley</i></center></h5>
                </div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Raphael <br> Townshend</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="./img/organizer/raphael_townshend.png" alt="Photo of Raphael Townshend"></center>
					<h5 class="section-heading"><center><i>Stanford University</i></center></h5>
				</div>

				<div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
					<h4 class="section-heading"><center>Ellen <br> Zhong</center></h4>
					<center><img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/ellen_zhong.jpg" alt="Photo of Ellen Zhong"></center>
					<h5 class="section-heading"><center><i>MIT</i></center></h5>
				</div>
            </div>
        </div>
    </div>

	<div id="related" class="content-section-b" style="border-top: 0">
        <div class="container" id="related">
		    <h2 class="section-heading">Related Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead"> Here are some <a href="https://neurips.cc/Conferences/2021/Schedule?type=Workshop">related workshops at NeurIPS</a> covering structure, biology, and healthcare this year:
                    <ul class="bulleted">
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21870">Learning Meaningful Representations for Life</a></li>
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21862">Machine Learning and the Physical Sciences</a> </li>
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21834">MLPH: Machine Learning in Public Health</a></li>
						<li><a href="https://neurips.cc/Conferences/2021/Schedule?showEvent=21849">AI for Science: Mind the Gaps</a></li>
						<li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21882">Differentiable Programming</a></li>
                    </ul>
				</div>
            </div>
        </div>
        <br/>
		<br/>
		<div class="container">
			<div class="row wow fadeInLeftBig"  data-animation-delay="200">
				<div class="col-md-12 wow fadeInLeftBig"  data-animation-delay="200">
					<div class="center">
						<img  class="center" style="" height="200" src="./img/letters.png" alt="Letters">
					</div>
				</div>
			</div>
		</div>
	</div>

	<div id="organizers" class="content-section-a" style="border-top: 0">
        <div class="container" id="sponsors">
		    <h2 class="section-heading">Sponsors</h2>
		    <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
				<div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200">
					<img  class="center" style="" height="100" src="./img/ATOMICAI_logo+name_Yellow_Red_black.png" alt="Atomic Logo">
                </div>
				<div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200">
					<img  class="center" style="" height="75" src="./img/deepmind-logo.png" alt="Deepmind Logo">
                </div>
				<div class="col-md-4"  data-animation-delay="200">
					<img  class="center" style="" height="60" src="./img/generate-bio-afpc-logo-cmyk-black.png" alt="Generate Bio Logo">
                </div>
            </div>
        </div>
    </div>

    <footer>
      <div class="container">
        <div class="row">
          <div class="col-md-12" id="contact">
            <h3 class="footer-title">Contact</h3>
            <p> Please contact <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a> with any questions.
            </p>


          </div> <!-- /col-xs-7 -->
        </div>
      </div>
    </footer>

    <!-- JavaScript -->
    <script src="js/jquery-1.10.2.js"></script>
    <script src="js/bootstrap.js"></script>
	<script src="js/owl.carousel.js"></script>
	<script src="js/script.js"></script>
	<!-- StikyMenu -->
	<script src="js/stickUp.min.js"></script>
	<script type="text/javascript">
	  jQuery(function($) {
		$(document).ready( function() {
		  $('.navbar-default').stickUp();

		});
	  });

	</script>
	<!-- Smoothscroll -->
	<script type="text/javascript" src="js/jquery.corner.js"></script>
	<script src="js/wow.min.js"></script>
	<script>
	 new WOW().init();
	</script>
	<script src="js/classie.js"></script>
	<!-- Magnific Popup core JS file -->
	<script src="js/jquery.magnific-popup.js"></script>
</body>

</html>
