<!doctype html>
<!-- FlatFy Theme - Andrea Galanti /-->
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->
<!--[if IE 9]>    <html class="no-js ie9" lang="en"> <![endif]-->
<!--[if gt IE 9]><!-->
<html>
<!--<![endif]-->
<script>
function showhide(id) {
  var x = document.getElementById(id);
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
<meta name="description" content="Workshop at the 37th Conference on Neural Information Processing Systems">
<meta name="author" content="">
<title>Machine Learning in Structural Biology</title>

<!-- Bootstrap core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">

<!-- Custom Google Web Font -->
<link href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
<link href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,100italic,300italic,400italic,700italic,900italic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=Arvo:400,700" rel="stylesheet" type="text/css">

<!-- Custom CSS-->
<link href="css/general.css" rel="stylesheet">

<!-- Owl-Carousel -->
<link href="css/custom.css" rel="stylesheet">
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/owl.theme.css" rel="stylesheet">
<link href="css/style.css" rel="stylesheet">
<link href="css/animate.css" rel="stylesheet">

<!-- Magnific Popup core CSS file -->
<link rel="stylesheet" href="css/magnific-popup.css">
<script src="js/modernizr-2.8.3.min.js"></script> <!-- Modernizr /-->
<!--[if IE 9]>
		<script src="js/PIE_IE9.js"></script>
	<![endif]-->
<!--[if lt IE 9]>
		<script src="js/PIE_IE678.js"></script>
	<![endif]-->

<!--[if lt IE 9]>
		<script src="js/html5shiv.js"></script>
	<![endif]-->

</head>

<body id="home">

<!-- Preloader -->
<div id="preloader">
  <div id="status"></div>
</div>

<!-- FullScreen -->
<div class="intro-header">
  <div class="col-xs-12 text-center abcen1">
    <br><br>
    <h1 class="h1_home wow fadeIn" data-wow-delay="0.4s">Machine Learning in Structural Biology</h1>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Workshop at the 38th Conference on Neural Information Processing Systems</h3>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">15th December 2024</h3>
    <ul class="list-inline intro-social-buttons">
      <!--<li><a href="https://neurips.cc/virtual/2022/workshop/50005" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">Livestream</span></a> </li>-->
      <!--
		<li><a href="https://eventhosts.gather.town/app/1UBzVZ8TnX03EOsa/mlsb" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">GatherTown</span></a>
		</li>
        -->
      <!--
				<li id="download" ><a href="" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">Gather</span></a>
				</li>
                -->
    </ul>
  </div>
  <!-- /.container -->
  <div class="col-xs-12 text-center abcen wow fadeIn">
    <div class="button_down "> <a class="imgcircle wow bounceInUp" data-wow-duration="1.5s"  href="#about"> <img class="img_scroll" src="img/icon/circle.png" alt="Go down to content"> </a> </div>
  </div>
</div>

<!-- NavBar-->
<nav class="navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
      <a class="navbar-brand" href="#home">MLSB</a> </div>
    <div class="collapse navbar-collapse navbar-right navbar-ex1-collapse">
      <ul class="nav navbar-nav">
        <li class="menuItem"><a href="#about">About</a></li>
        <li class="menuItem"><a href="#callforpapers">Call for Papers</a></li>
       
        <li class="menuItem"><a href="#presenterinfo">Presenter Information</a></li>
        <!-- <li class="menuItem"><a href="#faq">FAQ</a></li>-->
        <li class="menuItem"><a href="#invitedspeakers">Invited Speakers</a></li>
        <li class="menuItem"><a href="#challenge">Challenge</a></li>
        <!-- <li class="menuItem"><a href="#preregistration">Pre-Registration</a></li> -->
        <li class="menuItem"><a href="#schedule">Schedule</a></li>
        <li class="menuItem"><a href="#accepted">Accepted Papers</a></li>
        <li class="menuItem"><a href="#organizers">Organizers</a></li>
        <li class="menuItem"><a href="#related">Past Workshops</a></li>
        <li class="menuItem"><a href="#sponsors">Sponsors</a></li>
        <li class="menuItem"><a href="#contact">Contact</a></li>
      </ul>
    </div>
  </div>
</nav>

<!-- About -->
<div id="about" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">About</h2>
        <p class="lead"> Structural biology, the study of the 3D structure or shape of proteins and other biomolecules, has been transformed by breakthroughs from machine learning algorithms. Machine learning models are now routinely used by experimentalists to predict structures to aid in hypothesis generation and experimental design, accelerate the experimental process of structure determination (e.g. computer vision algorithms for cryo-electron microscopy), and have become a new industry standard for bioengineering new protein therapeutics (e.g. large language models for protein design). Despite all of this progress, there are still many active and open challenges for the field, such as modeling protein dynamics, predicting the structure of other classes of biomolecules such as RNA, learning and generalizing the underlying physics driving protein folding, and relating the structure of isolated proteins to the in vivo and contextual nature of their underlying function. These challenges are diverse and interdisciplinary, motivating new kinds of machine learning methods and requiring the development and maturation of standard benchmarks and datasets.  </p>

        <p class="lead">  Machine Learning in Structural Biology (MLSB),
          seeks to bring together field experts, practitioners, and students from across   academia, industry research groups, and pharmaceutical companies
          to focus on these new challenges and opportunities. This year, MLSB aims to bridge the theoretical and practical by addressing the outstanding computational and experimental problems at the forefront of our field. The intersection of artificial intelligence and structural biology promises to unlock new scientific discoveries and develop powerful design tools. </p>
        <p class="lead"> MLSB will be an in-person <a href="https://neurips.cc">NeurIPS</a> workshop on 15th December 2024 in MTG Rooms 11 & 12 at the Vancouver Convention Center.</p>
        <p class="lead"> Please contact the organizers at <a href = "mailto:workshopmlsb@gmail.com?">workshopmlsb@gmail.com</a> with any questions.</p>
        <p class="lead"> Stay updated on changes and workshop news by joining <a href="https://groups.google.com/g/workshopmlsb">our mailing list</a>. </p>
      </div>
    </div>
  </div>
</div>
<!--<div id ="faq" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">FAQ</h2>
        <p> <b>WIFI</b> <br/>
          network name: NeurIPS <br/>
          password: conference </p>
        <p> <b>SCHEDULE</b> MLSB will begin at 8:30am CST and end at 5:00pm CST on Saturday, Dec 3. The physical workshop will take place in Room 288 - 289 in the New Orleans Convention Center. The schedule of talks and poster sessions is available at our website, mlsb.io and on the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. </p>
        <p> <b>PANEL</b> -
          We will be hosting a panel discussion including our invited speakers on research directions in Machine Learning for Structural Biology from 2:50-3:50p CST. We are soliciting additional discussion topics and questions -- let us know what you would like to hear about by filling out this google form: <a href="https://forms.gle/fiJ6ZzsmepKL87tMA">https://forms.gle/fiJ6ZzsmepKL87tMA</a>. </p>
        <p> <b>POSTER SESSIONS</b> -
          We have scheduled two poster sessions at 11:15-12:15pm CST and 3:50-4:50pm CST. Posters can be set up before the conference begins, between 8:00-8:30am, during the break between 9:45-10:05am, or at the beginning of the first poster session.

          We ask that posters are 24"W x 36"H, as there may be space constraints in the room that limit the display of wider posters. However, if you already have a poster of a different size, it is acceptable to use that. </p>
        <p> <b>LUNCH + HAPPY HOUR</b> -
          MLSB will be providing boxed lunches to attendees during the workshop. In addition, there will be a happy hour during the afternoon poster session, where drinks will be served (non-alcoholic options will be available). </p>
        <p> <b>ORALS</b> -
          Each oral will have a 15 minute timeslot. We anticipate that talks should last approximately 10 minutes, allowing 5 minutes for questions and transition between speakers. If you are presenting in-person, you may check slides before the workshop (8:00-8:30am CST) or during the break (9:45-10:05am CST). </p>
        <p> Please check the schedule on our website, mlsb.io, for your presentation time. If you are not able to make this time, let us know as soon as possible. Additionally, below we have listed the format we currently have for each oral presentation (In-person, Live over Zoom, or Prerecorded). If this information is not correct, please let us know. You can contact us at workshopmlsb@gmail.com with any issues. </p>
        <p> <b>VIRTUAL PARTICIPATION</b> -
          MLSB will have some hybrid elements. There will be a livestream (a link is available at the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. Chat will be available on the NeurIPS workshop site for asking questions.

          Currently, there is no official venue for presenting posters virtually at MLSB. However, we are considering options to enable greater virtual participation over the next few days. If you have a poster at MLSB but plan to participate only virtually, please let us know by adding your name to this <a href="https://forms.gle/ntQDsULXaVkzVXYR8">google form</a> as soon as possible. </p>
      </div>
    </div>
  </div>
</div>-->

<div id ="presenterinfo" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Presenter Information</h2>
        <p>Congratulations to all accepted presenters! Please find some information on deadlines and expectations leading up to the MLSB Workshop!</p>
        <h3 class="section-heading">Posters</h3>
        <p>We ask all authors to prepare a poster that can be presented as part of our workshop. Posters must be 24W x 36H inches and will be taped to the wall. Poster boards will not be provided at the workshop. <b>We specifically ask for portrait layout because we will be tight on wall space.</b></p>
        <p> Additionally, a virtual copy of each poster must be uploaded to the NeurIPS poster upload <a href="https://neurips.cc/PosterUpload">portal</a>, by <b>Thursday, December 12</b>. Posters must be PNG with no more than 5120 width x 2880 height (no more than 10 MB). Thumbnail images should be 320 width x 256 height PNG and no more than 5 MB. 
          We know these are different dimensions than what we're asking for in-person posters, the poster upload dimensions are set by NeurIPS.
        </p><p>
          Users should log in using the neurips.cc account associated with their CMT email address. 
          If they did not already have a neurips.cc account, then it should have automatically been created and can be accessed by <a href="https://nips.cc/Profile/resetpassword">resetting the password</a>.
        </p>
        <h3 class="section-heading">Paper Camera-Ready</h3>
        <p>De-anonymized, camera-ready versions of the workshop paper will be due on <a href="https://cmt3.research.microsoft.com/MLSB2024">Microsoft CMT</a> by <b>Monday, Dec 2</b>. 
          Papers must indicate that they are NeurIPS MLSB workshop papers by using the modified NeurIPS style file <a href="https://raw.githubusercontent.com/mlsbio/mlsbio.github.io/master/neurips_mlsb_2024.sty">here.</a>
          Papers should be compiled with the `final` argument, e.g.
          <b>\usepackage[final]{neurips_mlsb_2024}</b></p>
        <p>We plan to make all camera-ready submitted papers available on the workshop website (<a href="https://www.mlsb.io/">https://www.mlsb.io/</a>). 
          If you would prefer that your work not be shared, then there is no need to submit a camera-ready version..</p>
        <h3 class="section-heading">Travel Award</h3>
        <p>This year we will try to cover as many workshop registrations as possible for student/academic attendees with oral presentations or posters who need financial assistance.
          If you would like to be considered, please fill out the following <a href="https://forms.gle/hK9AeWaMx8iiRhKp8">form</a> by <s><b>Friday, Nov 15</b></s> <b>Friday, Nov 8</b>.
          If you have any questions, please don't hesitate to contact us at <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a>.</p>
        <h3 class="section-heading">Key Dates</h3>
        <p class=""><u>Application for Registration Reimbursement</u>: <b>Friday, <s>November 15th, 2024</s> November 8th, 2024, at 11:59PM, Anywhere on Earth.</b></p>
        <p class=""><u>Camera-Ready PDF due on Microsoft CMT</u>: <b>Monday, December 2nd, 2024</b>.</p>
        <p class=""><u>Poster due</u>: <b>Thursday, December 12th, 2024</b>.</p>
      </div>
    </div>
  </div>
</div>



<!-- Call for Papers -->
<div id ="callforpapers" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Call For Papers</h2>
        <p>We welcome submissions of short papers leveraging machine learning to address problems in structural biology, including but not limited to:</p>
        <p>
        <ul class="bulleted">
          <li>Prediction of biomolecular structures, complexes, and interactions</li>
          <li>Design of or generative models for structure and/or sequence</li>
          <li>Methods for structure determination / biophysics (Cryo-EM/ET, NMR, crystallography, single-molecule methods, etc.)</li>
          <li>Geometric and symmetry-aware deep learning</li>
          <li>Conformational change, ensembles, and dynamics</li>
          <li>Integration of biomolecular physics</li>
          <li>Function and property prediction</li>
          <li>Structural bioinformatics and systems biology</li>
          <li>Therapeutic screening and design</li>
          <li>Language models and other implicit representations of protein structure</li>
          <li>Forward-looking position papers</li>
        </ul>
      </p>
        <p>We request anonymized PDF submissions by <strong>Friday, September 20, 2024, at 11:59PM, AoE (anywhere on earth)</strong> through our submission website on <a href="https://cmt3.research.microsoft.com/MLSB2024">CMT</a>. </p></p>
        <p>Papers should present novel work that has not been previously accepted at an archival venue at the time of submission.  Submissions should be a <em>maximum</em> of 5 pages (excluding references and appendices) in PDF format, using the <a href="https://media.neurips.cc/Conferences/NeurIPS2024/Styles.zip">NeurIPS style files</a>, and fully anonymized as per the requirements of NeurIPS. The NeurIPS checklist can be omitted from the submission. Submissions meeting these criteria will go through a light, double-blind review process. Reviewer comments will be returned to the authors as feedback.</p>
        <p>Accepted papers will be invited to present a poster at the workshop, with nominations of spotlight talks at the discretion of the organizers.</p>
        <p>New this year, we will have two special tracks for models for predicting protein-protein and protein-ligand interactions, evaluated on two new large-scale benchmarks, <a href="https://www.pinder.sh/">PINDER</a> and <a href="https://www.plinder.sh/">PLINDER</a>. The highest-performing open-source methods from these two tracks will receive invitations to a spotlight presentation. <strong>Stay tuned for more information on how to submit to these tracks.</strong>
        </p>
        <p>Like last year, authors that commit to open-sourcing code, model weights, and datasets used in the work will be given precedence for spotlight talks. This change only affects consideration for spotlights. Submissions that cannot make this commitment will still be considered for posters and will not be penalized for acceptance.</p>
        <p>This workshop is considered non-archival, however, authors of accepted contributions will have the option to make their work available through the workshop website. Presentation of work that is concurrently in submission is welcome. We welcome papers sharing encouraging work-in-progress results or forward-looking position papers that would benefit from feedback and community discussion at our workshop.</p>
        <h3 class="section-heading">Important Dates</h3>
        <p class=""><u>Submission Deadline</u>: <b>Friday, September 20th, 2024, at 11:59PM, Anywhere on Earth.</b></p>
        <p class=""><u>Notification of Acceptance</u>: Wednesday, October 9th, 2024.</p>
        <p class=""><u>Workshop Date</u>: December 15th 2024, Vancouver, Canada.</p>
      </div>
    </div>
  </div>
</div>



<div id="invitedspeakers" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="col-md-6 col-md-offset-3 text-center wrap_title">
      <h2>Invited Speakers</h2>
    </div>
    <div class="row">
      <!-- /.col-md-6 -->
      <div class="col-sm-4 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/erika_debenedictis.jpg" alt="Erika Alden deBenedictis">
        <h3>Erika Alden DeBenedictis</h3>
        <p class="lead" style="margin-bottom:0;">Group Leader, Francis Crick Institute</p>
        <a onclick="showhide('erika_debenedictis')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="erika_debenedictis">
          <p>Erika is a former astronomer and current synthetic biologist. In collaboration with Schmidt Futures, she launched the 2022 Bioautomation Challenge, a grant program that improves the reproducibility, scalability, and shareability of life science research with programmable experiments. She now leads the Biodesign Laboratory at the Francis Crick Institute in London, UK.</p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

    <div class="col-sm-4 wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/gabe_rocklin.jpg" alt="Gabe Rocklin">
      <h3>Gabe Rocklin</h3>
      <p class="lead" style="margin-bottom:0;" >Assistant Professor, Department of Pharmacology, Northwestern University.</p>
      <a onclick="showhide('gabe_rocklin')">Show/Hide Bio</a>
      <div style="display: none; text-align: left; text-indent: 30px;" id="gabe_rocklin">
        <p> Gabe is an assistant professor in the Department of Pharmacology at Northwestern University. His lab seeks to develop efficient “design-test-analyze” cycles for protein therapeutics by combining large-scale de novo computational protein design with high-throughput methods such as mass spectrometry proteomics and next-generation sequencing. Gabe is a recipient of the NIH Director’s New Innovator Award. 
        </p>
      </div>
      <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
    </div>

      <div class="col-sm-4 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/jennifer_listgarten.jpg" alt="Jennifer Listgarten">
        <h3>Jennifer Listgarten</h3>
        <p class="lead" style="margin-bottom:0;">Professor in EECS, UC Berkeley</p>
        <a onclick="showhide('jennifer_listgarten')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="jennifer_listgarten">
          <p>Jennifer is a professor in UC Berkeley’s EECS Department, Center for Computational Biology, the Bioengineering programme, and a steering committee member of the Berkeley AI Research (BAIR) Lab. Jennifer completed her PhD at the University of Toronto then spent ten years at Microsoft Research. Her research focuses on both developing machine learning methods focused on variational inference, bayesian optimization, and graphical models, and their applications to protein engineering, genome wide association studies, and molecular design.
          </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>
    </div>
      <!-- /.col-lg-4 -->
    <div class="row">
      <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/milot_mirdita.jpg" alt="Milot Mirdota">
        <h3>Milot Mirdita</h3>
        <p class="lead" style="margin-bottom:0;">Postdoctoral Researcher, Seoul National University.</p>
        <a onclick="showhide('milot_mirdita')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="milot_mirdita">
          <p>Milot is a postdoctoral researcher in the lab of Martin Steinegger at the Seoul National University. His expertise includes bioinformatics, metagenomics, protein structure prediction and machine learning. Milot is also one of the core developers and maintainers of many open-source tools like Foldseek, MMseqs2 and ColabFold, used by researchers around the world.  </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

      <!-- /.col-lg-4 -->
      <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/noelia_ferruz.jpg" alt="Noelia Ferruz">
        <h3>Noelia Ferruz</h3>
        <p class="lead" style="margin-bottom:0;">Group Leader, Center of Genomic Regulation, Barcelona.</p>
        <a onclick="showhide('noelia_ferruz')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="noelia_ferruz">
          <p>Noelia is a tenure-track group leader at the Centre of Genomic Regulation in Barcelona. She is also a Beatriu Pinós fellow at the Universitat de Girona. Noelia received her PhD at the Universitat Pompeu Fabra in Barcelona, analyzing high-throughput molecular dynamics data with machine learning techniques. Her group is interested in combining unsupervised deep learning, web development, data analysis, large-scale molecular dynamics, and experimental validation to design de novo proteins, including artificial enzymes. 
          </p>
        </div>
      </div>

   </div>

    <div class="row">
      <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/josh_abramson.png" alt="AlphaFold3 Team">
        <h3>Josh Abramson, AlphaFold3 </h3>
        <p class="lead" style="margin-bottom:0;">Senior ML Researcher, Google DeepMind.</p>
        <a onclick="showhide('alphafold3_team')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="alphafold3_team">
          <p> Josh is a senior machine learning researcher at Google Deepmind working on AlphaFold and related topics. 
            He played a key role in delivering AlphaFold 3 and has continuing interest in applications to drug discovery and protein design. 
            Prior to AlphaFold, he worked on interactivity and memory for agents in simulated environments.
          </p>
          <p>Built by a team of research scientists from Google DeepMind and Isomorphic Labs, AlphaFold3 is capable of predicting the joint structure of complexes including proteins, nucleic acids, small molecules, ions and modified residues.</p>
          </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

      <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/jeremy_wohlwend.png" alt="Boltz-1 Team">
        <h3>Jeremy Wohlwend, Boltz-1</h3>
        <p class="lead" style="margin-bottom:0;">PhD Student, Massachusetts Institute of Technology.</p>
        <a onclick="showhide('jeremy_wohlwend')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="jeremy_wohlwend">
          <p> Jeremy is a PhD student with Regina Barzilay at Massachusetts Institute of Technology, working on developing machine learning methods for the healthcare domain.
            Jeremy is one of the main contributors to the Boltz-1, the first fully open-source model for biomolecular structure prediction, in collaboration with other researchers at MIT and Genesis Therapeutics.
          </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

    </div>
  <br>
</div>

<!-- Schedule -->
<div id ="schedule" class="content-section-b">
  <div class="container">
    <h2 class="section-heading">Schedule (Pacific Standard Time)</h2>
    <div class="row">
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>08:30</th>
            <td colspan="4" rowspan="1" class="stage-saturn">Opening Remarks</td>
          </tr>
          <tr>
            <th>08:35</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Noelia Ferruz<br/>
              <p style="font-weight:normal"><em>Title: TBA</em></p></td>
          </tr>
          <tr>
            <th>08:40</th>
          </tr>
          <tr>
            <th>08:45</th>
          </tr>
          <tr>
            <th>08:50</th>
          </tr>
          <tr>
            <th>08:55</th>
          </tr>
          <tr>
            <th>09:00</th>
              <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">The OMG dataset: An Open MetaGenomic corpus for mixed-modality genomic language modeling<br/>
                <em>Yunha Hwang · Andre Cornman · Jacob West-Roberts · Martin Beracochea · Sergey Ovchinnikov · Simon Roux · Antonio Camargo · Milot Mirdita</em></p></td>
          </tr>
          <tr>
            <th>09:05</th>
          </tr>
          <tr>
            <th>09:10</th>
          </tr>
          <tr>
            <th>09:15</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Josh Abramson, AlphaFold3 Team<br/>
            <p style="font-weight:normal"><em>Biomolecular Structure Prediction with AlphaFold3</em></p></td>
          </tr>
          <tr>
            <th>09:20</th>
          </tr>
          <tr>
            <th>09:25</th>
          </tr>
          <tr>
            <th>09:30</th>
          </tr>
          <tr>
            <th>09:35</th>
          </tr>
          <tr>
            <th>09:40</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Break</td>
          </tr>
          <tr>
            <th>09:45</th>
          </tr>
          <tr>
            <th>09:50</th>
          </tr>
          <tr>
            <th>09:55</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Erika Alden DeBenedictis
            <p style="font-weight:normal"><em> Title: TBA</em></p>
            </td>
          </tr>
          <tr>
            <th>10:00</th>
          </tr>
          <tr>
            <th>10:05</th>
          </tr>
          <tr>
            <th>10:10</th>
          </tr>
          <tr>
            <th>10:15</th>
          </tr>
          <tr>
            <th>10:20</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Controllable All-Atom Generation of Protein Sequence and Structure from Sequence-Only Inputs<br/>
                <em>Amy Lu · Wilson Yan · Kevin Yang · Vladimir Gligorijevic · Kyunghyun Cho · Richard Bonneau · Pieter Abbeel · Nathan Frey
                </em></p>
            </td>
          </tr>
          <tr>
            <th>10:25</th>
          </tr>
          <tr>
            <th>10:30</th>
          </tr>
          <tr>
            <th>10:35</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Protein Language Model Fitness is a Matter of Preference<br/>
                <em>Cade Gordon · Amy Lu · Pieter Abbeel </em></p>
            </td>
          </tr>
          <tr>
            <th>10:40</th>
          </tr>
          <tr>
            <th>10:45</th>
          </tr>
          <tr>
            <th>10:50</th>
            <td colspan="4" rowspan="4" class="stage-saturn">Invited Speaker - Gabe Rocklin
            <p style="font-weight:normal"><em> Title: TBA </em></p>
            </td>
          </tr>
          <tr>
            <th>10:55</th>
          </tr>
          <tr>
            <th>11:00</th>
          </tr>
          <tr>
            <th>11:05</th>
          </tr>
          <tr>
            <th>11:10</th>
          </tr>
          <tr>
            <th>11:15</th>
            <td colspan="4" rowspan="1" class="stage-earth">Word from Sponsors</td>
          </tr>
          <tr>
            <th>11:20</th>
            <td colspan="4" rowspan="12" class="stage-saturn">Poster Session/Lunch</td>
          </tr>
          <tr>
            <th>11:25</th>
          </tr>
          <tr>
            <th>11:30</th>
          </tr>
          <tr>
            <th>11:35</th>
          </tr>
          <tr>
            <th>11:40</th>
          </tr>
          <tr>
            <th>11:45</th>
          </tr>
          <tr>
            <th>11:50</th>
          </tr>
          <tr>
            <th>11:55</th>
          </tr>
          <tr>
            <th>12:00</th>
          </tr>
          <tr>
            <th>12:05</th>
          </tr>
          <tr>
            <th>12:10</th>
          </tr>
          <tr>
            <th>12:15</th>
          </tr>
          <tr>
            <th>12:20</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Jennifer Listgarten
            <p style="font-weight:normal"><em>Title: TBA</em></p>
            </td>
          </tr>
          <tr>
            <th>12:25</th>
          </tr>
          <tr>
            <th>12:30</th>
          </tr>
          <tr>
            <th>12:35</th>
          </tr>
          <tr>
            <th>12:40</th>
          </tr>
          <tr>
            <th>12:45</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Generative modeling of protein ensembles guided by crystallographic electron densities<br/>
                <em>Sai Advaith Maddipatla · Nadav Sellam · Sanketh Vedula · Ailie Marx · Alex Bronstein</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>12:50</th>
          </tr>
          <tr>
            <th>12:55</th>
          </tr>
        </table>
      </div>
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>01:00</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Improving Ab-Initio Cryo-EM Reconstruction with Semi-Amortized Pose Inference<br/>
                <em>Shayan Shekarforoush · David Lindell · Marcus Brubaker · David Fleet</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>01:05</th>
          </tr>
          <tr>
            <th>01:10</th>
          </tr>
          <tr>
            <th>01:15</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Break</td>
          </tr>
          <tr>
            <th>01:20</th>
          </tr>
          <tr>
            <th>01:25</th>
          </tr>
          <tr>
            <th>01:30</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Milot Mirdita
              <p style="font-weight:normal"><em>Title: TBA</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>01:35</th>
          </tr>
          <tr>
            <th>01:40</th>
          </tr>
          <tr>
            <th>01:45</th>
          </tr>
          <tr>
            <th>01:50</th>
          </tr>
          <tr>
            <th>01:55</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Jeremy Wohlwend, Boltz-1
              <p style="font-weight:normal"><em>Title: Democratizing Biomolecular Structure Prediction with Boltz-1</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>02:00</th>
          </tr>
          <tr>
            <th>02:05</th>
          </tr>
          <tr>
            <th>02:10</th>
          </tr>
          <tr>
            <th>02:15</th>
          </tr>
          <tr>
            <th>02:20</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">FlowPacker: protein side-chain packing with torsional flow matching<br/>
                <em>Jin Sub Lee · Philip Kim</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>02:25</th>
          </tr>
          <tr>
            <th>02:30</th>
          </tr>
          <tr>
            <th>02:35</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">HelixFlow, SE(3)–equivariant Full-atom Design of Peptides With Flow-matching Models<br/>
                <em>Xuezhi Xie · Pedro A Valiente · Jisun Kim · Jin Sub Lee · Philip Kim</em>
              </p>
            </td>
          </tr>
          <tr>
            <th>02:40</th>
          </tr>
          <tr>
            <th>02:45</th>
          </tr>
          <tr>
            <th>02:50</th>
            <td colspan="4" rowspan="2" class="stage-earth">Break</td>
          </tr>
          <tr>
            <th>02:55</th>
          </tr>
          <tr>
            <th>03:00</th>
            <td colspan="4" rowspan="1" class="stage-saturn">Word from PLINDER / PINDER competition</td>
          </tr>
          <tr>
            <th>03:05</th>
            <td colspan="4" rowspan="2" class="stage-earth">PLINDER / PINDER competition talks</td>
          </tr>
          <tr>
            <th>03:10</th>
          </tr>
          <tr>
            <th>03:15</th>
            <td colspan="4" rowspan="8" class="stage-saturn">Panel Session</td>
          </tr>
          <tr>
            <th>03:20</th>
          </tr>
          <tr>
            <th>03:25</th>
          </tr>
          <tr>
            <th>03:30</th>
          </tr>
          <tr>
            <th>03:35</th>
          </tr>
          <tr>
            <th>03:40</th>
          </tr>
          <tr>
            <th>03:45</th>
          </tr>
          <tr>
            <th>03:50</th>
          </tr>
          <tr>
            <th>03:55</th>
            <td colspan="4" rowspan="1" class="stage-earth">Closing Remarks</td>
          </tr>
          <tr>
            <th>04:00</th>
            <td colspan="4" rowspan="12" class="stage-saturn">Poster Session / Happy Hour</td>
          </tr>
          <tr>
            <th>04:05</th>
          </tr>
          <tr>
            <th>04:10</th>
          </tr>
          <tr>
            <th>04:15</th>
          </tr>
          <tr>
            <th>04:20</th>
          </tr>
          <tr>
            <th>04:25</th>
          </tr>
          <tr>
            <th>04:30</th>
          </tr>
          <tr>
            <th>04:35</th>
          </tr>
          <tr>
            <th>04:40</th>
          </tr>
          <tr>
            <th>04:45</th>
          </tr>
          <tr>
            <th>04:50</th>
          </tr>
          <tr>
            <th>04:55</th>
          </tr>
          <tr>
            <th>05:00</th>
          </tr>
        </table>
      </div>
    </div>
  </div>

  <div id="accepted" class="content-section-b" style="border-top: 0">
      <div class="container">
        <h2 class="section-heading">Accepted Papers</h2>
        <div class="row wow fadeInLeftBig"  data-animation-delay="200">
          <div class="col-md-12 wow fadeInRightBig"  data-animation-delay="200">
            <ul class="paperlist">
              <li>
                <p class="title">LatentDE: Latent-based Directed Evolution accelerated by Gradient Ascent for Protein Sequence Design</p>
                <p class="authors">Thanh Tran, Nhat Khang Ngo, Duy Nguyen, Truong Son Hy</p>
                <p class="preprints"><a href="/papers_2024/LatentDE:_Latent-based_Directed_Evolution_accelerated_by_Gradient_Ascent_for_Protein_Sequence_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Assessing interaction recovery of predicted protein-ligand poses</p>
                <p class="authors">Frederic Dreyer, David Errington, Cedric Bouysset, Constantin Schneider</p>
                <p class="preprints"><a href="/papers_2024/Assessing_interaction_recovery_of_predicted_protein-ligand_poses.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Improving Inverse Folding models at Protein Stability Prediction without additional Training or Data</p>
                <p class="authors">Oliver Dutton, Sandro Bottaro, Michele Invernizzi, Istvan Redl, Albert Chung, Falk Hoffmann, Louie Henderson, Stefano Ruschetta, Fabio Airoldi, Benjamin M J Owens, Patrik Foerch, Carlo Fisicaro, Kamil Tamiola</p>
                <p class="preprints"><a href="/papers_2024/Improving_Inverse_Folding_models_at_Protein_Stability_Prediction_without_additional_Training_or_Data.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Improving Antibody Design with Force-Guided Sampling in Diffusion Models</p>
                <p class="authors">Paulina Kulyte, Francisco Vargas, Simon Mathis, Yuguang Wang, Jose Miguel Hernandez-Lobato, Pietro Liò</p>
                <p class="preprints"><a href="/papers_2024/Improving_Antibody_Design_with_Force-Guided_Sampling_in_Diffusion_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Equivariant Blurring Diffusion for Multiscale Generation of Molecular Conformer</p>
                <p class="authors">Jiwoong Park, Yang Shen</p>
                <p class="preprints"><a href="/papers_2024/Equivariant_Blurring_Diffusion_for_Multiscale_Generation_of_Molecular_Conformer.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Active Learning for Affinity Prediction of Antibodies</p>
                <p class="authors">Alexandra Gessner, Sebastian Ober, Owen Vickery, Dino Oglic, Talip Ucar</p>
                <p class="preprints"><a href="/papers_2024/Active_Learning_for_Affinity_Prediction_of_Antibodies.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">IgBlend: Unifying 3D Structure and Sequence for Antibody LLMs</p>
                <p class="authors">Cedric Malherbe, Talip Ucar</p>
                <p class="preprints"><a href="/papers_2024/IgBlend:_Unifying_3D_Structure_and_Sequence_for_Antibody_LLMs.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Learning the Language of Protein Structures</p>
                <p class="authors">Jeremie DONA, Benoit Gaujac, Timothy Atkinson, Liviu Copoiu, Thomas Pierrot, Thomas Barrett</p>
                <p class="preprints"><a href="/papers_2024/Learning_the_Language_of_Protein_Structures.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">moPPIt: De Novo Generation of Motif-Specific Binders with Protein Language Models</p>
                <p class="authors">Tong Chen, Yinuo Zhang, Zachary Quinn, Pranam Chatterjee</p>
                <p class="preprints"><a href="/papers_2024/moPPIt:_De_Novo_Generation_of_Motif-Specific_Binders_with_Protein_Language_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Improving generalisability of 3D binding affinity models in low data regimes</p>
                <p class="authors">Julia Buhmann, Ward Haddadin, Alan Bilsland, Lukáš Pravda, Hagen Triendl</p>
                <p class="preprints"><a href="/papers_2024/Improving_generalisability_of_3D_binding_affinity_models_in_low_data_regimes.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Active Learning for Energy-Based Antibody Optimization and Enhanced Screening</p>
                <p class="authors">Kairi Furui, Masahito Ohue</p>
                <p class="preprints"><a href="/papers_2024/Active_Learning_for_Energy-Based_Antibody_Optimization_and_Enhanced_Screening.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Conditional Enzyme Generation Using Protein Language Models with Adapters</p>
                <p class="authors">Jason Yang, Aadyot Bhatnagar, Jeffrey Ruffolo, Ali Madani</p>
                <p class="preprints"><a href="/papers_2024/Conditional_Enzyme_Generation_Using_Protein_Language_Models_with_Adapters.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Improving Structural Plausibility in 3D Molecule Generation via Property-Conditioned Training with Distorted Molecules</p>
                <p class="authors">Lucy Vost, Vijil Chenthamarakshan, Payel Das, Charlotte Deane</p>
                <p class="preprints"><a href="/papers_2024/Improving_Structural_Plausibility_in_3D_Molecule_Generation_via_Property-Conditioned_Training_with_Distorted_Molecules.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Understanding Protein-DNA Interactions by Paying Attention to Protein and Genomics Foundation Models</p>
                <p class="authors">Dhruva Abhijit Rajwade, Erica Wang, Aryan Satpathy, Alexander Brace, Hongyu Guo, Arvind Ramanathan, Shengchao Liu, Animashree Anandkumar</p>
                <p class="preprints"><a href="/papers_2024/Understanding_Protein-DNA_Interactions_by_Paying_Attention_to_Protein_and_Genomics_Foundation_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">SPECTRE: A Spectral Transformer for Molecule Identification</p>
                <p class="authors">Wangdong Xu</p>
                <p class="preprints"><a href="/papers_2024/SPECTRE:_A_Spectral_Transformer_for_Molecule_Identification.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">FlowPacker: protein side-chain packing with torsional flow matching</p>
                <p class="authors">Jin Sub Lee, Philip Kim</p>
                <p class="preprints"><a href="/papers_2024/FlowPacker:_protein_side-chain_packing_with_torsional_flow_matching.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Similarity-Quantized Relative Difference Learning for Improved Molecular Activity Prediction</p>
                <p class="authors">Karina Zadorozhny, Kangway Chuang, Bharath Sathappan, Ewan Wallace, Vishnu Sresht, Colin Grambow</p>
                <p class="preprints"><a href="/papers_2024/Similarity-Quantized_Relative_Difference_Learning_for_Improved_Molecular_Activity_Prediction.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">HERMES: Holographic Equivariant neuRal network model for Mutational Effect and Stability prediction</p>
                <p class="authors">Gian Marco Visani, Michael Pun, William Galvin, Eric Daniel, Kevin Borisiak, Utheri Wagura, Armita Nourmohammad</p>
                <p class="preprints"><a href="/papers_2024/HERMES:_Holographic_Equivariant_neuRal_network_model_for_Mutational_Effect_and_Stability_prediction.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Adapting protein language models for structure-conditioned design</p>
                <p class="authors">Jeffrey Ruffolo, Aadyot Bhatnagar, Joel Beazer, Stephen Nayfach, Jordan Russ, Emily Hill, Riffat Hussain, Joseph Gallagher, Ali Madani</p>
                <p class="preprints"><a href="/papers_2024/Adapting_protein_language_models_for_structure-conditioned_design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Allo-Allo: Data-efficient prediction of allosteric sites</p>
                <p class="authors">Tianze Dong, Christopher Kan, Kapil Devkota, Rohit Singh</p>
                <p class="preprints"><a href="/papers_2024/Allo-Allo:_Data-efficient_prediction_of_allosteric_sites.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">CryoSPIN: Improving Ab-Initio Cryo-EM Reconstruction with Semi-Amortized Pose Inference</p>
                <p class="authors">Shayan Shekarforoush, David Lindell, Marcus Brubaker, David Fleet</p>
                <p class="preprints"><a href="/papers_2024/CryoSPIN:_Improving_Ab-Initio_Cryo-EM_Reconstruction_with_Semi-Amortized_Pose_Inference.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">GFlowNet Pretraining with Inexpensive Rewards</p>
                <p class="authors">Mohit Pandey, Gopeshh Subbaraj, Emmanuel Bengio</p>
                <p class="preprints"><a href="/papers_2024/GFlowNet_Pretraining_with_Inexpensive_Rewards.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Benchmarking text-integrated protein language model embeddings and embedding fusion on diverse downstream tasks</p>
                <p class="authors">Young Su Ko</p>
                <p class="preprints"><a href="/papers_2024/Benchmarking_text-integrated_protein_language_model_embeddings_and_embedding_fusion_on_diverse_downstream_tasks.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">RNAgrail: graph neural network and diffusion model for RNA 3D structure prediction</p>
                <p class="authors">Marek Justyna</p>
                <p class="preprints"><a href="/papers_2024/RNAgrail:_graph_neural_network_and_diffusion_model_for_RNA_3D_structure_prediction.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">The OMG dataset: An Open MetaGenomic corpus for mixed-modality genomic language modeling</p>
                <p class="authors">Yunha Hwang, Andre Cornman, Jacob West-Roberts, Martin Beracochea, Sergey Ovchinnikov, Simon  Roux, Antonio Camargo, Milot Mirdita</p>
                <p class="preprints"><a href="/papers_2024/The_OMG_dataset:_An_Open_MetaGenomic_corpus_for_mixed-modality_genomic_language_modeling.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Functional Alignment of Protein Language Models via Reinforcement Learning with Experimental Feedback</p>
                <p class="authors">Nathaniel Blalock, Srinath Seshadri, Philip Romero, Agrim Babbar, Sarah Fahlberg</p>
                <p class="preprints"><a href="/papers_2024/Functional_Alignment_of_Protein_Language_Models_via_Reinforcement_Learning_with_Experimental_Feedback.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Antibody Library Design by Seeding Linear Programming with Inverse Folding and Protein Language Models</p>
                <p class="authors">Conor Hayes, Andre Goncalves, Steven Magana-Zook, Ahmet Solak, Daniel Faissol, Mikel Landajuela</p>
                <p class="preprints"><a href="/papers_2024/Antibody_Library_Design_by_Seeding_Linear_Programming_with_Inverse_Folding_and_Protein_Language_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">EpiGraph: Recommender-Style Graph Neural Networks for Highly Accurate Prediction of Conformational B-Cell Epitopes</p>
                <p class="authors">Jung-Eun Shin, Yen-Lin Chen, Nathan Rollins, Thomas Hopf, Jordan Anderson, Michael Cianci, Daniela Cipolletta, Jyothsna Visweswaraiah, Yi Xing, Colin Lipper, Kevin Otipoby, Nathan Higginson-Scott, Ryan Peckner</p>
                <p class="preprints"><a href="/papers_2024/EpiGraph:_Recommender-Style_Graph_Neural_Networks_for_Highly_Accurate_Prediction_of_Conformational_B-Cell_Epitopes.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">MF-LAL: Drug Compound Generation Using Multi-Fidelity Latent Space Active Learning</p>
                <p class="authors">Peter Eckmann, Dongxia Wu, Germano Heinzelmann, Michael Gilson, Rose Yu</p>
                <p class="preprints"><a href="/papers_2024/MF-LAL:_Drug_Compound_Generation_Using_Multi-Fidelity_Latent_Space_Active_Learning.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Higher-Order Message Passing for Glycan Representation Learning</p>
                <p class="authors">Roman Joeres, Daniel Bojar</p>
                <p class="preprints"><a href="/papers_2024/Higher-Order_Message_Passing_for_Glycan_Representation_Learning.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">LOCAS: Multi-label mRNA Localization with Supervised Contrastive Learning</p>
                <p class="authors">Abrar  Abir, Md Toki Tahmid, M. Saifur Rahman</p>
                <p class="preprints"><a href="/papers_2024/LOCAS:_Multi-label_mRNA_Localization_with_Supervised_Contrastive_Learning.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Does Structural Information Improve ESM3 for Protein Binding Affinity Prediction?</p>
                <p class="authors">Thomas Loux, Dianzhuo Wang</p>
                <p class="preprints"><a href="/papers_2024/Does_Structural_Information_Improve_ESM3_for_Protein_Binding_Affinity_Prediction?.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Unified Sampling and Ranking for Protein Docking with DFMDock</p>
                <p class="authors">Lee-Shin Chu, Sudeep Sarma, Jeffrey Gray</p>
                <p class="preprints"><a href="/papers_2024/Unified_Sampling_and_Ranking_for_Protein_Docking_with_DFMDock.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Expanding Automated Multiconformer Ligand Modeling to Macrocycles and Fragments</p>
                <p class="authors">Jessica Flowers</p>
                <p class="preprints"><a href="/papers_2024/Expanding_Automated_Multiconformer_Ligand_Modeling_to_Macrocycles_and_Fragments.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Protein Sequence Domain Annotation using Language Models</p>
                <p class="authors">Arpan Sarkar, Kumaresh Krishnan, Sean Eddy</p>
                <p class="preprints"><a href="/papers_2024/Protein_Sequence_Domain_Annotation_using_Language_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">ProtComposer: Compositional Protein Structure Generation with 3D Ellipsoids</p>
                <p class="authors">Hannes Stärk, Bowen Jing, Tomas Geffner, Jason Yim, Tommi Jaakkola, Arash Vahdat, Karsten Kreis</p>
                <p class="preprints"><a href="/papers_2024/ProtComposer:_Compositional_Protein_Structure_Generation_with_3D_Ellipsoids.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Rapid protein structure assessment via a forward model for NMR spectra</p>
                <p class="authors">Benjamin Harding, Chad Rienstra, Hannah Wayment-Steele, Ziling Hu, Frank Delaglio, Rajat Garg, Katherine Henzler-Wildman, Timothy Grant</p>
                <p class="preprints"><a href="/papers_2024/Rapid_protein_structure_assessment_via_a_forward_model_for_NMR_spectra.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">PropEn: Optimizing Proteins with Implicit Guidance</p>
                <p class="authors">Natasa Tagasovska, Vladimir Gligorijevic, Kyunghyun Cho, Andreas Loukas</p>
                <p class="preprints"><a href="/papers_2024/PropEn:_Optimizing_Proteins_with_Implicit_Guidance.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Bayesian Optimisation for Protein Sequence Design: Gaussian Processes with Zero-Shot Protein Language Model Prior Mean</p>
                <p class="authors">Carolin Benjamins, Shikha Surana, Oliver Bent, Marius Lindauer, Paul Duckworth</p>
                <p class="preprints"><a href="/papers_2024/Bayesian_Optimisation_for_Protein_Sequence_Design:_Gaussian_Processes_with_Zero-Shot_Protein_Language_Model_Prior_Mean.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Bio-xLSTM: Generative modeling, representation and in-context learning of biological and chemical sequences</p>
                <p class="authors">Niklas Schmidinger, Lisa Schneckenreiter, Philipp Seidl, Johannes Schimunek, Sohvi Luukkonen, Pieter-Jan Hoedt, Johannes Brandstetter, Andreas Mayr, Sepp Hochreiter, Guenter Klambauer</p>
                <p class="preprints"><a href="/papers_2024/Bio-xLSTM:_Generative_modeling,_representation_and_in-context_learning_of_biological_and_chemical_sequences.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Retrieval Augmented Protein Language Models for Protein Structure Prediction</p>
                <p class="authors">Peter Lee, Xavier Cheng, Eric Xingyi, Le Song</p>
                <p class="preprints"><a href="/papers_2024/Retrieval_Augmented_Protein_Language_Models_for_Protein_Structure_Prediction.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">CompassDock: A Comprehensive Accurate Assessment Approach for Deep Learning-Based Molecular Docking in Inference and Fine-Tuning</p>
                <p class="authors">Ahmet Sarigun, Vedran Franke, Bora Uyar, Altuna Akalin</p>
                <p class="preprints"><a href="/papers_2024/CompassDock:_A_Comprehensive_Accurate_Assessment_Approach_for_Deep_Learning-Based_Molecular_Docking_in_Inference_and_Fine-Tuning.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">BoostMD: Accelerating Molecular Sampling using ML Force Field Feature</p>
                <p class="authors">Lars  Schaaf, Ilyes Batatia, Christoph Brunken, Thomas Barrett, Jules Tilly</p>
                <p class="preprints"><a href="/papers_2024/BoostMD:_Accelerating_Molecular_Sampling_using_ML_Force_Field_Feature.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">DockFormer: Efficient Multi-Modal Receptor-Ligand Interaction Prediction using Pair Transformer</p>
                <p class="authors">Ben Shor, Dina Schneidman</p>
                <p class="preprints"><a href="/papers_2024/DockFormer:_Efficient_Multi-Modal_Receptor-Ligand_Interaction_Prediction_using_Pair_Transformer.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Cryo-EM images are intrinsically low dimensional</p>
                <p class="authors">Luke Evans, Octavian-Vlad Murad, Lars Dingeldein, Pilar Cossio, Roberto Covino, Marina Meila</p>
                <p class="preprints"><a href="/papers_2024/Cryo-EM_images_are_intrinsically_low_dimensional.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Open-source Tools for CryoET Particle Picking Machine Learning Competitions</p>
                <p class="authors">Kyle  Harrington, Zhuowen Zhao, Jonathan Schwartz, Saugat Kandel, Utz Ermel, Mohammadreza Paraan, Clinton Potter, Bridget Carragher</p>
                <p class="preprints"><a href="/papers_2024/Open-source_Tools_for_CryoET_Particle_Picking_Machine_Learning_Competitions.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Protein Language Model Fitness is a Matter of Preference</p>
                <p class="authors">Cade Gordon, Amy Lu, Pieter Abbeel</p>
                <p class="preprints"><a href="/papers_2024/Protein_Language_Model_Fitness_is_a_Matter_of_Preference.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">AptaBLE: An Enhanced Deep Learning Platform for Aptamer Protein Interaction Prediction and Design</p>
                <p class="authors">Sawan Patel, Sherwood Yao, Keith Fraser, Adam Friedman, Zhangzhi Peng, Pranam Chatterjee, Owen Yao</p>
                <p class="preprints"><a href="/papers_2024/AptaBLE:_An_Enhanced_Deep_Learning_Platform_for_Aptamer_Protein_Interaction_Prediction_and_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Balancing Locality and Reconstruction in Protein Structure Tokenizer</p>
                <p class="authors">Jiayou Zhang, Barthélémy Meynard, Jing Gong, Xavier Cheng, Eric Xing, Le Song</p>
                <p class="preprints"><a href="/papers_2024/Balancing_Locality_and_Reconstruction_in_Protein_Structure_Tokenizer.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">What has AlphaFold3 learned about antibody and nanobody docking, and what remains unsolved?</p>
                <p class="authors">Fatima Hitawala, Jeffrey Gray</p>
                <p class="preprints"><a href="/papers_2024/What_has_AlphaFold3_learned_about_antibody_and_nanobody_docking,_and_what_remains_unsolved?.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">HelixFlow, SE(3)–equivariant Full-atom Design of Peptides With Flow-matching Models</p>
                <p class="authors">Xuezhi Xie, Pedro  A Valiente, Jisun Kim, Jin Sub Lee, Philip Kim</p>
                <p class="preprints"><a href="/papers_2024/HelixFlow,_SE(3)–equivariant_Full-atom_Design_of_Peptides_With_Flow-matching_Models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">MolMix: A Simple Yet Effective Baseline for Multimodal Molecular Representation Learning</p>
                <p class="authors">Andrei Manolache, Dragos-Constantin Tantaru, Mathias Niepert</p>
                <p class="preprints"><a href="/papers_2024/MolMix:_A_Simple_Yet_Effective_Baseline_for_Multimodal_Molecular_Representation_Learning.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Integrating Macromolecular X-ray Diffraction Data with Variational Inference</p>
                <p class="authors">luis aldama, Kevin Dalton, Doeke Hekstra</p>
                <p class="preprints"><a href="/papers_2024/Integrating_Macromolecular_X-ray_Diffraction_Data_with_Variational_Inference.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Fine-Tuning Discrete Diffusion Models via Reward Optimization: Applications to DNA and Protein Design</p>
                <p class="authors">Chenyu Wang, Masatoshi Uehara, Yichun He, Amy Wang, Tommaso Biancalani, Avantika Lal, Tommi Jaakkola, Sergey Levine, Hanchen Wang, Aviv Regev</p>
                <p class="preprints"><a href="/papers_2024/Fine-Tuning_Discrete_Diffusion_Models_via_Reward_Optimization:_Applications_to_DNA_and_Protein_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Low-N OpenFold fine-tuning improves peptide design without additional structures</p>
                <p class="authors">Theo Sternlieb, Jakub Otwinowski, Sam Sinai, Jeffrey Chan</p>
                <p class="preprints"><a href="/papers_2024/Low-N_OpenFold_fine-tuning_improves_peptide_design_without_additional_structures.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">SPRINT Enables Interpretable and Ultra-Fast Virtual Screening against Thousands of Proteomes</p>
                <p class="authors">Andrew McNutt, Abhinav Adduri, Caleb Ellington, Monica Dayao, Eric Xing, Hosein Mohimani, David Koes</p>
                <p class="preprints"><a href="/papers_2024/SPRINT_Enables_Interpretable_and_Ultra-Fast_Virtual_Screening_against_Thousands_of_Proteomes.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Ranking protein-peptide binding affinities with protein language models</p>
                <p class="authors">Charles Chalas, Michael Dunne, Michael Dunne, charles chalas</p>
                <p class="preprints"><a href="/papers_2024/Ranking_protein-peptide_binding_affinities_with_protein_language_models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Generating and scoring stable proteins using joint structure and sequence modeling</p>
                <p class="authors">Yehlin Cho, Justas Dauparas, Kotaro Tsuboyama, Gabriel  Rocklin, Sergey Ovchinnikov</p>
                <p class="preprints"><a href="/papers_2024/Generating_and_scoring_stable_proteins_using_joint_structure_and_sequence_modeling.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">FusOn-pLM: A Fusion Oncoprotein-Specific Language Model  via Adjusted Rate Masking</p>
                <p class="authors">Sophia Vincoff, Shrey Goel, Kseniia Kholina, Rishab Pulugurta, Pranay Vure, Pranam Chatterjee</p>
                <p class="preprints"><a href="/papers_2024/FusOn-pLM:_A_Fusion_Oncoprotein-Specific_Language_Model__via_Adjusted_Rate_Masking.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Systems-Structure-Based Drug Design</p>
                <p class="authors">Vincent Zaballa, Elliot Hui</p>
                <p class="preprints"><a href="/papers_2024/Systems-Structure-Based_Drug_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Learning the language of protein-protein interactions with ESM-Multimer</p>
                <p class="authors">Varun Ullanat, Bowen Jing, Samuel Sledzieski, Dr. Bonnie Berger</p>
                <p class="preprints"><a href="/papers_2024/Learning_the_language_of_protein-protein_interactions_with_ESM-Multimer.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Guided Multi-objective Generative AI for Structure-based Drug Design</p>
                <p class="authors">Amit  Kadan, Kevin Ryczko, Erika Lloyd, Adrian Roitberg, Takeshi Yamazaki</p>
                <p class="preprints"><a href="/papers_2024/Guided_Multi-objective_Generative_AI_for_Structure-based_Drug_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Tradeoffs of alignment-based and protein language models for predicting viral mutation effects</p>
                <p class="authors">Noor Youssef, Sarah Gurev, Navami Jain, Debora Marks</p>
                <p class="preprints"><a href="/papers_2024/Tradeoffs_of_alignment-based_and_protein_language_models_for_predicting_viral_mutation_effects.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">IgFlow: Flow Matching for De Novo Antibody Design</p>
                <p class="authors">Sanjay Nagaraj, Amir Shanehsazzadeh, Hyun Park, Jonathan King, Simon Levine</p>
                <p class="preprints"><a href="/papers_2024/IgFlow:_Flow_Matching_for_De_Novo_Antibody_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Generating and evaluating diverse sequences for protein backbones</p>
                <p class="authors">Yo Akiyama, Sergey Ovchinnikov</p>
                <p class="preprints"><a href="/papers_2024/Generating_and_evaluating_diverse_sequences_for_protein_backbones.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">SuperMetal: A Generative AI Framework for Rapid and Precise Metal Ion Location Prediction in Proteins</p>
                <p class="authors">Xiaobo Lin, Zhaoqian Su, Yunchao Liu, Jingxian Liu, Xiaohan Kuang, Peter  Cummings, Jesse Spencer-Smith, Jens Meiler</p>
                <p class="preprints"><a href="/papers_2024/SuperMetal:_A_Generative_AI_Framework_for_Rapid_and_Precise_Metal_Ion_Location_Prediction_in_Proteins.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Antibody DomainBed: Out-of-Distribution Generalization in Therapeutic Protein Design</p>
                <p class="authors">Natasa Tagasovska, Ji Won Park, Matthieu Kirchmeyer, Nathan Frey, Andrew Watkins, Aya Ismail, Arian Jamasb, Edith Lee, Tyler Bryson, Stephen Ra, Kyunghyun Cho</p>
                <p class="preprints"><a href="/papers_2024/Antibody_DomainBed:_Out-of-Distribution_Generalization_in_Therapeutic_Protein_Design.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Controllable All-Atom Generation of Protein Sequence and Structure from Sequence-Only Inputs</p>
                <p class="authors">Amy Lu, Wilson Yan, Kevin Yang, Vladimir Gligorijevic, Kyunghyun Cho, Richard Bonneau, Pieter Abbeel, Nathan Frey</p>
                <p class="preprints"><a href="/papers_2024/Controllable_All-Atom_Generation_of_Protein_Sequence_and_Structure_from_Sequence-Only_Inputs.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Loop-Diffusion: an equivariant diffusion model for designing and scoring protein loops</p>
                <p class="authors">Kevin Borisiak, Gian Marco Visani, Armita Nourmohammad</p>
                <p class="preprints"><a href="/papers_2024/Loop-Diffusion:_an_equivariant_diffusion_model_for_designing_and_scoring_protein_loops.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">ProteinZen: combining latent and SE(3) flow matching for all-atom protein generation</p>
                <p class="authors">Alex Li, Tanja Kortemme</p>
                <p class="preprints"><a href="/papers_2024/ProteinZen:_combining_latent_and_SE(3)_flow_matching_for_all-atom_protein_generation.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">TomoPicker: Annotation-Efficient Particle Picking in Cellular cryo-electron Tomograms</p>
                <p class="authors">Mostofa Rafid Uddin, Ajmain Yasar Ahmed, Md Toki Tahmid, Md Zarif Ul Alam, Min Xu</p>
                <p class="preprints"><a href="/papers_2024/TomoPicker:_Annotation-Efficient_Particle_Picking_in_Cellular_cryo-electron_Tomograms.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Exploring Categorical Flow Matching for 3D De Novo Molecule Generation</p>
                <p class="authors">Ian Dunn, David Koes</p>
                <p class="preprints"><a href="/papers_2024/Exploring_Categorical_Flow_Matching_for_3D_De_Novo_Molecule_Generation.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">SynFlowNet: Design of Diverse and Novel Molecules with Synthesis Constraints</p>
                <p class="authors">Miruna Cretu, Charles Harris, Ilia Igashov, Arne Schneuing, Marwin  Segler, Bruno Correia, Julien Roy, Emmanuel Bengio, Pietro Lió</p>
                <p class="preprints"><a href="/papers_2024/SynFlowNet:_Design_of_Diverse_and_Novel_Molecules_with_Synthesis_Constraints.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">RNA-DCGen: Dual Constrained RNA Sequence Generation with LLM-Attack</p>
                <p class="authors">Haz Sameen Shahgir, Md. Rownok Zahan Ratul, Md Toki Tahmid, Khondker Salman Sayeed, Atif  Rahman</p>
                <p class="preprints"><a href="/papers_2024/RNA-DCGen:_Dual_Constrained_RNA_Sequence_Generation_with_LLM-Attack.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">RNA-GPT: Multimodal Generative System for RNA Sequence Understanding</p>
                <p class="authors">YIJIA XIAO, Edward Sun, Yiqiao Jin, Wei Wang</p>
                <p class="preprints"><a href="/papers_2024/RNA-GPT:_Multimodal_Generative_System_for_RNA_Sequence_Understanding.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Capturing Protein Dynamics: Encoding Temporal and Spatial Dynamics from Molecular Dynamics Simulations</p>
                <p class="authors">Vignesh Bhethanabotla, Amin Tavakoli, Animashree Anandkumar, William Goddard</p>
                <p class="preprints"><a href="/papers_2024/Capturing_Protein_Dynamics:_Encoding_Temporal_and_Spatial_Dynamics_from_Molecular_Dynamics_Simulations.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">ProPicker: Promptable Segmentation for Particle Picking in Cryogenic Electron Tomography</p>
                <p class="authors">Simon Wiedemann, Zalan Fabian, Mahdi Soltanolkotabi, Reinhard Heckel</p>
                <p class="preprints"><a href="/papers_2024/ProPicker:_Promptable_Segmentation_for_Particle_Picking_in_Cryogenic_Electron_Tomography.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Estimating protein flexibility via uncertainty quantification of structure prediction models</p>
                <p class="authors">Charlotte Sweeney, Nele Quast, Fabian Spoendlin, Yee Whye Teh</p>
                <p class="preprints"><a href="/papers_2024/Estimating_protein_flexibility_via_uncertainty_quantification_of_structure_prediction_models.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Generative modeling of protein ensembles guided by crystallographic electron densities</p>
                <p class="authors">Sai Advaith Maddipatla, Nadav Bojan Sellam, Sanketh Vedula, Ailie Marx, Alex Bronstein</p>
                <p class="preprints"><a href="/papers_2024/Generative_modeling_of_protein_ensembles_guided_by_crystallographic_electron_densities.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Energy-Based Flow Matching for Molecular Docking</p>
                <p class="authors">Wenyin Zhou, Christopher Sprague, Hossein Azizpour )</p>
                <p class="preprints"><a href="/papers_2024/Energy-Based_Flow_Matching_for_Molecular_Docking.pdf">[paper]</a></p>
                </li>
                <li>
                <p class="title">Controlling multi-state conformational equilibria of dynamic proteins with Frame2seq</p>
                <p class="authors">Deniz Akpinaroglu, Dominic Grisingher, Stephanie Crilly, Tanja Kortemme</p>
                <p class="preprints"><a href="/papers_2024/Controlling_multi-state_conformational_equilibria_of_dynamic_proteins_with_Frame2seq.pdf">[paper]</a></p>
                </li>                
            </ul>
          </div>
        </div>
      </div>
    </div>
  

  </div>


<div id ="challenge" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Challenge 2024</h2>
        <p>This year we are running a challenge on the <a href="https://pinder.sh">Pinder</a> and <a href="https://plinder.sh">Plinder</a> datasets to evaluate how well the community is currently doing for protein-protein interaction prediction and protein-ligand complex prediction. </p>
        <h3 class="section-heading">Details of the challenge</h3>
        
        <p>
        
        
        <p>To submit your trained model you will need to make an inference docker image on HuggingFace Spaces using the following templates:
          <ul class="bulleted">
            <li><a href="https://huggingface.co/spaces/MLSB/pinder_inference_template">Pinder inference template</a></li>
            <li><a href="https://huggingface.co/spaces/MLSB/plinder_inference_template">Plinder inference template</a></li>
          </ul>

        </p>

        <h3 class="section-heading"> Rules for model training</h3>
        <p>
        <ul class="bulleted">
        <li>Participants MUST use the sequences and SMILES in the provided train and validation sets from PINDER or PLINDER. In order to ensure no leakage, external data augmentation is not allowed.</li>
          <li>If starting structures/conformations need to be generated for the model, then this can only be done from the training and validation sequences and SMILES. Note that this is only the case for train & validation - no external folding methods or starting structures are allowed for the test set under any circumstance!. Only the predicted structures/conformers themselves may be used in this way, the embeddings or models used to generate such predictions may not. E.g. it is not valid to “distill” a method that was not trained on PLINDER/PINDER</li>
          <li>The PINDER and PLINDER datasets should be used independently; combining the sets is considered augmentation and is not allowed.</li>

            <li>For inference, only the inputs provided in the evaluation sets may be used: canonical sequences, structures and MSAs; no alternate templates or sequences are permitted. The inputs that will be used by assessors for each challenge track is as follows: 
              <ul class="bulleted">
                <li>PLINDER: <code>(SMILES, monomer protein structure, monomer FASTA, monomer MSA)</code></li>
                <li>PINDER: <code>(monomer protein structure 1, monomer protein structure 2, FASTA 1, FASTA 2, MSA 1, MSA 2)</code></li>
              </ul>
            </li>
              <li>Model selection must be performed exclusively on the validation set designed for this purpose within the PINDER and PLINDER datasets.</li>
                <li>Methods relying on any model derivatives or embeddings trained on structures outside the PINDER/PLINDER training set are not permitted (e.g., ESM2, MSA: allowed; ESM3/ESMFold/SAProt/UniMol: not allowed).</li>
        </ul>
        </p>
        <p> Please find the technical documentation for how to use the datasets for the challenge:
          <ul class="bulleted">
            <li><a href="https://pinder-org.github.io/pinder/pinder-mlsb.html">PINDER</a></li>
            <li><a href="https://plinder-org.github.io/plinder/examples/mlsb_challenge.html">PLINDER</a></li>
            
          </ul>  
          </p>

        <h3 class="section-heading"> Rules for valid inference pipeline </h3>

        <p>Submission system will use Hugging Face Spaces. To qualify for submission, each team must:</p>
        <p>
          <ul class="bulleted">
            <li>Provide an MLSB submission ID or a link to a preprint/paper describing their methodology. This publication does not have to specifically report training or evaluation on the P(L)INDER dataset. Previously published methods, such as DiffDock, only need to link their existing paper. Note that entry into this competition does not equate to an MLSB workshop paper submission.</li>
<li>Create a copy of the provided inference template.

  Go to the top right corner of the page and click on the drop-down menu (vertical ellipsis) right next to the “Community”, then select “Duplicate this space”.
</li>
<li>Change files in the newly create space to reflect the peculiarities of your model 
<ul class="bulleted">
  <li>Edit <code>requirements.txt</code> to capture all dependencies.</li>
  <li>Include a <code>inference_app.py</code> file. This contains a <code>predict</code> function that should be modified to reflect the specifics of inference using their model.</li>
<li>Include a <code>train.py</code> file to ensure that training and model selection use only the PINDER/PLINDER datasets and to clearly show any additional hyperparameters used.</li>
<li>Provide a LICENSE file that allows for reuse, derivative works, and distribution of the provided software and weights (e.g., MIT or Apache2 license).</li>
<li>Modify the Dockerfile as appropriate (including selecting the right base image)</li>
</ul>

</li>
<li>Submit to the leaderboard via the <a href="https://huggingface.co/spaces/MLSB/leaderboard2024" rel="nofollow">designated form</a>.
  <ul dir="auto">
  <li>On submission page, add reference to the newly created space in the format username/space (e.g mlsb/alphafold3)</li>
  </ul>
  </li>





        <h3 class="section-heading">How to submit and view results</h3>
        <iframe
        src="https://mlsb-leaderboard2024.hf.space"
        frameborder="0"
        width="100%"
        height="700px"
      ></iframe>
        
        <h3 class="section-heading">Metrics</h3>
Primary Ranking Metric:
<ul class="bulleted">
<li>PLINDER: lDDT-PLI</li>
<li>PINDER: DockQ</li>
</ul>
<p>Other metrics computed by PINDER/PLINDER will be displayed on the leaderboard but will not influence the ranking.</p>
<p>The winners will be invited to present their work at the MLSB workshop.</p>

<h3 class="section-heading">Evaluation Datasets</h3>

<p>Although the exact composition of the eval set will be shared at a future date, below we provide an overview of the dataset and what to expect</p>

<p><ul class="bulleted">
  <li>Two leaderboards, one for each of PINDER and PLINDER, will be created using a single evaluation set for each.</li>
  <li>Evaluation sets will be subsets of 150-200 structures from the current PINDER and PLINDER test splits (subsets to enable reasonable eval runtime).</li>
  <li>Each evaluation sample will contain a predefined input/output to ensure performance assessment is model-dependent, not input-dependent.</li>
  <li>The focus will be exclusively on flexible docking/co-folding, with a single canonical structure per protein, sampled from apo and predicted structures.</li>
  <li>Monomer input structures will be sampled from paired structures available in PINDER/PLINDER, balanced between apo and predicted structures and stratified by "flexibility" level according to specified conformational difference thresholds.</li>
  </ul></p>
        
        <h3 class="section-heading">Key Dates</h3>


        <p class=""><u>Training workshop</u> September 24th, 2024, virtual (<a href="https://unibas.zoom.us/meeting/register/u5EkcOusrzsiHtLDImocB5PM0RLK9vC-g-kW#/registration">Register here</a>)</p>
<p><u>Leaderboard Opens:</u> October 9th, 2024 (following acceptance notifications for MLSB).</p>
<p><u>Leaderboard Closes:</u> <s>November 9th, 2024</s> November 18th, 2024</p>
<p><u>Winner Notification:</u> Wednesday, November 27th, 2024 </p>


<h3 class="section-heading">Questions?</h3>
<p>If you have trouble we invite you to <a href="https://discord.gg/NFVgccusBj">join the PINDER/PLINDER discord server</a></p>



<div class="img-thumbnail">

  <img alt="" width="150px"  loading="lazy" src="https://plinder.sh/icons/vantai_logo.svg" style="color: transparent; padding:20px">

  
  <img alt="" width="150px" class="p-3" src="https://plinder.sh/_next/image?url=%2Ficons%2Fnvidia.png&amp;w=3840&amp;q=75" style="padding:20px" >
  
  <img alt="" width="150px"   src="https://plinder.sh/icons/swiss_institute_of_bioinformatics.svg" style="color: transparent; padding:20px"><img alt="" width="150px"  src="https://plinder.sh/_next/image?url=%2Ficons%2Fbiozentrum.png&amp;w=3840&amp;q=75" style="color: transparent;padding:20px">
   
  

  <img alt="" class="img-fluid p-3" width="150px" loading="lazy" src="https://plinder.sh/_next/image?url=%2Ficons%2Funiversity_of_basel.png&amp;w=3840&amp;q=75" style="color: transparent; padding:20px">
    
  <img alt="" class="p-3" width="150px" src="https://plinder.sh/_next/image?url=%2Ficons%2Fmit_csail.png&amp;w=3840&amp;q=75" style="color: transparent;padding:20px"><img alt="" width="150px"  src="https://plinder.sh/_next/image?url=%2Ficons%2Fmit.png&amp;w=3840&amp;q=75" style="color: transparent;"><img  width="150px" class="p-3" src="https://plinder.sh/_next/image?url=%2Ficons%2Feuro_hpc.webp&amp;w=3840&amp;q=75" style="color: transparent; padding:20px"><img width="100px" src="https://plinder.sh/_next/image?url=%2Ficons%2Feuropean_commission.png&amp;w=3840&amp;q=75" style="color: transparent;">
  <img src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo-with-title.svg" alt="HuggingFace" width="200px" style="padding:20px">
     
</div>

      </div>
    </div>
  </div>
</div> 

<div id="organizers" class="content-section-a" style="border-top: 0">
  <div class="container">
    <h2 class="section-heading">Organizers</h2>
    <div class="row wow fadeInRightBig"  data-animation-delay="200">
      <div class="row">
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/gabriele_corso.png" alt="Photo of Gabriele Corso">
        </center>
        <h4 class="section-heading">
            <center>
              Gabriele Corso<br>
              <i><font size="3">MIT</font></i>
            </center>
          </h4>
      </div>
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/gina.jpg" alt="Photo of Gina El Nesr">
        </center>
        <h4 class="section-heading">
            <center>
              Gina El Nesr<br>
              <i><font size="3">Stanford University</font></i>
            </center>
          </h4>
      </div>
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/vignesh.png" alt="Photo of Vignesh Ram Somnath">
        </center>
        <h4 class="section-heading">
            <center>
              Vignesh Ram Somnath<br>
              <i><font size="3">ETH Zurich</font></i>
            </center>
          </h4>
      </div>
      </div>

      <div class="col-sm-6 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/zeming_lin.jpg" alt="Photo of Ellen Zhong">
        </center>
        <h4 class="section-heading">
          <center>
              Zeming Lin<br>
            <i><font size="3">EvolutionaryScale</font></i>
          </center>
        </h4>
      </div>
      
        <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/simon_duerr.png" alt="Photo of Simon Duerr">
          </center>
          <h4 class="section-heading">
            <center>
              Simon Duerr<br>
              <i><font size="3">EPFL</font></i>
            </center>
          </h4>
        </div>
      </div>

      <div class="row">
        
        <div class="col-sm-6 wow fadeInRightBig"  data-animation-delay="200">
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/hannah_wayment_steele.png" alt="Photo of Hannah Wayment-Steele">
          </center>
          <h4 class="section-heading">
            <center>
              Hannah Wayment-Steele<br>
              <i><font size="3">University of Wisconsin–Madison</font></i>
            </center>
          </h4>
        </div>

        <div class="row">
          <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
            <center>
              <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/sergey.jpg" alt="Photo of Sergey Ovchinnikov">
            </center>
            <h4 class="section-heading">
              <center>
                Sergey Ovchinnikov<br>
                <i><font size="3">MIT</font></i>
              </center>
            </h4>
          </div>

    </div>
      </div>
    </div>
  </div>

	<div id="related" class="content-section-b" style="border-top: 0">
        <div class="container" id="related">
		    <h2 class="section-heading">Past Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead">Here are links to previous MLSB Workshops at NeurIPS!</p>
                    <ul class="bulleted">
                        <li><a href="index_2023.html"><span class="network-name">2023 Workshop</span></a> </li>
                        <li><a href="index_2022.html"><span class="network-name">2022 Workshop</span></a> </li>
                        <li><a href="index_2021.html"><span class="network-name">2021 Workshop</span></a> </li>
                        <li><a href="index_2020.html"><span class="network-name">2020 Workshop</span></a> </li>
                    </ul>
				</div>
            </div>
        </div>
        <br/>
		<br/>
		</div>
	</div>

<div id="related" class="content-section-b" style="border-top: 0">
        <!-- <div class="container" id="related">
        <h2 class="section-heading">Related Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead"> Here are some <a href="https://neurips.cc/Conferences/2023/Schedule?type=Workshop">related workshops at NeurIPS</a> covering structure, biology, and physical science this year:
                    <ul class="bulleted">
                        <li><a href="https://genbio-workshop.github.io">Generative AI and Biology</a></li>
                        <li><a href="https://ai4d3.github.io">New Frontiers of AI for Drug Discovery and Development</a> </li>
                        <li><a href="https://ml4physicalsciences.github.io/2023/">Machine Learning and the Physical Science
                        </a></li>
            <li><a href="https://ai4sciencecommunity.github.io">AI for Science: from Theory to Practice</a></li>
            <li><a href="https://diffusionworkshop.github.io">Diffusion Models</a></li>
                    </ul>
        </div>
            </div>
        </div> -->
        <br/>
    <br/>
  <div id="sponsors" class="content-section-a" style="border-top: 0">
    <div class="container">
      <h2 class="section-heading">Sponsors</h2>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/sponsors/ATOMICAI_logo+name_Yellow_Red_black.png" alt="Atomic Logo"> </div>
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/sponsors/third-rock.jpeg" alt="TRV Logo"> </div>
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="100" src="./img/sponsors/evo_scale.svg" alt="EvolutionaryScale"> </div>
      </div>
      <br>
      <br>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="100" src="./img/sponsors/xaira_black.png" alt="Xiara"> </div>
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/sponsors/prescient.png" alt="Prescient"> </div>
        <div class="col-md-4"  data-animation-delay="200"> <img  class="center" style="" height="80" src="./img/sponsors/generate-bio-afpc-logo-cmyk-black.png" alt="Generate Bio"> </div>
    </div>
  </div>
</div>
<footer>
  <div class="container">
    <div class="row">
      <div class="col-md-12" id="contact">
        <h3 class="footer-title">Contact</h3>
        <p> Please contact <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a> with any questions. <br><br><br></p>
      </div>
      <!-- /col-xs-7 -->
    </div>
  </div>
</footer>

<!-- JavaScript -->
<script src="js/jquery-1.10.2.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/owl.carousel.js"></script>
<script src="js/script.js"></script>
<!-- StikyMenu -->
<script src="js/stickUp.min.js"></script>
<script type="text/javascript">
	  jQuery(function($) {
		$(document).ready( function() {
		  $('.navbar-default').stickUp();

		});
	  });

	</script>
<!-- Smoothscroll -->
<script type="text/javascript" src="js/jquery.corner.js"></script>
<script src="js/wow.min.js"></script>
<script>
	 new WOW().init();
	</script>
<script src="js/classie.js"></script>
<!-- Magnific Popup core JS file -->
<script src="js/jquery.magnific-popup.js"></script>
</body>
</html>
