<!doctype html>
<!-- FlatFy Theme - Andrea Galanti /-->
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->
<!--[if IE 9]>    <html class="no-js ie9" lang="en"> <![endif]-->
<!--[if gt IE 9]><!-->
<html>
<!--<![endif]-->
<script>
function showhide(id) {
  var x = document.getElementById(id);
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
<meta name="description" content="Workshop at the 37th Conference on Neural Information Processing Systems">
<meta name="author" content="">
<title>Machine Learning in Structural Biology</title>

<!-- Bootstrap core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">

<!-- Custom Google Web Font -->
<link href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
<link href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,100italic,300italic,400italic,700italic,900italic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=Arvo:400,700" rel="stylesheet" type="text/css">

<!-- Custom CSS-->
<link href="css/general.css" rel="stylesheet">

<!-- Owl-Carousel -->
<link href="css/custom.css" rel="stylesheet">
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/owl.theme.css" rel="stylesheet">
<link href="css/style.css" rel="stylesheet">
<link href="css/animate.css" rel="stylesheet">

<!-- Magnific Popup core CSS file -->
<link rel="stylesheet" href="css/magnific-popup.css">
<script src="js/modernizr-2.8.3.min.js"></script> <!-- Modernizr /-->
<!--[if IE 9]>
		<script src="js/PIE_IE9.js"></script>
	<![endif]-->
<!--[if lt IE 9]>
		<script src="js/PIE_IE678.js"></script>
	<![endif]-->

<!--[if lt IE 9]>
		<script src="js/html5shiv.js"></script>
	<![endif]-->

</head>

<body id="home">

<!-- Preloader -->
<div id="preloader">
  <div id="status"></div>
</div>

<!-- FullScreen -->
<div class="intro-header">
  <div class="col-xs-12 text-center abcen1">
    <br><br>
    <h1 class="h1_home wow fadeIn" data-wow-delay="0.4s">Machine Learning in Structural Biology</h1>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Workshop at the 38th Conference on Neural Information Processing Systems</h3>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">15th December 2024</h3>
    <ul class="list-inline intro-social-buttons">
      <!--<li><a href="https://neurips.cc/virtual/2022/workshop/50005" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">Livestream</span></a> </li>-->
      <!--
		<li><a href="https://eventhosts.gather.town/app/1UBzVZ8TnX03EOsa/mlsb" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">GatherTown</span></a>
		</li>
        -->
      <!--
				<li id="download" ><a href="" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">Gather</span></a>
				</li>
                -->
    </ul>
  </div>
  <!-- /.container -->
  <div class="col-xs-12 text-center abcen wow fadeIn">
    <div class="button_down "> <a class="imgcircle wow bounceInUp" data-wow-duration="1.5s"  href="#about"> <img class="img_scroll" src="img/icon/circle.png" alt="Go down to content"> </a> </div>
  </div>
</div>

<!-- NavBar-->
<nav class="navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
      <a class="navbar-brand" href="#home">MLSB</a> </div>
    <div class="collapse navbar-collapse navbar-right navbar-ex1-collapse">
      <ul class="nav navbar-nav">
        <li class="menuItem"><a href="#about">About</a></li>
        <li class="menuItem"><a href="#callforpapers">Call for Papers</a></li>
       
        <li class="menuItem"><a href="#presenterinfo">Presenter Information</a></li>
        <!-- <li class="menuItem"><a href="#faq">FAQ</a></li>-->
        <li class="menuItem"><a href="#invitedspeakers">Invited Speakers</a></li>
        <li class="menuItem"><a href="#challenge">Challenge</a></li>
        <!-- <li class="menuItem"><a href="#preregistration">Pre-Registration</a></li> -->
        <!-- <li class="menuItem"><a href="#schedule">Schedule</a></li> -->
        <!-- <li class="menuItem"><a href="#accepted">Accepted Papers</a></li> -->
        <li class="menuItem"><a href="#organizers">Organizers</a></li>
        <!-- <li class="menuItem"><a href="#related">Past Workshops</a></li> -->
        <li class="menuItem"><a href="#sponsors">Sponsors</a></li>
        <li class="menuItem"><a href="#contact">Contact</a></li>
      </ul>
    </div>
  </div>
</nav>

<!-- About -->
<div id="about" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">About</h2>
        <p class="lead"> Structural biology, the study of the 3D structure or shape of proteins and other biomolecules, has been transformed by breakthroughs from machine learning algorithms. Machine learning models are now routinely used by experimentalists to predict structures to aid in hypothesis generation and experimental design, accelerate the experimental process of structure determination (e.g. computer vision algorithms for cryo-electron microscopy), and have become a new industry standard for bioengineering new protein therapeutics (e.g. large language models for protein design). Despite all of this progress, there are still many active and open challenges for the field, such as modeling protein dynamics, predicting the structure of other classes of biomolecules such as RNA, learning and generalizing the underlying physics driving protein folding, and relating the structure of isolated proteins to the in vivo and contextual nature of their underlying function. These challenges are diverse and interdisciplinary, motivating new kinds of machine learning methods and requiring the development and maturation of standard benchmarks and datasets.  </p>

        <p class="lead">  Machine Learning in Structural Biology (MLSB),
          seeks to bring together field experts, practitioners, and students from across   academia, industry research groups, and pharmaceutical companies
          to focus on these new challenges and opportunities. This year, MLSB aims to bridge the theoretical and practical by addressing the outstanding computational and experimental problems at the forefront of our field. The intersection of artificial intelligence and structural biology promises to unlock new scientific discoveries and develop powerful design tools. </p>
        <p class="lead"> MLSB will be an in-person <a href="https://neurips.cc">NeurIPS</a> workshop on 15th December 2024 in MTG Rooms 11 & 12 at the Vancouver Convention Center.</p>
        <p class="lead"> Please contact the organizers at <a href = "mailto:workshopmlsb@gmail.com?">workshopmlsb@gmail.com</a> with any questions.</p>
        <p class="lead"> Stay updated on changes and workshop news by joining <a href="https://groups.google.com/g/workshopmlsb">our mailing list</a>. </p>
      </div>
    </div>
  </div>
</div>
<!--<div id ="faq" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">FAQ</h2>
        <p> <b>WIFI</b> <br/>
          network name: NeurIPS <br/>
          password: conference </p>
        <p> <b>SCHEDULE</b> MLSB will begin at 8:30am CST and end at 5:00pm CST on Saturday, Dec 3. The physical workshop will take place in Room 288 - 289 in the New Orleans Convention Center. The schedule of talks and poster sessions is available at our website, mlsb.io and on the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. </p>
        <p> <b>PANEL</b> -
          We will be hosting a panel discussion including our invited speakers on research directions in Machine Learning for Structural Biology from 2:50-3:50p CST. We are soliciting additional discussion topics and questions -- let us know what you would like to hear about by filling out this google form: <a href="https://forms.gle/fiJ6ZzsmepKL87tMA">https://forms.gle/fiJ6ZzsmepKL87tMA</a>. </p>
        <p> <b>POSTER SESSIONS</b> -
          We have scheduled two poster sessions at 11:15-12:15pm CST and 3:50-4:50pm CST. Posters can be set up before the conference begins, between 8:00-8:30am, during the break between 9:45-10:05am, or at the beginning of the first poster session.

          We ask that posters are 24"W x 36"H, as there may be space constraints in the room that limit the display of wider posters. However, if you already have a poster of a different size, it is acceptable to use that. </p>
        <p> <b>LUNCH + HAPPY HOUR</b> -
          MLSB will be providing boxed lunches to attendees during the workshop. In addition, there will be a happy hour during the afternoon poster session, where drinks will be served (non-alcoholic options will be available). </p>
        <p> <b>ORALS</b> -
          Each oral will have a 15 minute timeslot. We anticipate that talks should last approximately 10 minutes, allowing 5 minutes for questions and transition between speakers. If you are presenting in-person, you may check slides before the workshop (8:00-8:30am CST) or during the break (9:45-10:05am CST). </p>
        <p> Please check the schedule on our website, mlsb.io, for your presentation time. If you are not able to make this time, let us know as soon as possible. Additionally, below we have listed the format we currently have for each oral presentation (In-person, Live over Zoom, or Prerecorded). If this information is not correct, please let us know. You can contact us at workshopmlsb@gmail.com with any issues. </p>
        <p> <b>VIRTUAL PARTICIPATION</b> -
          MLSB will have some hybrid elements. There will be a livestream (a link is available at the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. Chat will be available on the NeurIPS workshop site for asking questions.

          Currently, there is no official venue for presenting posters virtually at MLSB. However, we are considering options to enable greater virtual participation over the next few days. If you have a poster at MLSB but plan to participate only virtually, please let us know by adding your name to this <a href="https://forms.gle/ntQDsULXaVkzVXYR8">google form</a> as soon as possible. </p>
      </div>
    </div>
  </div>
</div>-->

<div id ="presenterinfo" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Presenter Information</h2>
        <p>Congratulations to all accepted presenters! Please find some information on deadlines and expectations leading up to the MLSB Workshop!</p>
        <h3 class="section-heading">Posters</h3>
        <p>We ask all authors to prepare a poster that can be presented as part of our workshop. Posters must be 24W x 36H inches and will be taped to the wall. Poster boards will not be provided at the workshop. <b>We specifically ask for portrait layout because we will be tight on wall space.</b></p>
        <p> Additionally, a virtual copy of each poster must be uploaded to the NeurIPS poster upload <a href="https://neurips.cc/PosterUpload">portal</a>, by <b>Thursday, December 12</b>. Posters must be PNG with no more than 5120 width x 2880 height (no more than 10 MB). Thumbnail images should be 320 width x 256 height PNG and no more than 5 MB. 
          We know these are different dimensions than what we're asking for in-person posters, the poster upload dimensions are set by NeurIPS.
        </p><p>
          Users should log in using the neurips.cc account associated with their CMT email address. 
          If they did not already have a neurips.cc account, then it should have automatically been created and can be accessed by <a href="https://nips.cc/Profile/resetpassword">resetting the password</a>.
        </p>
        <h3 class="section-heading">Paper Camera-Ready</h3>
        <p>De-anonymized, camera-ready versions of the workshop paper will be due on <a href="https://cmt3.research.microsoft.com/MLSB2024">Microsoft CMT</a> by <b>Monday, Dec 2</b>. 
          Papers must indicate that they are NeurIPS MLSB workshop papers by using the modified NeurIPS style file <a href="https://raw.githubusercontent.com/mlsbio/mlsbio.github.io/master/neurips_mlsb_2024.sty">here.</a>
          Papers should be compiled with the `final` argument, e.g.
          <b>\usepackage[final]{neurips_mlsb_2024}</b></p>
        <p>We plan to make all camera-ready submitted papers available on the workshop website (<a href="https://www.mlsb.io/">https://www.mlsb.io/</a>). 
          If you would prefer that your work not be shared, then there is no need to submit a camera-ready version..</p>
        <h3 class="section-heading">Travel Award</h3>
        <p>This year we will try to cover as many workshop registrations as possible for student/academic attendees with oral presentations or posters who need financial assistance.
          If you would like to be considered, please fill out the following <a href="https://forms.gle/hK9AeWaMx8iiRhKp8">form</a> by <s><b>Friday, Nov 15</b></s> <b>Friday, Nov 8</b>.
          If you have any questions, please don't hesitate to contact us at <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a>.</p>
        <h3 class="section-heading">Key Dates</h3>
        <p class=""><u>Application for Registration Reimbursement</u>: <b>Friday, <s>November 15th, 2024</s> November 8th, 2024, at 11:59PM, Anywhere on Earth.</b></p>
        <p class=""><u>Camera-Ready PDF due on Microsoft CMT</u>: <b>Monday, December 2nd, 2024</b>.</p>
        <p class=""><u>Poster due</u>: <b>Thursday, December 12th, 2024</b>.</p>
      </div>
    </div>
  </div>
</div>



<!-- Call for Papers -->
<div id ="callforpapers" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Call For Papers</h2>
        <p>We welcome submissions of short papers leveraging machine learning to address problems in structural biology, including but not limited to:</p>
        <p>
        <ul class="bulleted">
          <li>Prediction of biomolecular structures, complexes, and interactions</li>
          <li>Design of or generative models for structure and/or sequence</li>
          <li>Methods for structure determination / biophysics (Cryo-EM/ET, NMR, crystallography, single-molecule methods, etc.)</li>
          <li>Geometric and symmetry-aware deep learning</li>
          <li>Conformational change, ensembles, and dynamics</li>
          <li>Integration of biomolecular physics</li>
          <li>Function and property prediction</li>
          <li>Structural bioinformatics and systems biology</li>
          <li>Therapeutic screening and design</li>
          <li>Language models and other implicit representations of protein structure</li>
          <li>Forward-looking position papers</li>
        </ul>
      </p>
        <p>We request anonymized PDF submissions by <strong>Friday, September 20, 2024, at 11:59PM, AoE (anywhere on earth)</strong> through our submission website on <a href="https://cmt3.research.microsoft.com/MLSB2024">CMT</a>. </p></p>
        <p>Papers should present novel work that has not been previously accepted at an archival venue at the time of submission.  Submissions should be a <em>maximum</em> of 5 pages (excluding references and appendices) in PDF format, using the <a href="https://media.neurips.cc/Conferences/NeurIPS2024/Styles.zip">NeurIPS style files</a>, and fully anonymized as per the requirements of NeurIPS. The NeurIPS checklist can be omitted from the submission. Submissions meeting these criteria will go through a light, double-blind review process. Reviewer comments will be returned to the authors as feedback.</p>
        <p>Accepted papers will be invited to present a poster at the workshop, with nominations of spotlight talks at the discretion of the organizers.</p>
        <p>New this year, we will have two special tracks for models for predicting protein-protein and protein-ligand interactions, evaluated on two new large-scale benchmarks, <a href="https://www.pinder.sh/">PINDER</a> and <a href="https://www.plinder.sh/">PLINDER</a>. The highest-performing open-source methods from these two tracks will receive invitations to a spotlight presentation. <strong>Stay tuned for more information on how to submit to these tracks.</strong>
        </p>
        <p>Like last year, authors that commit to open-sourcing code, model weights, and datasets used in the work will be given precedence for spotlight talks. This change only affects consideration for spotlights. Submissions that cannot make this commitment will still be considered for posters and will not be penalized for acceptance.</p>
        <p>This workshop is considered non-archival, however, authors of accepted contributions will have the option to make their work available through the workshop website. Presentation of work that is concurrently in submission is welcome. We welcome papers sharing encouraging work-in-progress results or forward-looking position papers that would benefit from feedback and community discussion at our workshop.</p>
        <h3 class="section-heading">Important Dates</h3>
        <p class=""><u>Submission Deadline</u>: <b>Friday, September 20th, 2024, at 11:59PM, Anywhere on Earth.</b></p>
        <p class=""><u>Notification of Acceptance</u>: Wednesday, October 9th, 2024.</p>
        <p class=""><u>Workshop Date</u>: December 15th 2024, Vancouver, Canada.</p>
      </div>
    </div>
  </div>
</div>



<div id="invitedspeakers" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="col-md-6 col-md-offset-3 text-center wrap_title">
      <h2>Invited Speakers</h2>
    </div>
    <div class="row">
      <!-- /.col-md-6 -->
      <div class="col-sm-4 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/erika_debenedictis.jpg" alt="Erika Alden deBenedictis">
        <h3>Erika Alden DeBenedictis</h3>
        <p class="lead" style="margin-bottom:0;">Group Leader, Francis Crick Institute</p>
        <a onclick="showhide('erika_debenedictis')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="erika_debenedictis">
          <p>Erika is a former astronomer and current synthetic biologist. In collaboration with Schmidt Futures, she launched the 2022 Bioautomation Challenge, a grant program that improves the reproducibility, scalability, and shareability of life science research with programmable experiments. She now leads the Biodesign Laboratory at the Francis Crick Institute in London, UK.</p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>
      <!-- /.col-md-6 -->
    <!-- row -->

    <div class="col-sm-4 wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/gabe_rocklin.jpg" alt="Gabe Rocklin">
      <h3>Gabe Rocklin</h3>
      <p class="lead" style="margin-bottom:0;" >Assistant Professor, Department of Pharmacology, Northwestern University.</p>
      <a onclick="showhide('gabe_rocklin')">Show/Hide Bio</a>
      <div style="display: none; text-align: left; text-indent: 30px;" id="gabe_rocklin">
        <p> Gabe is an assistant professor in the Department of Pharmacology at Northwestern University. His lab seeks to develop efficient “design-test-analyze” cycles for protein therapeutics by combining large-scale de novo computational protein design with high-throughput methods such as mass spectrometry proteomics and next-generation sequencing. Gabe is a recipient of the NIH Director’s New Innovator Award. 
        </p>
      </div>
      <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
    </div>

      <div class="col-sm-4 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/jennifer_listgarten.jpg" alt="Jennifer Listgarten">
        <h3>Jennifer Listgarten</h3>
        <p class="lead" style="margin-bottom:0;">Professor in EECS, UC Berkeley</p>
        <a onclick="showhide('jennifer_listgarten')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="jennifer_listgarten">
          <p>Jennifer is a professor in UC Berkeley’s EECS Department, Center for Computational Biology, the Bioengineering programme, and a steering committee member of the Berkeley AI Research (BAIR) Lab. Jennifer completed her PhD at the University of Toronto then spent ten years at Microsoft Research. Her research focuses on both developing machine learning methods focused on variational inference, bayesian optimization, and graphical models, and their applications to protein engineering, genome wide association studies, and molecular design.
          </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>
    </div>
      <!-- /.col-lg-4 -->
    <div class="row">
      <div class="col-sm-4  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/milot_mirdita.jpg" alt="Milot Mirdota">
        <h3>Milot Mirdita</h3>
        <p class="lead" style="margin-bottom:0;">Postdoctoral Researcher, Seoul National University.</p>
        <a onclick="showhide('milot_mirdita')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="milot_mirdita">
          <p>Milot is a postdoctoral researcher in the lab of Martin Steinegger at the Seoul National University. His expertise includes bioinformatics, metagenomics, protein structure prediction and machine learning. Milot is also one of the core developers and maintainers of many open-source tools like Foldseek, MMseqs2 and ColabFold, used by researchers around the world.  </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

      <!-- /.col-lg-4 -->
      <div class="col-sm-4  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/noelia_ferruz.jpg" alt="Noelia Ferruz">
        <h3>Noelia Ferruz</h3>
        <p class="lead" style="margin-bottom:0;">Group Leader, Center of Genomic Regulation, Barcelona.</p>
        <a onclick="showhide('noelia_ferruz')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="noelia_ferruz">
          <p>Noelia is a tenure-track group leader at the Centre of Genomic Regulation in Barcelona. She is also a Beatriu Pinós fellow at the Universitat de Girona. Noelia received her PhD at the Universitat Pompeu Fabra in Barcelona, analyzing high-throughput molecular dynamics data with machine learning techniques. Her group is interested in combining unsupervised deep learning, web development, data analysis, large-scale molecular dynamics, and experimental validation to design de novo proteins, including artificial enzymes. 
          </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>

      <div class="col-sm-4  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/alphafold3.jpg" alt="AlphaFold3 Team">
        <h3>AlphaFold3 Team</h3>
        <p class="lead" style="margin-bottom:0;">A model for predicting biomolecular interactions.</p>
        <a onclick="showhide('alphafold3_team')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="alphafold3_team">
          <p> The AlphaFold3 team, comprised of research scientists from Google DeepMind and Isomorphic Labs, have developed a model for predicting biomolecular interactions. Their tool, AlphaFold3, extends the capabilities of AlphaFold2 and AlphaFold-Multimer, and is capable of predicting the joint structure of complexes including proteins, nucleic acids, small molecules, ions and modified residues </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      </div>
      <!-- /.col-lg-4 -->

      <!-- <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/gevorg.jpg" alt="Gevorg Grigoryan">
        <h3>Gevorg Grigoryan</h3>
        <p class="lead" style="margin-bottom:0;">Co-Founder and CTO of Generate Biomedicines. <br> Associate Professor at Dartmouth College.</p>
        <a onclick="showhide('gevorg')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="gevorg">
          <p> Gevorg Grigoryan, PhD, is a co-founder and chief technical officer at Generate Biomedicines. He is also a research associate professor of computer science, biological sciences, and chemistry at Dartmouth College. He received two bachelor’s degrees, in computer science and biochemistry, and went on to obtain a PhD from the Massachusetts Institute of Technology, where he studied computational protein design and modeling of protein interactions. After completing his postdoctoral work at the University of Pennsylvania in 2011, Gevorg was appointed to the faculty at Dartmouth, where he obtained tenure in 2017.</p> <p>Prior to co-founding Generate Biomedicines, Gevorg’s research at Dartmouth focused on understanding the relationship between amino-acid sequence and structure, seeking rigorous ways of inferring principles governing protein folding, interaction, and function from data. The powerful ideas that were discovered suggested a fundamentally new and promising paradigm for the design of macro-molecules that was based on general principles inferred from data. Armed with this new and powerful understanding, Gevorg envisioned the possibility of a company that uses the breakthrough of data-driven protein design, powered by Machine Learning, to transform therapeutic discovery towards a faster, more precise, and less expensive endeavor. Collaborating with colleagues at Flagship Pioneering brought this vision into reality, leading to the founding of Generate Biomedicines. The most enjoyable part of Gevorg’s role as the founding Chief Technology Officer at Generate has been to work with incredibly talented scientist on some of the world’s most challenging problems in molecular and therapeutic science.</p><p> Gevorg has authored over 50 peer-reviewed papers, including in Nature, Science, PNAS, and other top journals and conferences, and his work has received awards and recognition from the Alfred P. Sloan Foundation, the National Institutes of Health, the National Science Foundation, and the American Cancer Society. </p>
        </div>
        <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> -->
      <!-- </div> -->
        <!-- <div class="col-sm-4  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/rfdiffusion.gif" class=gif alt="RFDiffusion">
          <h3>RF Diffusion Team</h3>
          <p class="lead" style="margin-bottom:0;">A diffusion model for protein design.</p>
          <a onclick="showhide('rfdiffusion')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="rfdiffusion">
          <p>The RFdiffusion team, comprised of computational biologists from UW Medicine, Columbia University, and MIT, have developed a diffusion model for protein design. Their tool, RFdiffusion, is able to generate novel proteins, addresses challenges related to <i>de novo</i> binder design, symmetric architecture, enzyme scaffolding, and more. Now open source, RFdiffusion is poised to accelerate scientific discovery and potentially transform medicine, vaccine development, and advanced materials development. </p>
        </div> -->
    <!-- /.row -->

  </div>
  <br>
</div>

<!-- Schedule -->
<!-- <div id ="schedule" class="content-section-b">
  <div class="container">
    <h2 class="section-heading">Schedule (CST)</h2>
    <div class="row">
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>08:30</th>
            <td colspan="4" rowspan="1" class="stage-saturn">Opening Remarks</td>
          </tr>
          <tr>
            <th>08:35</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Kyunghyun Cho<br/>
              <p style="font-weight:normal"><em>Health system scale language models for clinical and operational decision making</em></p></td>
          </tr>
          <tr>
            <th>08:40</th>
          </tr>
          <tr>
            <th>08:45</th>
          </tr>
          <tr>
            <th>08:50</th>
          </tr>
          <tr>
            <th>08:55</th>
          </tr>
          <tr>
            <th>09:00</th>
              <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Validation of de novo designed water-soluble and membrane proteins by in silico folding and melting<br/>
                <em>Alvaro Martin · Carolin Berner · Sergey Ovchinnikov · Anastassia Vorobieva </em></p></td>
          </tr>
          <tr>
            <th>09:05</th>
          </tr>
          <tr>
            <th>09:10</th>
          </tr>
          <tr>
            <th>09:15</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Tanja Kortemme<br/><p style="font-weight:normal"><em>Accurate and tunable de novo protein shapes for new functions</em></p></td>
          </tr>
          <tr>
            <th>09:20</th>
          </tr>
          <tr>
            <th>09:25</th>
          </tr>
          <tr>
            <th>09:30</th>
          </tr>
          <tr>
            <th>09:35</th>
          </tr>
          <tr>
            <th>09:40</th>
            <td colspan="4" rowspan="4" class="stage-saturn">Break</td>
          </tr>
          <tr>
            <th>09:45</th>
          </tr>
          <tr>
            <th>09:50</th>
          </tr>
          <tr>
            <th>09:55</th>
          </tr>
          <tr>
            <th>10:00</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Bridget Carragher<p style="font-weight:normal"><em> A CryoET Data Portal to Foster a Collaboration between the Machine Learning and CryoET Communities</em></p></td>
          </tr>
          <tr>
            <th>10:05</th>
          </tr>
          <tr>
            <th>10:10</th>
          </tr>
          <tr>
            <th>10:15</th>
          </tr>
          <tr>
            <th>10:20</th>
          </tr>
          <tr>
            <th>10:25</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">AlphaFold Meets Flow Matching for Generating Protein Ensembles<br/>
                <em>Bowen Jing · Bonnie Berger · Tommi Jaakkola </em></p></td>
          </tr>
          <tr>
            <th>10:30</th>
          </tr>
          <tr>
            <th>10:35</th>
          </tr>
          <tr>
            <th>10:40</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">DSMBind: an unsupervised generative modeling framework for binding energy prediction<br/>
                <em>Wengong Jin · Caroline Uhler · Nir Hacohen</em></p></td>
          </tr>
          <tr>
            <th>10:45</th>
          </tr>
          <tr>
            <th>10:50</th>
          </tr>
          <tr>
            <th>10:55</th>
            <td colspan="4" rowspan="4" class="stage-saturn">Invited Speaker - Polly Fordyce<p style="font-weight:normal"><em> Leveraging microfluidics for high-throughput and quantitative biochemistry and biophysics</em></p></td>
          </tr>
          <tr>
            <th>11:00</th>
          </tr>
          <tr>
            <th>11:05</th>
          </tr>
          <tr>
            <th>11:10</th>
          </tr>
          <tr>
            <th>11:15</th>
          </tr>
          <tr>
            <th>11:20</th>
            <td colspan="4" rowspan="16" class="stage-earth">Poster Session/Lunch</td>
          </tr>
          <tr>
            <th>11:25</th>
          </tr>
          <tr>
            <th>11:30</th>
          </tr>
          <tr>
            <th>11:35</th>
          </tr>
          <tr>
            <th>11:40</th>
          </tr>
          <tr>
            <th>11:45</th>
          </tr>
          <tr>
            <th>11:50</th>
          </tr>
          <tr>
            <th>11:55</th>
          </tr>
          <tr>
            <th>12:00</th>
          </tr>
          <tr>
            <th>12:05</th>
          </tr>
          <tr>
            <th>12:10</th>
          </tr>
          <tr>
            <th>12:15</th>
          </tr>
          <tr>
            <th>12:20</th>
          </tr>
          <tr>
            <th>12:25</th>
          </tr>
          <tr>
            <th>12:30</th>
          </tr>
          <tr>
            <th>12:35</th>
          </tr>
          <tr>
            <th>12:40</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Gevorg Grigoryan<p style="font-weight:normal"><em>Illuminating protein space with a programmable generative model</em></p></td>
          </tr>
          <tr>
            <th>12:45</th>
          </tr>
          <tr>
            <th>12:50</th>
          </tr>
          <tr>
            <th>12:55</th>
          </tr>
          <tr>
            <th>01:00</th>
          </tr>
        </table>
      </div>
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>01:05</th>
              <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Protein generation with evolutionary diffusion: sequence is all you need<br/>
                <em>Sarah Alamdari · Nitya Thakkar · Rianne van den Berg · Alex Lu · Nicolo Fusi · Ava P Amini  · Kevin Yang</em></p></td>
          </tr>
          <tr>
            <th>01:10</th>
          </tr>
          <tr>
            <th>01:15</th>
          </tr>
          <tr>
            <th>01:20</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Jason Yim / Brian Trippe <p style="font-weight:normal"><em>De novo design of protein structure and function with RFdiffusion</em></p></td>
          </tr>
          <tr>
            <th>01:25</th>
          </tr>
          <tr>
            <th>01:30</th>
          </tr>
          <tr>
            <th>01:35</th>
          </tr>
          <tr>
            <th>01:40</th>
          </tr>
          <tr>
            <th>01:45</th>
            <td colspan="4" rowspan="3" class="stage-earth">Break</td>
          </tr>
          <tr>
            <th>01:50</th>
          </tr>
          <tr>
            <th>01:55</th>
          </tr>
          <tr>
            <th>02:00</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">DiffDock-Pocket: Diffusion for Pocket-Level Docking with Sidechain Flexibility<br/>
                <em>Michael Plainer · Marcella Toth · Simon Dobers · Hannes Stärk · Gabriele Corso · Céline Marquet · Regina Barzilay</em></p></td>
          </tr>
          <tr>
            <th>02:05</th>
          </tr>
          <tr>
            <th>02:10</th>
          </tr>
          <tr>
            <th>02:15</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">PoseCheck: Generative Models for 3D Structure-based Drug Design Produce Unrealistic Poses<br/>
                <em>Charles Harris · Kieran Didi · Arian R. Jamasb · Chaitanya Joshi · Simon V Mathis · Pietro Lió · Tom Blundell </em></p></td>
          </tr>
          <tr>
            <th>02:20</th>
          </tr>
          <tr>
            <th>02:25</th>
          </tr>
          <tr>
            <th>02:30</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Rhiju Das <p style="font-weight:normal"><em>World-wide competitions and the RNA folding problem</em></p></td>
          </tr>
          <tr>
            <th>02:35</th>
          </tr>
          <tr>
            <th>02:40</th>
          </tr>
          <tr>
            <th>02:45</th>
          </tr>
          <tr>
            <th>02:50</th>
          </tr>
          <tr>
            <th>02:55</th>
            <td colspan="4" rowspan="1" class="stage-earth">Break</td>
          </tr>
          <tr>
            <th>03:00</th>
            <td colspan="4" rowspan="9" class="stage-saturn">Panel Discussion</td>
          </tr>
          <tr>
            <th>03:05</th>
          </tr>
          <tr>
            <th>03:10</th>
          </tr>
          <tr>
            <th>03:15</th>
          </tr>
          <tr>
            <th>03:20</th>
          </tr>
          <tr>
            <th>03:25</th>
          </tr>
          <tr>
            <th>03:30</th>
          </tr>
          <tr>
            <th>03:35</th>
          </tr>
          <tr>
            <th>03:40</th>
          </tr>
          <tr>
            <th>03:45</th>
			<td colspan="4" rowspan="15" class="stage-earth">Poster Session / Happy Hour</td>
          </tr>
          <tr>
            <th>03:50</th>
          </tr>
          <tr>
            <th>03:55</th>
          </tr>
          <tr>
            <th>04:00</th>
          </tr>
          <tr>
            <th>04:05</th>
          </tr>
          <tr>
            <th>04:10</th>
          </tr>
          <tr>
            <th>04:15</th>
          </tr>
          <tr>
            <th>04:20</th>
          </tr>
          <tr>
            <th>04:25</th>
          </tr>
          <tr>
            <th>04:30</th>
          </tr>
          <tr>
            <th>04:35</th>
          </tr>
          <tr>
            <th>04:40</th>
          </tr>
          <tr>
            <th>04:45</th>
          </tr>
          <tr>
            <th>04:50</th>
          </tr>
          <tr>
            <th>04:55</th>
          </tr>
          <tr>
            <th>05:00</th>
            <td colspan="4" rowspan="0" class="stage-saturn">Closing Remarks</td>
          </tr>
        </table>
      </div>
    </div>
  </div> -->

  <!-- <div id="accepted" class="content-section-b" style="border-top: 0">
      <div class="container">
        <h2 class="section-heading">Accepted Papers</h2>
        <div class="row wow fadeInLeftBig"  data-animation-delay="200">
          <div class="col-md-12 wow fadeInRightBig"  data-animation-delay="200">
            <ul class="paperlist">
              <li>
                <p class="title">ESMFold Hallucinates Native-Like Protein Sequences</p>
                <p class="authors">Jeliazko Jeliazkov, Diego del Alamo, Joel Karpiak</p>
                <p class="preprints"><a href="/papers_2023/ESMFold_Hallucinates_Native-Like_Protein_Sequences.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.05.23.541774v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Conditioned Protein Structure Prediction</p>
                <p class="authors">Tengyu Xie, Zilin Song, Jing Huang</p>
                <p class="preprints"><a href="/papers_2023/Conditioned_Protein_Structure_Prediction.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.09.24.559171v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Stable Online and Offline Reinforcement Learning for Antibody CDRH3 Design</p>
                <p class="authors">Yannick Vogt, Mehdi Naouar, Maria Kalweit, Christoph Cornelius Miething, Justus Duyster, Roland Mertelsmann, Gabriel Kalweit, Joschka Boedecker</p>
                <p class="preprints"><a href="/papers_2023/Stable_Online_and_Offline_Reinforcement_Learning_for_Antibody_CDRH3_Design.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Guiding diffusion models for antibody sequence and structure co-design with developability properties</p>
                <p class="authors">Amelia Villegas-Morcillo, Jana M Weber, Marcel JT Reinders</p>
                <p class="preprints"><a href="/papers_2023/Guiding_diffusion_models_for_antibody_sequence_and_structure_co-design_with_developability_properties.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.11.22.568230v2">[preprint]</a></p>
              </li>
              <li>
                <p class="title">AlphaFold Distillation for Protein Design</p>
                <p class="authors">Igor Melnyk, Aurelie Lozano, Payel Das, Vijil Chenthamarakshan</p>
                <p class="preprints"><a href="/papers_2023/AlphaFold_Distillation_for_Protein_Design.pdf">[paper]</a><a href="https://arxiv.org/abs/2210.03488">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Binding Oracle: Fine-Tuning From Stability to Binding Free Energy</p>
                <p class="authors">Chengyue Gong, Adam Klivans, Jordan Wells, James Loy, Qiang Liu, Alex Dimakis, Daniel  J Diaz</p>
                <p class="preprints"><a href="/papers_2023/Binding_Oracle_Fine-Tuning_From_Stability_to_Binding_Free_Energy.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Scalable Multimer Structure Prediction using Diffusion Models</p>
                <p class="authors">Peter Pao-Huang, Bowen Jing, Bonnie Berger</p>
                <p class="preprints"><a href="/papers_2023/Scalable_Multimer_Structure_Prediction_using_Diffusion_Models.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Implicit Geometry and Interaction Embeddings Improve Few-Shot Molecular Property Prediction</p>
                <p class="authors">Chris Fifty, Joseph M Paggi, Ehsan Amid, Jure Leskovec, Ron Dror</p>
                <p class="preprints"><a href="/papers_2023/Implicit_Geometry_and_Interaction_Embeddings_Improve_Few-Shot_Molecular_Property_Prediction.pdf">[paper]</a><a href="https://arxiv.org/abs/2302.02055">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Molecular Diffusion Models with Virtual Receptors</p>
                <p class="authors">Matan Halfon, Eyal Rozenberg, Ehud Rivlin, Daniel Freedman</p>
                <p class="preprints"><a href="/papers_2023/Molecular_Diffusion_Models_with_Virtual_Receptors.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">CESPED: a new benchmark for supervised particle pose estimation in Cryo-EM.</p>
                <p class="authors">Ruben Sanchez Garcia, Michael Saur, Javier Vargas, Carl Poelking, Charlotte Deane</p>
                <p class="preprints"><a href="/papers_2023/CESPED_a_new_benchmark_for_supervised_particle_pose_estimation_in_Cryo-EM.pdf">[paper]</a><a href="https://arxiv.org/abs/2311.06194">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Learning Scalar Fields for Molecular Docking with Fast Fourier Transforms</p>
                <p class="authors">Bowen Jing, Tommi Jaakkola, Bonnie Berger</p>
                <p class="preprints"><a href="/papers_2023/Learning_Scalar_Fields_for_Molecular_Docking_with_Fast_Fourier_Transforms.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">VN-EGNN: Equivariant Graph Neural Networks with Virtual Nodes Enhance Protein Binding Site Identification</p>
                <p class="authors">Florian Sestak, Lisa Schneckenreiter, Sepp Hochreiter, Andreas Mayr, Guenter Klambauer</p>
                <p class="preprints"><a href="/papers_2023/VN-EGNN_Equivariant_Graph_Neural_Networks_with_Virtual_Nodes_Enhance_Protein_Binding_Site_Identification.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Enhancing Ligand Pose Sampling for Machine Learning–Based Docking</p>
                <p class="authors">Patricia Suriana, Ron Dror</p>
                <p class="preprints"><a href="/papers_2023/Enhancing_Ligand_Pose_Sampling_for_Machine_Learning–Based_Docking.pdf">[paper]</a><a href="https://arxiv.org/abs/2312.00191">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Improved encoding of ensembles in PDBx/mmCIF</p>
                <p class="authors">Stephanie A Wankowicz, James Fraser</p>
                <p class="preprints"><a href="https://chemrxiv.org/engage/chemrxiv/article-details/655cd6bb2c3c11ed71d888a9">[preprint]</a></p>
              </li>
              <li>
                <p class="title">AlphaFold Meets Flow Matching for Generating Protein Ensembles</p>
                <p class="authors">Bowen Jing, Bonnie Berger, Tommi Jaakkola</p>
                <p class="preprints"><a href="/papers_2023/AlphaFold_Meets_Flow_Matching_for_Generating_Protein_Ensembles.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">The Discovery of Binding Modes Requires Rethinking Docking Generalization</p>
                <p class="authors">Gabriele Corso, Arthur Deng, Nicholas Polizzi, Regina Barzilay, Tommi Jaakkola</p>
                <p class="preprints"><a href="/papers_2023/The_Discovery_of_Binding_Modes_Requires_Rethinking_Docking_Generalization.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Conformational sampling and interpolation using language-based protein folding neural networks</p>
                <p class="authors">Diego del Alamo, Jeliazko Jeliazkov, Daphne Truan, Joel Karpiak</p>
                <p class="preprints"><a href="/papers_2023/Conformational_sampling_and_interpolation_using_language-based_protein_folding_neural_networks.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">FLIGHTED: Inferring Fitness Landscapes from Noisy High-Throughput Experimental Data</p>
                <p class="authors">Vikram Sundar, Boqiang Tu, Lindsey Guan, Kevin Esvelt</p>
                <p class="preprints"><a href="/papers_2023/FLIGHTED_Inferring_Fitness_Landscapes_from_Noisy_High-Throughput_Experimental_Data.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Contrasting Sequence with Structure: Pre-training Graph Representations with PLMs</p>
                <p class="authors">Louis Robinson, Timothy Atkinson, Liviu Copoiu, Patrick Bordes, Thomas Pierrot, Thomas D Barrett</p>
                <p class="preprints"><a href="/papers_2023/Contrasting_Sequence_with_Structure_Pre-training_Graph_Representations_with_PLMs.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Target-Aware Variational Auto-Encoders for Ligand Generation with Multi-Modal Protein Modeling</p>
                <p class="authors">Khang Nhat Ngo, Truong Son Hy</p>
                <p class="preprints"><a href="/papers_2023/Target-Aware_Variational_Auto-Encoders_for_Ligand_Generation_with_Multi-Modal_Protein_Modeling.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.08.10.552868v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">SE(3) denoising score matching for unsupervised binding energy prediction and nanobody design</p>
                <p class="authors">Wengong Jin, Xun Chen, Amrita Vetticaden, Siranush Sarzikova, Raktima Raychowdhury, Caroline Uhler, Nir Hacohen†</p>
                <p class="preprints"><a href="/papers_2023/SE3_denoising_score_matching_for_unsupervised_binding_energy_prediction_and_nanobody_design.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Fast non-autoregressive inverse folding with discrete diffusion</p>
                <p class="authors">John J Yang, Jason Yim, Regina Barzilay, Tommi Jaakkola</p>
                <p class="preprints"><a href="/papers_2023/Fast_non-autoregressive_inverse_folding_with_discrete_diffusion.pdf">[paper]</a><a href="https://arxiv.org/abs/2312.02447">[preprint]</a></p>
              </li>
              <li>
                <p class="title">TopoDiff: Improving Protein Backbone Generation with Topology-aware Latent Encoding</p>
                <p class="authors">Yuyang Zhang, Zihui Ma, Haipeng Gong</p>
                <p class="preprints"><a href="/papers_2023/TopoDiff_Improving_Protein_Backbone_Generation_with_Topology-aware_Latent_Encoding.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Harmonic Prior Self-conditioned Flow Matching for Multi-Ligand Docking and Binding Site Design</p>
                <p class="authors">Hannes Stärk, Bowen Jing, Regina Barzilay, Tommi Jaakkola</p>
                <p class="preprints"><a href="/papers_2023/Harmonic_Prior_Self-conditioned_Flow_Matching_for_Multi-Ligand_Docking_and_Binding_Site_Design.pdf">[paper]</a><a href="https://arxiv.org/abs/2310.05764">[preprint]</a></p>
              </li>
              <li>
                <p class="title">CrysFormer: Protein Crystallography Prediction via 3d Patterson Maps and Partial Structure Attention</p>
                <p class="authors">Chen Dun, Tom Pan, Shikai Jin, Ria Stevens, Mitchell D Miller, George Phillips, Anastasios Kyrillidis</p>
                <p class="preprints"><a href="/papers_2023/CrysFormer_Protein_Crystallography_Prediction_via_3d_Patterson_Maps_and_Partial_Structure_Attention.pdf">[paper]</a><a href="https://arxiv.org/abs/2310.03899">[preprint]</a></p>
              </li>
              <li>
                <p class="title">PoseCheck: Generative Models for 3D Structure-based Drug Design Produce Unrealistic Poses</p>
                <p class="authors">Charles Harris, Kieran Didi, Arian R Jamasb, Chaitanya K Joshi, Simon V Mathis, Pietro Lió, Tom L Blundell</p>
                <p class="preprints"><a href="/papers_2023/PoseCheck_Generative_Models_for_3D_Structure-based_Drug_Design_Produce_Unrealistic_Poses.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Sampling Protein Language Models for Functional Protein Design</p>
                <p class="authors">Jeremie Theddy Darmawan, Yarin Gal, Pascal Notin</p>
                <p class="preprints"><a href="/papers_2023/Sampling_Protein_Language_Models_for_Functional_Protein_Design.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">A framework for conditional diffusion modelling with applications in protein design</p>
                <p class="authors">Kieran Didi, Francisco Vargas, Simon V Mathis, Vincent Dutordoir, Emile Mathieu, Urszula Julia Komorowska, Pietro Lió</p>
                <p class="preprints"><a href="/papers_2023/A_framework_for_conditional_diffusion_modelling_with_applications_in_protein_design.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">DiffRNAFold: Generating RNA Tertiary Structures with Latent Space Diffusion</p>
                <p class="authors">Mihir N Bafna, Vikranth Keerthipati, Subhash Kanaparthi, Ruochi Zhang</p>
                <p class="preprints"><a href="/papers_2023/DiffRNAFold_Generating_RNA_Tertiary_Structures_with_Latent_Space_Diffusion.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Pair-EGRET: Enhancing the prediction of protein-protein interaction sites through graph attention networks and protein language models</p>
                <p class="authors">Ramisa Alam, Sazan Mahbub, Shamsuzzoha Bayzid</p>
                <p class="preprints"><a href="/papers_2023/Pair-EGRET_Enhancing_the_prediction_of_protein-protein_interaction_sites_through_graph_attention_networks_and_protein_language_models.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">FlexiDock: Compositional diffusion models for flexible molecular docking</p>
                <p class="authors">Zichen Wang, Balasubramaniam Srinivasan, Zhengyuan Shen, George Karypis, Huzefa Rangwala</p>
                <p class="preprints"><a href="/papers_2023/FlexiDock_Compositional_diffusion_models_for_flexible_molecular_docking.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">In vitro validated antibody design against multiple therapeutic antigens using generative inverse folding</p>
                <p class="authors">Amir Shanehsazzadeh, Julian Alverio, George Kasun, Simon Levine-Gottreich, Jibran A Khan, Chelsea Chung, Nicolas Diaz, Breanna K Luton, Ysis Tarter, Cailen McCloskey, Katherine B Bateman, Hayley Carter,
					Dalton Chapman, Rebecca Consbruck, Alec Jaeger, Christa Kohnert, Gaelin Kopec-Belliveau, John M Sutton, Zheyuan Guo, Gustavo Canales, Kai Ejan, Emily Marsh, Alyssa Ruelos, Rylee Ripley, Brooke Stoddard, Rodante Caguiat,
					Kyra Price, Matthew Saunders, Jared Sharp, Douglas Ganini da Silva, Audrey Feltner, Jake Ripley, Megan E Bryant, Danni Castillo, Joshua Meier, Christian M Stegmann, Katherine Moran, Christine Lemke, Shaheed Abdulhaqq, 
					Lillian R Klug, Sharrol Bachas</p>
                <p class="preprints"><a href="/papers_2023/In_vitro_validated_antibody_design_against_multiple_therapeutic_antigens_using_generative_inverse_folding.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Evaluating Zero-Shot Scoring for In Vitro Antibody Binding Prediction with Experimental Validation</p>
                <p class="authors">Divya V Nori, Simon V Mathis, Amir P Shanehsazzadeh</p>
                <p class="preprints"><a href="/papers_2023/Evaluating_Zero-Shot_Scoring_for_In_Vitro_Antibody_Binding_Prediction_with_Experimental_Validation.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">PDB-Struct: A Comprehensive Benchmark for Structure-based Protein Design</p>
                <p class="authors">Chuanrui Wang, Bozitao Zhong, Zuobai Zhang, Narendra Chaudhary, Sanchit Misra, Jian Tang</p>
                <p class="preprints"><a href="/papers_2023/PDB-Struct_A_Comprehensive_Benchmark_for_Structure-based_Protein_Design.pdf">[paper]</a><a href="https://arxiv.org/abs/2312.00080">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Optimizing protein language models with Sentence Transformers</p>
                <p class="authors">Istvan Redl, Fabio Airoldi, Sandro Bottaro, Albert Chung, Oliver  Dutton, Carlo  Fisicaro, Patrik Foerch, Louie Henderson, Falk Hoffmann, Michele Invernizzi, Benjamin M J Owens, Stefano Ruschetta, Kamil Tamiola</p>
                <p class="preprints"><a href="/papers_2023/Optimizing_protein_language_models_with_Sentence_Transformers.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">DiffDock-Pocket: Diffusion for Pocket-Level Docking with Sidechain Flexibility</p>
                <p class="authors">Michael Plainer, Marcella Toth, Simon Dobers, Hannes Stärk, Gabriele Corso, Céline Marquet, Regina Barzilay</p>
                <p class="preprints"><a href="/papers_2023/DiffDock-Pocket_Diffusion_for_Pocket-Level_Docking_with_Sidechain_Flexibility.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Transition Path Sampling with Boltzmann Generator-based MCMC Moves</p>
                <p class="authors">Michael Plainer, Hannes Stärk, Charlotte Bunne, Stephan Günnemann</p>
                <p class="preprints"><a href="/papers_2023/Transition_Path_Sampling_with_Boltzmann_Generator-based_MCMC_Moves.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Pre-training Sequence, Structure, and Surface Features for Comprehensive Protein Representation Learning</p>
                <p class="authors">Youhan Lee, Hasun Yu, Jaemyung Lee, Jaehoon Kim</p>
                <p class="preprints"><a href="/papers_2023/Pre-training_Sequence_Structure_and_Surface_Features_for_Comprehensive_Protein_Representation_Learning.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Inpainting Protein Sequence and Structure with ProtFill</p>
                <p class="authors">Elizaveta Kozlova, Arthur Valentin, Daniel Nakhaee-Zadeh Gutierrez</p>
                <p class="preprints"><a href="/papers_2023/Inpainting_Protein_Sequence_and_Structure_with_ProtFill.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Investigating Protein-DNA Binding Energetic of Mismatched DNA</p>
                <p class="authors">Ruben Solozabal Ochoa de Retana, Tamir Avioz, Yunxiang LI, Le Song, Martin Takac, Ariel Afek</p>
                <p class="preprints"></p>
              </li>
              <li>
                <p class="title">AntiFold: Improved antibody structure design using inverse folding</p>
                <p class="authors">Alissa M Hummer, Magnus H Høie, Tobias Olsen, Morten Nielsen, Charlotte Deane</p>
                <p class="preprints"><a href="/papers_2023/AntiFold_Improved_antibody_structure_design_using_inverse_folding.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Improved B-cell epitope prediction using AlphaFold2 modeling and inverse folding latent representations</p>
                <p class="authors">Paolo Marcatili</p>
                <p class="preprints"></p>
              </li>
              <li>
                <p class="title">Combining Structure and Sequence for Superior Fitness Prediction</p>
                <p class="authors">Steffanie Paul, Aaron Kollasch, Pascal Notin, Debora Marks</p>
                <p class="preprints"><a href="/papers_2023/Combining_Structure_and_Sequence_for_Superior_Fitness_Prediction.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Epitope-specific antibody design using diffusion models on the latent space of ESM embeddings</p>
                <p class="authors">Tomer Cohen, Dina Schneidman-Duhovny</p>
                <p class="preprints"><a href="/papers_2023/Epitope-specific_antibody_design_using_diffusion_models_on_the_latent_space_of_ESM_embeddings.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Protein language models learn evolutionary statistics of interacting sequence motifs</p>
                <p class="authors">Zhidian Zhang, Hannah K Wayment-Steele, Garyk Brixi, Matteo Dal Peraro, Dorothee Kern, Sergey Ovchinnikov</p>
                <p class="preprints"></p>
              </li>
              <li>
                <p class="title">Using artificial sequence coevolution to predict disulfide-rich peptide structures with experimental connectivity in AlphaFold</p>
                <p class="authors">Gabriella Gerlach, John M Nicoludis</p>
                <p class="preprints"><a href="/papers_2023/Using_artificial_sequence_coevolution_to_predict_disulfide-rich_peptide_structures_with_experimental_connectivity_in_AlphaFold.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Preferential Bayesian Optimisation for Protein Design with Ranking-Based Fitness Predictors</p>
                <p class="authors">Alex Hawkins-Hooker, Paul Duckworth, Oliver Bent</p>
                <p class="preprints"><a href="/papers_2023/Preferential_Bayesian_Optimisation_for_Protein_Design_with_Ranking-Based_Fitness_Predictors.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">FAFormer: Frame Averaging Transformer for Predicting Nucleic Acid-Protein Interactions</p>
                <p class="authors">Tinglin Huang, Zhenqiao Song, Rex Ying, Wengong Jin</p>
                <p class="preprints"><a href="/papers_2023/FAFormer_Frame_Averaging_Transformer_for_Predicting_Nucleic_Acid-Protein_Interactions.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">LightMHC: A Light Model for pMHC Structure Prediction with Graph Neural Networks</p>
                <p class="authors">Antoine P Delaunay, Yunguan Fu, Nikolai Gorbushin, Robert McHardy, Bachir Djermani, Liviu Copoiu, Michael Rooney, Maren Lang, Andrey Tovchigrechko, Ugur Sahin, Karim Beguir, Nicolas Lopez Carranza</p>
                <p class="preprints"><a href="/papers_2023/LightMHC_A_Light_Model_for_pMHC_Structure_Prediction_with_Graph_Neural_Networks.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.11.21.568015v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">FrameDiPT: SE(3) Diffusion Model for Protein Structure Inpainting</p>
                <p class="authors">Cheng Zhang, Adam Leach, Thomas Makkink, Miguel Arbesu, Ibtissem Kadri, Daniel Luo, Liron Mizrahi, Sabrine Krichen, Maren Lang, Andrey Tovchigrechko, Nicolas Lopez Carranza, Ugur Sahin, Karim Beguir, Michael Rooney, Yunguan Fu</p>
                <p class="preprints"><a href="/papers_2023/FrameDiPT_SE3_Diffusion_Model_for_Protein_Structure_Inpainting.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.11.21.568057v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">An Active Learning Framework for ML-Assisted Labeling of Cryo-EM Micrographs</p>
                <p class="authors">Robert Kiewisz, Tristan Bepler</p>
                <p class="preprints"><a href="/papers_2023/An_Active_Learning_Framework_for_ML-Assisted_Labeling_of_Cryo-EM_Micrographs.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Validation of de novo designed water-soluble and membrane proteins by in silico folding and melting.</p>
                <p class="authors">Alvaro Martin, Carolin Berner, Sergey Ovchinnikov, Anastassia Vorobieva</p>
                <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2023.06.06.543955v2">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Structure, Surface and Interface Informed Protein Language Model</p>
                <p class="authors">Ioan Ieremie, Mahesan Niranjan, Rob M Ewing</p>
                <p class="preprints"><a href="/papers_2023/Structure_Surface_and_Interface_Informed_Protein_Language_Model.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">De Novo Short Linear Motif (SLiM) Discovery With AlphaFold-Multimer</p>
                <p class="authors">Theo Sternlieb, Abhishaike Mahajan, Davian Ho, Jeffrey Chan</p>
                <p class="preprints"><a href="/papers_2023/De_Novo_Short_Linear_Motif_SLiM_Discovery_With_AlphaFold-Multimer.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">AF2BIND: Predicting ligand-binding sites using the pair representation of AlphaFold2</p>
                <p class="authors">Artem Gazizov, Anna Lian, Casper Goverde, Sergey Ovchinnikov, Nicholas Polizzi</p>
                <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2023.10.15.562410v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Protein generation with evolutionary diffusion: sequence is all you need</p>
                <p class="authors">Sarah Alamdari, Nitya Thakkar, Rianne van den Berg, Alex Lu, Nicolo Fusi, Ava P Amini, Kevin Yang</p>
                <p class="preprints"><a href="/papers_2023/Protein_generation_with_evolutionary_diffusion_sequence_is_all_you_need.pdf">[paper]</a><a href="https://www.biorxiv.org/content/10.1101/2023.09.11.556673v1">[preprint]</a></p>
              </li>
              <li>
                <p class="title">LatentDock: Protein-Protein Docking with Latent Diffusion</p>
                <p class="authors">Matt McPartlon, Céline Marquet, Tomas Geffner, Danny Kovtum, Alexander Goncearenco, Zachary Carpenter, Luca Naef, Michael Bronstein, Jinbo Xu</p>
                <p class="preprints"><a href="/papers_2023/LatentDock_Protein-Protein_Docking_with_Latent_Diffusion.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">HiFi-NN annotates the microbial dark matter with Enzyme Commission numbers</p>
                <p class="authors">Gavin Ayres, Geraldene Munsamy, Michael Heinzinger, Noelia Ferruz, Kevin Yang, Philipp Lorenz</p>
                <p class="preprints"><a href="/papers_2023/HiFi-NN_annotates_the_microbial_dark_matter_with_Enzyme_Commission_numbers.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Towards Joint Sequence-Structure Generation of Nucleic Acid and Protein Complexes with SE(3)-Discrete Diffusion</p>
                <p class="authors">Alex Morehead, Jeffrey A Ruffolo, Aadyot Bhatnagar, Ali Madani</p>
                <p class="preprints"><a href="/papers_2023/Towards_Joint_Sequence-Structure_Generation_of_Nucleic_Acid_and_Protein_Complexes_with_SE3-Discrete_Diffusion.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">SO(3)-Equivariant Representation Learning in 2D Images</p>
                <p class="authors">Darnell Granberry, Alireza Nasiri, Jiayi Shou, Alex J Noble, Tristan Bepler</p>
                <p class="preprints"><a href="/papers_2023/SO3-Equivariant_Representation_Learning_in_2D_Images.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">HelixDiff: Conditional Full-atom Design of Peptides With Diffusion Models</p>
                <p class="authors">Xuezhi Xie, Pedro A Valiente, Jisun Kim, Philip Kim</p>
                <p class="preprints"><a href="/papers_2023/HelixDiff_Conditional_Full-atom_Design_of_Peptides_With_Diffusion_Models.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">DiffMaSIF: Surface-based Protein-Protein Docking with Diffusion Models</p>
                <p class="authors">Freyr Sverrisson, Mehmet Akdel, Dylan Abramson, Jean Feydy, Alexander Goncearenco, Yusuf Adeshina, Daniel Kovtun, Céline Marquet, Xuejin Zhang, David Baugher, Zachary Wayne Carpenter, Luca Naef, Michael Bronstein, Bruno Correia</p>
                <p class="preprints"><a href="/papers_2023/DiffMaSIF_Surface-based_Protein-Protein_Docking_with_Diffusion_Models.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">FLAb: Benchmarking deep learning methods for antibody fitness prediction</p>
                <p class="authors">Michael F Chungyoun, Jeffrey A Ruffolo, Jeffrey Gray</p>
                <p class="preprints"><a href="/papers_2023/FLAb_Benchmarking_deep_learning_methods_for_antibody_fitness_prediction.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Parameter-Efficient Fine-Tuning of Protein Language Models Improves Prediction of Protein-Protein Interactions</p>
                <p class="authors">Samuel Sledzieski, Meghana Kshirsagar, Bonnie Berger, Rahul Dodhia, Juan M Lavista Ferres</p>
                <p class="preprints"><a href="/papers_2023/Parameter-Efficient_Fine-Tuning_of_Protein_Language_Models_Improves_Prediction_of_Protein-Protein_Interactions.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">TriFold: A New Architecture for Predicting Protein Sequences from Structural Data</p>
                <p class="authors">Harish K Srinivasan, Jian Zhou</p>
                <p class="preprints"><a href="/papers_2023/TriFold_A_New_Architecture_for_Predicting_Protein_Sequences_from_Structural_Data.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">End-to-End Sidechain Modeling in AlphaFold2: Attention May or May Not Be All That You Need</p>
                <p class="authors">Jonathan King, David Koes</p>
                <p class="preprints"><a href="/papers_2023/End-to-End_Sidechain_Modeling_in_AlphaFold2_Attention_May_or_May_Not_Be_All_That_You_Need.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Coarse-graining via reparametrization avoids force-matching and back-mapping</p>
                <p class="authors">Nima Dehmamy, Csaba Both, Subhro Das, Tommi Jaakkola</p>
                <p class="preprints"><a href="/papers_2023/Coarse-graining_via_reparametrization_avoids_force-matching_and_back-mapping.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">SE3Lig: SE(3)-equivariant CNNs for the reconstruction of cofactors and ligands in protein structures</p>
                <p class="authors">Siddharth Bhadra-Lobo, Anushriya Subedy, Sagar Khare, Guillaume Lamoureux</p>
                <p class="preprints"><a href="/papers_2023/SE3Lig_SE3-equivariant_CNNs_for_the_reconstruction_of_cofactors_and_ligands_in_protein_structures.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Cramming Protein Language Model Training in 24 GPU Hours</p>
                <p class="authors">Nathan C Frey, Taylor Joren, Aya A Ismail, Allen Goodman, Stephen Ra, Richard Bonneau, Kyunghyun Cho, Vladimir Gligorijevic</p>
                <p class="preprints"><a href="/papers_2023/Cramming_Protein_Language_Model_Training_in_24_GPU_Hours.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Preparation Of Labeled Cryo-ET Datasets For Training And Evaluation Of Machine Learning Models</p>
                <p class="authors">Aygul Ishemgulova, Alex Noble, Tristan Bepler, Alex De Marco</p>
                <p class="preprints"><a href="/papers_2023/Preparation_Of_Labeled_Cryo-ET_Datasets_For_Training_And_Evaluation_Of_Machine_Learning_Models.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">EMPOT: partial alignment of density maps and rigid body fitting using unbalanced Gromov-Wasserstein divergence</p>
                <p class="authors">Aryan Tajmir Riahi, Chenwei Zhang, James Chen, Anne Condon, Khanh Dao Duc</p>
                <p class="preprints"><a href="/papers_2023/EMPOT_partial_alignment_of_density_maps_and_rigid_body_fitting_using_unbalanced_Gromov-Wasserstein_divergence.pdf">[paper]</a><a href="https://arxiv.org/abs/2311.00850">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Fast protein backbone generation with SE(3) flow matching</p>
                <p class="authors">Jason Yim, Andrew Campbell, Andrew Foong, Sarah Lewis, Victor Satorras, Michael Gastegger, Bas Veeling, Jose Jimenez-Luna, Regina Barzilay, Tommi Jaakkola, Frank Noe</p>
                <p class="preprints"><a href="/papers_2023/Fast_protein_backbone_generation_with_SE3_flow_matching.pdf">[paper]</a><a href="https://arxiv.org/abs/2310.05297">[preprint]</a></p>
              </li>
              <li>
                <p class="title">Frame2seq: structure-conditioned masked language modeling for protein sequence design</p>
                <p class="authors">Deniz Akpinaroglu, Kosuke Seki, Eleanor Zhu, Tanja Kortemme</p>
                <p class="preprints"><a href="/papers_2023/Frame2seq_structure-conditioned_masked_language_modeling_for_protein_sequence_design.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Structure-Conditioned Generative Models for De Novo Ligand Generation: A Pharmacophore Assessment</p>
                <p class="authors">Shannon Smith, Leo Gendelev, Kangway V Chuang, Seth Harris</p>
                <p class="preprints"><a href="/papers_2023/Structure-Conditioned_Generative_Models_for_De_Novo_Ligand_Generation_A_Pharmacophore_Assessment.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Jointly Embedding Protein Structures and Sequences through Residue Level Alignment</p>
                <p class="authors">Foster Birnbaum, Saachi Jain, Aleksander Madry, Amy Keating</p>
                <p class="preprints"><a href="/papers_2023/Jointly_Embedding_Protein_Structures_and_Sequences_through_Residue_Level_Alignment.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Evaluating Representation Learning on the Protein Structure Universe</p>
                <p class="authors">Arian R Jamasb, Alex Morehead, Chaitanya K Joshi, Zuobai Zhang, Kieran Didi, Simon V Mathis, Charles Harris, Jian Tang, Mila), Jianlin  Cheng, Pietro Lió, Tom Blundell</p>
                <p class="preprints"><a href="/papers_2023/Evaluating_Representation_Learning_on_the_Protein_Structure_Universe.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Enhancing Antibody Language Models with Structural Information</p>
                <p class="authors">Justin Barton, Jacob Galson, Jinwoo Leem</p>
                <p class="preprints"><a href="/papers_2023/Enhancing_Antibody_Language_Models_with_Structural_Information.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Amortized Pose Estimation for X-Ray Single Particle Imaging</p>
                <p class="authors">Jay Shenoy, Axel Levy, Frederic Poitevin, Gordon Wetzstein</p>
                <p class="preprints"><a href="/papers_2023/Amortized_Pose_Estimation_for_X-Ray_Single_Particle_Imaging.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Rethinking Performance Measures of RNA Secondary Structure Problems</p>
                <p class="authors">Frederic Runge, Jörg KH Franke, Daniel Fertmann, Frank Hutter</p>
                <p class="preprints"><a href="/papers_2023/Rethinking_Performance_Measures_of_RNA_Secondary_Structure_Problems.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Structure-based and leakage-free data splits for rigorous protein function evaluation</p>
                <p class="authors">Charlotte Rochereau, Mohammed AlQuraishi, Arthur Valentin, Gergo Nikolenyi</p>
                <p class="preprints"></p>
              </li>
              <li>
                <p class="title">Uncovering sequence diversity from a known protein structure</p>
                <p class="authors">Luca alessandro silva, Barthélémy Meynard, Carlo Lucibello, Christoph Feinauer</p>
                <p class="preprints"><a href="/papers_2023/Uncovering_sequence_diversity_from_a_known_protein_structure.pdf">[paper]</a></p>
              </li>
              <li>
                <p class="title">Exploiting language models for protein discovery with latent walk-jump sampling</p>
                <p class="authors">Sai Pooja Mahajan, Nathan Frey, Daniel Berenberg, Joseph Kleinhenz, Richard Bonneau, Vladimir Gligorijevic, Andrew Watkins, Saeed Saremi</p>
                <p class="preprints"><a href="/papers_2023/Exploiting_language_models_for_protein_discovery_with_latent_walk-jump_sampling.pdf">[paper]</a></p>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div> -->
  

  </div>


<div id ="challenge" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Challenge 2024</h2>
        <p>This year we are running a challenge on the <a href="https://pinder.sh">Pinder</a> and <a href="https://plinder.sh">Plinder</a> datasets to evaluate how well the community is currently doing for protein-protein interaction prediction and protein-ligand complex prediction. </p>
        <h3 class="section-heading">Details of the challenge</h3>
        
        <p>
        
        
        <p>To submit your trained model you will need to make an inference docker image on HuggingFace Spaces using the following templates:
          <ul class="bulleted">
            <li><a href="https://huggingface.co/spaces/MLSB/pinder_inference_template">Pinder inference template</a></li>
            <li><a href="https://huggingface.co/spaces/MLSB/plinder_inference_template">Plinder inference template</a></li>
          </ul>

        </p>

        <h3 class="section-heading"> Rules for model training</h3>
        <p>
        <ul class="bulleted">
        <li>Participants MUST use the sequences and SMILES in the provided train and validation sets from PINDER or PLINDER. In order to ensure no leakage, external data augmentation is not allowed.</li>
          <li>If starting structures/conformations need to be generated for the model, then this can only be done from the training and validation sequences and SMILES. Note that this is only the case for train & validation - no external folding methods or starting structures are allowed for the test set under any circumstance!. Only the predicted structures/conformers themselves may be used in this way, the embeddings or models used to generate such predictions may not. E.g. it is not valid to “distill” a method that was not trained on PLINDER/PINDER</li>
          <li>The PINDER and PLINDER datasets should be used independently; combining the sets is considered augmentation and is not allowed.</li>

            <li>For inference, only the inputs provided in the evaluation sets may be used: canonical sequences, structures and MSAs; no alternate templates or sequences are permitted. The inputs that will be used by assessors for each challenge track is as follows: 
              <ul class="bulleted">
                <li>PLINDER: <code>(SMILES, monomer protein structure, monomer FASTA, monomer MSA)</code></li>
                <li>PINDER: <code>(monomer protein structure 1, monomer protein structure 2, FASTA 1, FASTA 2, MSA 1, MSA 2)</code></li>
              </ul>
            </li>
              <li>Model selection must be performed exclusively on the validation set designed for this purpose within the PINDER and PLINDER datasets.</li>
                <li>Methods relying on any model derivatives or embeddings trained on structures outside the PINDER/PLINDER training set are not permitted (e.g., ESM2, MSA: allowed; ESM3/ESMFold/SAProt/UniMol: not allowed).</li>
        </ul>
        </p>
        <p> Please find the technical documentation for how to use the datasets for the challenge:
          <ul class="bulleted">
            <li><a href="https://pinder-org.github.io/pinder/pinder-mlsb.html">PINDER</a></li>
            <li><a href="https://plinder-org.github.io/plinder/examples/mlsb_challenge.html">PLINDER</a></li>
            
          </ul>  
          </p>

        <h3 class="section-heading"> Rules for valid inference pipeline </h3>

        <p>Submission system will use Hugging Face Spaces. To qualify for submission, each team must:</p>
        <p>
          <ul class="bulleted">
            <li>Provide an MLSB submission ID or a link to a preprint/paper describing their methodology. This publication does not have to specifically report training or evaluation on the P(L)INDER dataset. Previously published methods, such as DiffDock, only need to link their existing paper. Note that entry into this competition does not equate to an MLSB workshop paper submission.</li>
<li>Create a copy of the provided inference template.

  Go to the top right corner of the page and click on the drop-down menu (vertical ellipsis) right next to the “Community”, then select “Duplicate this space”.
</li>
<li>Change files in the newly create space to reflect the peculiarities of your model 
<ul class="bulleted">
  <li>Edit <code>requirements.txt</code> to capture all dependencies.</li>
  <li>Include a <code>inference_app.py</code> file. This contains a <code>predict</code> function that should be modified to reflect the specifics of inference using their model.</li>
<li>Include a <code>train.py</code> file to ensure that training and model selection use only the PINDER/PLINDER datasets and to clearly show any additional hyperparameters used.</li>
<li>Provide a LICENSE file that allows for reuse, derivative works, and distribution of the provided software and weights (e.g., MIT or Apache2 license).</li>
<li>Modify the Dockerfile as appropriate (including selecting the right base image)</li>
</ul>

</li>
<li>Submit to the leaderboard via the <a href="https://huggingface.co/spaces/MLSB/leaderboard2024" rel="nofollow">designated form</a>.
  <ul dir="auto">
  <li>On submission page, add reference to the newly created space in the format username/space (e.g mlsb/alphafold3)</li>
  </ul>
  </li>





        <h3 class="section-heading">How to submit and view results</h3>
        <iframe
        src="https://mlsb-leaderboard2024.hf.space"
        frameborder="0"
        width="100%"
        height="700px"
      ></iframe>
        
        <h3 class="section-heading">Metrics</h3>
Primary Ranking Metric:
<ul class="bulleted">
<li>PLINDER: lDDT-PLI</li>
<li>PINDER: DockQ</li>
</ul>
<p>Other metrics computed by PINDER/PLINDER will be displayed on the leaderboard but will not influence the ranking.</p>
<p>The winners will be invited to present their work at the MLSB workshop.</p>

<h3 class="section-heading">Evaluation Datasets</h3>

<p>Although the exact composition of the eval set will be shared at a future date, below we provide an overview of the dataset and what to expect</p>

<p><ul class="bulleted">
  <li>Two leaderboards, one for each of PINDER and PLINDER, will be created using a single evaluation set for each.</li>
  <li>Evaluation sets will be subsets of 150-200 structures from the current PINDER and PLINDER test splits (subsets to enable reasonable eval runtime).</li>
  <li>Each evaluation sample will contain a predefined input/output to ensure performance assessment is model-dependent, not input-dependent.</li>
  <li>The focus will be exclusively on flexible docking/co-folding, with a single canonical structure per protein, sampled from apo and predicted structures.</li>
  <li>Monomer input structures will be sampled from paired structures available in PINDER/PLINDER, balanced between apo and predicted structures and stratified by "flexibility" level according to specified conformational difference thresholds.</li>
  </ul></p>
        
        <h3 class="section-heading">Key Dates</h3>


        <p class=""><u>Training workshop</u> September 24th, 2024, virtual (<a href="https://unibas.zoom.us/meeting/register/u5EkcOusrzsiHtLDImocB5PM0RLK9vC-g-kW#/registration">Register here</a>)</p>
<p><u>Leaderboard Opens:</u> October 9th, 2024 (following acceptance notifications for MLSB).</p>
<p><u>Leaderboard Closes:</u> <s>November 9th, 2024</s> November 18th, 2024</p>
<p><u>Winner Notification:</u> Wednesday, November 27th, 2024 </p>


<h3 class="section-heading">Questions?</h3>
<p>If you have trouble we invite you to <a href="https://discord.gg/NFVgccusBj">join the PINDER/PLINDER discord server</a></p>



<div class="img-thumbnail">

  <img alt="" width="150px"  loading="lazy" src="https://plinder.sh/icons/vantai_logo.svg" style="color: transparent; padding:20px">

  
  <img alt="" width="150px" class="p-3" src="https://plinder.sh/_next/image?url=%2Ficons%2Fnvidia.png&amp;w=3840&amp;q=75" style="padding:20px" >
  
  <img alt="" width="150px"   src="https://plinder.sh/icons/swiss_institute_of_bioinformatics.svg" style="color: transparent; padding:20px"><img alt="" width="150px"  src="https://plinder.sh/_next/image?url=%2Ficons%2Fbiozentrum.png&amp;w=3840&amp;q=75" style="color: transparent;padding:20px">
   
  

  <img alt="" class="img-fluid p-3" width="150px" loading="lazy" src="https://plinder.sh/_next/image?url=%2Ficons%2Funiversity_of_basel.png&amp;w=3840&amp;q=75" style="color: transparent; padding:20px">
    
  <img alt="" class="p-3" width="150px" src="https://plinder.sh/_next/image?url=%2Ficons%2Fmit_csail.png&amp;w=3840&amp;q=75" style="color: transparent;padding:20px"><img alt="" width="150px"  src="https://plinder.sh/_next/image?url=%2Ficons%2Fmit.png&amp;w=3840&amp;q=75" style="color: transparent;"><img  width="150px" class="p-3" src="https://plinder.sh/_next/image?url=%2Ficons%2Feuro_hpc.webp&amp;w=3840&amp;q=75" style="color: transparent; padding:20px"><img width="100px" src="https://plinder.sh/_next/image?url=%2Ficons%2Feuropean_commission.png&amp;w=3840&amp;q=75" style="color: transparent;">
  <img src="https://huggingface.co/datasets/huggingface/brand-assets/resolve/main/hf-logo-with-title.svg" alt="HuggingFace" width="200px" style="padding:20px">
     
</div>

      </div>
    </div>
  </div>
</div> 

<div id="organizers" class="content-section-a" style="border-top: 0">
  <div class="container">
    <h2 class="section-heading">Organizers</h2>
    <div class="row wow fadeInRightBig"  data-animation-delay="200">
      <div class="row">
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/gabriele_corso.png" alt="Photo of Gabriele Corso">
        </center>
        <h4 class="section-heading">
            <center>
              Gabriele Corso<br>
              <i><font size="3">MIT</font></i>
            </center>
          </h4>
      </div>
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/gina.jpg" alt="Photo of Gina El Nesr">
        </center>
        <h4 class="section-heading">
            <center>
              Gina El Nesr<br>
              <i><font size="3">Stanford University</font></i>
            </center>
          </h4>
      </div>
      <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/vignesh.png" alt="Photo of Vignesh Ram Somnath">
        </center>
        <h4 class="section-heading">
            <center>
              Vignesh Ram Somnath<br>
              <i><font size="3">ETH Zurich</font></i>
            </center>
          </h4>
      </div>
      </div>

      <div class="col-sm-6 wow fadeInRightBig"  data-animation-delay="200">
        <center>
          <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/zeming_lin.jpg" alt="Photo of Ellen Zhong">
        </center>
        <h4 class="section-heading">
          <center>
              Zeming Lin<br>
            <i><font size="3">EvolutionaryScale</font></i>
          </center>
        </h4>
      </div>
      
        <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/simon_duerr.png" alt="Photo of Simon Duerr">
          </center>
          <h4 class="section-heading">
            <center>
              Simon Duerr<br>
              <i><font size="3">EPFL</font></i>
            </center>
          </h4>
        </div>
      </div>

      <div class="row">
        
        <div class="col-sm-6 wow fadeInRightBig"  data-animation-delay="200">
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/hannah_wayment_steele.png" alt="Photo of Hannah Wayment-Steele">
          </center>
          <h4 class="section-heading">
            <center>
              Hannah Wayment-Steele<br>
              <i><font size="3">University of Wisconsin–Madison</font></i>
            </center>
          </h4>
        </div>

        <div class="row">
          <div class="col-sm-4 wow fadeInRightBig"  data-animation-delay="200">
            <center>
              <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="180" width="180" src="img/organizer/sergey.jpg" alt="Photo of Sergey Ovchinnikov">
            </center>
            <h4 class="section-heading">
              <center>
                Sergey Ovchinnikov<br>
                <i><font size="3">MIT</font></i>
              </center>
            </h4>
          </div>

    </div>
      </div>
    </div>
  </div>

	<div id="related" class="content-section-b" style="border-top: 0">
        <div class="container" id="related">
		    <h2 class="section-heading">Past Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead">Here are links to previous MLSB Workshops at NeurIPS!</p>
                    <ul class="bulleted">
                        <li><a href="index_2023.html"><span class="network-name">2023 Workshop</span></a> </li>
                        <li><a href="index_2022.html"><span class="network-name">2022 Workshop</span></a> </li>
                        <li><a href="index_2021.html"><span class="network-name">2021 Workshop</span></a> </li>
                        <li><a href="index_2020.html"><span class="network-name">2020 Workshop</span></a> </li>
                    </ul>
				</div>
            </div>
        </div>
        <br/>
		<br/>
		</div>
	</div>

<div id="related" class="content-section-b" style="border-top: 0">
        <!-- <div class="container" id="related">
        <h2 class="section-heading">Related Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead"> Here are some <a href="https://neurips.cc/Conferences/2023/Schedule?type=Workshop">related workshops at NeurIPS</a> covering structure, biology, and physical science this year:
                    <ul class="bulleted">
                        <li><a href="https://genbio-workshop.github.io">Generative AI and Biology</a></li>
                        <li><a href="https://ai4d3.github.io">New Frontiers of AI for Drug Discovery and Development</a> </li>
                        <li><a href="https://ml4physicalsciences.github.io/2023/">Machine Learning and the Physical Science
                        </a></li>
            <li><a href="https://ai4sciencecommunity.github.io">AI for Science: from Theory to Practice</a></li>
            <li><a href="https://diffusionworkshop.github.io">Diffusion Models</a></li>
                    </ul>
        </div>
            </div>
        </div> -->
        <br/>
    <br/>
  <div id="sponsors" class="content-section-a" style="border-top: 0">
    <div class="container">
      <h2 class="section-heading">Sponsors</h2>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/ATOMICAI_logo+name_Yellow_Red_black.png" alt="Atomic Logo"> </div>
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/third-rock.jpeg" alt="TRV Logo"> </div>
        <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="100" src="./img/evo_scale.svg" alt="EvolutionaryScale"> </div>
      </div>
      <br>
      <br>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <!-- <div class="col-md-3 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="170" src="./img/neurosnap.png" alt="Neurosnap"> </div> -->
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/prescient.png" alt="Prescient"> </div>
        <div class="col-md-4"  data-animation-delay="200"> <img  class="center" style="" height="80" src="./img/generate-bio-afpc-logo-cmyk-black.png" alt="Generate Bio"> </div>
    </div>
  </div>
</div>
<footer>
  <div class="container">
    <div class="row">
      <div class="col-md-12" id="contact">
        <h3 class="footer-title">Contact</h3>
        <p> Please contact <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a> with any questions. <br><br><br></p>
      </div>
      <!-- /col-xs-7 -->
    </div>
  </div>
</footer>

<!-- JavaScript -->
<script src="js/jquery-1.10.2.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/owl.carousel.js"></script>
<script src="js/script.js"></script>
<!-- StikyMenu -->
<script src="js/stickUp.min.js"></script>
<script type="text/javascript">
	  jQuery(function($) {
		$(document).ready( function() {
		  $('.navbar-default').stickUp();

		});
	  });

	</script>
<!-- Smoothscroll -->
<script type="text/javascript" src="js/jquery.corner.js"></script>
<script src="js/wow.min.js"></script>
<script>
	 new WOW().init();
	</script>
<script src="js/classie.js"></script>
<!-- Magnific Popup core JS file -->
<script src="js/jquery.magnific-popup.js"></script>
</body>
</html>
