<!doctype html>
<!-- FlatFy Theme - Andrea Galanti /-->
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->
<!--[if IE 9]>    <html class="no-js ie9" lang="en"> <![endif]-->
<!--[if gt IE 9]><!-->
<html>
<!--<![endif]-->
<script>
function showhide(id) {
  var x = document.getElementById(id);
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
<meta name="description" content="Workshop at the 36th Conference on Neural Information Processing Systems">
<meta name="author" content="">
<title>Machine Learning in Structural Biology</title>

<!-- Bootstrap core CSS -->
<link href="css/bootstrap.min.css" rel="stylesheet">

<!-- Custom Google Web Font -->
<link href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
<link href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900,100italic,300italic,400italic,700italic,900italic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=Arvo:400,700" rel="stylesheet" type="text/css">

<!-- Custom CSS-->
<link href="css/general.css" rel="stylesheet">

<!-- Owl-Carousel -->
<link href="css/custom.css" rel="stylesheet">
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/owl.theme.css" rel="stylesheet">
<link href="css/style.css" rel="stylesheet">
<link href="css/animate.css" rel="stylesheet">

<!-- Magnific Popup core CSS file -->
<link rel="stylesheet" href="css/magnific-popup.css">
<script src="js/modernizr-2.8.3.min.js"></script> <!-- Modernizr /--> 
<!--[if IE 9]>
		<script src="js/PIE_IE9.js"></script>
	<![endif]--> 
<!--[if lt IE 9]>
		<script src="js/PIE_IE678.js"></script>
	<![endif]--> 

<!--[if lt IE 9]>
		<script src="js/html5shiv.js"></script>
	<![endif]-->

</head>

<body id="home">

<!-- Preloader -->
<div id="preloader">
  <div id="status"></div>
</div>

<!-- FullScreen -->
<div class="intro-header">
  <div class="col-xs-12 text-center abcen1">
    <h1 class="h1_home wow fadeIn" data-wow-delay="0.4s">Machine Learning in Structural Biology</h1>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Workshop at the 36th Conference on Neural Information Processing Systems</h3>
    <h3 class="h3_home wow fadeIn" data-wow-delay="0.6s">Saturday, December 3rd, 2022</h3>
    <ul class="list-inline intro-social-buttons">
      <li><a href="https://neurips.cc/virtual/2022/workshop/50005" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">Livestream</span></a> </li>
      <!--
		<li><a href="https://eventhosts.gather.town/app/1UBzVZ8TnX03EOsa/mlsb" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">GatherTown</span></a>
		</li>
        -->
      <li><a href="index_2021.html" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">2021 Workshop</span></a> </li>
      <li><a href="index_2020.html" class="btn  btn-lg mybutton_cyano wow fadeIn" data-wow-delay="0.8s"><span class="network-name">2020 Workshop</span></a> </li>
      <!--
				<li id="download" ><a href="" class="btn  btn-lg mybutton_cyano wow swing wow fadeIn" data-wow-delay="1.2s"><span class="network-name">Gather</span></a>
				</li>
                -->
    </ul>
  </div>
  <!-- /.container -->
  <div class="col-xs-12 text-center abcen wow fadeIn">
    <div class="button_down "> <a class="imgcircle wow bounceInUp" data-wow-duration="1.5s"  href="#about"> <img class="img_scroll" src="img/icon/circle.png" alt="Go down to content"> </a> </div>
  </div>
</div>

<!-- NavBar-->
<nav class="navbar-default" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-ex1-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
      <a class="navbar-brand" href="#home">MLSB</a> </div>
    <div class="collapse navbar-collapse navbar-right navbar-ex1-collapse">
      <ul class="nav navbar-nav">
        <li class="menuItem"><a href="#about">About</a></li>
        <!-- <li class="menuItem"><a href="#callforpapers">Call for Papers</a></li> -->
        <li class="menuItem"><a href="#faq">FAQ</a></li>
        <li class="menuItem"><a href="#invitedspeakers">Invited Speakers</a></li>
        <!-- <li class="menuItem"><a href="#preregistration">Pre-Registration</a></li> -->
        <li class="menuItem"><a href="#schedule">Schedule</a></li>
        <li class="menuItem"><a href="#accepted">Accepted Papers</a></li>
        <li class="menuItem"><a href="#organizers">Organizers</a></li>
        <!--<li class="menuItem"><a href="#related">Related Workshops</a></li> -->
        <li class="menuItem"><a href="#sponsors">Sponsors</a></li>
        <!-- <li class="menuItem"><a href="#contact">Contact</a></li> -->
      </ul>
    </div>
  </div>
</nav>

<!-- About -->
<div id="about" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">About</h2>
        <p class="lead"> In only a few years, structural biology, the study of the 3D structure or shape of proteins and
          other biomolecules, has been transformed by breakthroughs from machine learning algorithms.
          Machine learning models are now routinely being used by experimentalists to predict structures
          that can help answer real biological questions (e.g. AlphaFold), accelerate the experimental
          process of structure determination (e.g. computer vision algorithms for cryo-electron
          microscopy), and have become a new industry standard for bioengineering new protein
          therapeutics (e.g. large language models for protein design). Despite all this progress, there are
          still many active and open challenges for the field, such as modeling protein dynamics,
          predicting higher order complexes, pushing towards generalization of protein folding physics,
          and relating the structure of proteins to the in vivo and contextual nature of their underlying
          function. These challenges are diverse and interdisciplinary, motivating new kinds of machine
          learning systems and requiring the development and maturation of standard benchmarks and
          datasets. </p>
        <p class="lead"> In this exciting time for the field, our workshop, “Machine Learning in Structural Biology” (MLSB),
          seeks to bring together relevant experts, practitioners, and students across a broad community
          to focus on these challenges and opportunities. We believe the union of these communities, including the
          geometric and graph learning communities, NLP researchers, and structural biologists with
          domain expertise at our workshop can help spur
          new ideas, spark collaborations, and advance the impact of machine learning in structural
          biology. Progress at this intersection promises to unlock new scientific discoveries and the ability
          to design novel medicines. </p>
        <p class="lead"> MLSB will be an in-person workshop with hybrid elements (livestream + chat for questions) in <b>Room 288 - 289 on Saturday, December 3rd, 2022</b> at <a href="https://neurips.cc">NeurIPS</a>. </p>
        <p class="lead"> Stay updated on changes and workshop news by joining <a href="https://groups.google.com/g/workshopmlsb">our mailing list</a>. </p>
      </div>
    </div>
  </div>
</div>
<div id ="faq" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInLeftBig" data-animation-delay="200">
        <h2 class="section-heading">FAQ</h2>
        <p> <b>WIFI</b> <br/>
          network name: NeurIPS <br/>
          password: conference </p>
        <p> <b>SCHEDULE</b> MLSB will begin at 8:30am CST and end at 5:00pm CST on Saturday, Dec 3. The physical workshop will take place in Room 288 - 289 in the New Orleans Convention Center. The schedule of talks and poster sessions is available at our website, mlsb.io and on the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. </p>
        <p> <b>PANEL</b> -
          We will be hosting a panel discussion including our invited speakers on research directions in Machine Learning for Structural Biology from 2:50-3:50p CST. We are soliciting additional discussion topics and questions -- let us know what you would like to hear about by filling out this google form: <a href="https://forms.gle/fiJ6ZzsmepKL87tMA">https://forms.gle/fiJ6ZzsmepKL87tMA</a>. </p>
        <p> <b>POSTER SESSIONS</b> - 
          We have scheduled two poster sessions at 11:15-12:15pm CST and 3:50-4:50pm CST. Posters can be set up before the conference begins, between 8:00-8:30am, during the break between 9:45-10:05am, or at the beginning of the first poster session.
          
          We ask that posters are 24"W x 36"H, as there may be space constraints in the room that limit the display of wider posters. However, if you already have a poster of a different size, it is acceptable to use that. </p>
        <p> <b>LUNCH + HAPPY HOUR</b> - 
          MLSB will be providing boxed lunches to attendees during the workshop. In addition, there will be a happy hour during the afternoon poster session, where drinks will be served (non-alcoholic options will be available). </p>
        <p> <b>ORALS</b> - 
          Each oral will have a 15 minute timeslot. We anticipate that talks should last approximately 10 minutes, allowing 5 minutes for questions and transition between speakers. If you are presenting in-person, you may check slides before the workshop (8:00-8:30am CST) or during the break (9:45-10:05am CST). </p>
        <p> Please check the schedule on our website, mlsb.io, for your presentation time. If you are not able to make this time, let us know as soon as possible. Additionally, below we have listed the format we currently have for each oral presentation (In-person, Live over Zoom, or Prerecorded). If this information is not correct, please let us know. You can contact us at workshopmlsb@gmail.com with any issues. </p>
        <p> <b>VIRTUAL PARTICIPATION</b> - 
          MLSB will have some hybrid elements. There will be a livestream (a link is available at the <a href="https://neurips.cc/virtual/2022/workshop/50005">NeurIPS website</a>. Chat will be available on the NeurIPS workshop site for asking questions.
          
          Currently, there is no official venue for presenting posters virtually at MLSB. However, we are considering options to enable greater virtual participation over the next few days. If you have a poster at MLSB but plan to participate only virtually, please let us know by adding your name to this <a href="https://forms.gle/ntQDsULXaVkzVXYR8">google form</a> as soon as possible. </p>
      </div>
    </div>
  </div>
</div>
<!--
<div id ="callforpapers" class="content-section-b">
  <div class="container">
    <div class="row">
      <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
        <h2 class="section-heading">Call For Papers</h2>
        <p>We welcome submissions of short papers leveraging machine learning to address problems in structural biology, including but not limited to:</p>
        <ul class="bulleted">
          <li>Prediction of biomolecular structures, complexes, and interactions</li>
          <li>Generative models for structure and/or sequence</li>
          <li>Protein, RNA, and DNA design</li>
          <li>Protein structure determination including cryo-EM/ET</li>
          <li>Geometric and symmetry-aware deep learning</li>
          <li>Conformational change, ensembles, and dynamics</li>
          <li>Integration of biomolecular physics</li>
          <li>Function and property prediction</li>
          <li>Structural bioinformatics and systems biology</li>
          <li>Language models and other representations of proteins</li>
          <li>Forward-looking position papers</li>
        </ul>
        <p>We request anonymized PDF submissions by <strong>Thursday, September 29th, 2022, at 11:59PM</strong> anywhere on earth through our submission website on <a href="https://cmt3.research.microsoft.com/MLSB2022">CMT</a>. </p>
        <p>Papers should present novel work that has not been previously accepted at an archival venue at the time of submission.  Submissions should be a <em>maximum</em> of 5 pages (excluding references and appendices) in PDF format, using the <a href="https://neurips.cc/Conferences/2022/PaperInformation/StyleFiles">NeurIPS style files</a>, and fully anonymized as per the requirements of NeurIPS. The NeurIPS checklist can be omitted from the submission. Submissions meeting these criteria will go through a light, double-blind review process. Reviewer comments will be returned to the authors as feedback.</p>
        <p>Accepted papers will be invited to present a poster at the workshop, with nominations of spotlight talks at the discretion of the organizers.  This workshop is considered non-archival and does not publish proceedings, however authors of accepted contributions will have the option to make their work available through the workshop website. Presentation of work that is concurrently in submission is welcome. We welcome papers sharing encouraging work-in-progress results or forward-looking position papers that would benefit from feedback and community discussion at our workshop. </p>
        <h3 class="section-heading">Important Dates</h3>
        <p class="">Submission Deadline: <b>Thursday, September 29th, 2022, 11:59 PM, Anywhere on Earth.</b></p>
        <p class="">Notification of Acceptance: Friday, October 20th, 2022.</p>
        <p class="">Workshop Date: Saturday, December 3rd, 2022.</p>
      </div>
    </div>
  </div>
</div>
-->
<div id="invitedspeakers" class="content-section-a" style="border-top: 0">
  <div class="container">
    <div class="col-md-6 col-md-offset-3 text-center wrap_title">
      <h2>Invited Speakers</h2>
    </div>
    <div class="row">
      <div class="col-sm-6 wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/david_fleet.jpg" alt="David Fleet">
        <h3>David Fleet</h3>
        <p class="lead" style="margin-bottom:0;" >Professor of Computer Science at University of Toronto.</p>
        <a onclick="showhide('david_fleet')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="david_fleet">
          <p> David Fleet is a Professor of Computer Science at the University of Toronto
            and leads the Google Research Brain Team in Toronto. His group has developed several
            vision-based algorithms for reconstructing protein structure from cryo-electron
            microscopy (cryo-EM) data. He is an scientific advisor for Structura Biotechnology whose
            software platform for cryo-EM image processing is widely used in experimental structural
            biology. </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> --> 
      </div>
      <!-- /.col-md-6 -->
      <div class="col-sm-6 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/alexander_rives.jpg" alt="Alexander Rives">
        <h3>Alexander Rives</h3>
        <p class="lead" style="margin-bottom:0;">Research Scientist at Meta AI.</p>
        <a onclick="showhide('alexander_rives')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="alexander_rives">
          <p>Alexander Rives is a Research Scientist at Meta AI. His group
            focuses on unsupervised representation learning from large-scale protein sequence and
            structure data for protein design. </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> --> 
      </div>
      <!-- /.col-md-6 --> 
    </div>
    <!-- row -->
    
    <div class="row tworow">
      <div class="col-sm-6 wow fadeInDown text-center"> <img  class="magnify img-rounded" src="img/speakers/kathryn_tunyasuvunakool.jpg" alt="Kathryn Tunyasuvunakool">
        <h3>Kathryn Tunyasuvunakool</h3>
        <p class="lead" style="margin-bottom:0;">Staff Research Scientist at DeepMind.</p>
        <a onclick="showhide('kathryn_tunyasuvunakool')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="kathryn_tunyasuvunakool">
          <p>Kathryn Tunyasuvunakool is a Staff Research Scientist at DeepMind,
            where she worked on the development of AlphaFold, an machine learning algorithm for protein structure
            prediction deemed Science Magazine’s Breakthrough of the Year in 2021. She has
            played an important role in the creation of the AlphaFold protein structure database, which has made the computational predictions
            more accessible to biologists. </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> --> 
      </div>
      <!-- /.col-lg-4 -->
      
      <div class="col-sm-6  wow fadeInDown text-center"> <img class="magnify img-rounded" src="img/speakers/max_welling.jpg" alt="Max Welling">
        <h3>Max Welling</h3>
        <p class="lead" style="margin-bottom:0;">Distinguished Scientist at Microsoft Research.</p>
        <a onclick="showhide('max_welling')">Show/Hide Bio</a>
        <div style="display: none; text-align: left; text-indent: 30px;" id="max_welling">
          <p>Max Welling is a Distinguished Scientist at Microsoft Research and the
            director of Microsoft Research Amsterdam. </p>
        </div>
        <!-- <p><a class="btn btn-embossed btn-primary view" role="button">View Details</a></p> --> 
      </div>
      <!-- /.col-lg-4 --> 
      
    </div>
    <!-- /.row --> 
    
  </div>
</div>

<!-- Schedule -->
<div id ="schedule" class="content-section-b">
  <div class="container">
    <h2 class="section-heading">Schedule (CST)</h2>
    <div class="row">
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>08:30</th>
            <td colspan="4" rowspan="1" class="stage-saturn">Opening Remarks</td>
          </tr>
          <tr>
            <th>08:35</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - David Fleet</td>
          </tr>
          <tr>
            <th>08:40</th>
          </tr>
          <tr>
            <th>08:45</th>
          </tr>
          <tr>
            <th>08:50</th>
          </tr>
          <tr>
            <th>08:55</th>
          </tr>
          <tr>
            <th>09:00</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Latent Space Diffusion Models of Cryo-EM Structure <br/>
                <em>Karsten Kreis · Tim Dockhorn · Zihao Li · Ellen Zhong</em></p></td>
          </tr>
          <tr>
            <th>09:05</th>
          </tr>
          <tr>
            <th>09:10</th>
          </tr>
          <tr>
            <th>09:15</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Protein Structure and Sequence Generation with Equivariant Denoising Diffusion Probabilistic Models<br />
                <em>Namrata Anand · Tudor Achim</em></p></td>
          </tr>
          <tr>
            <th>09:20</th>
          </tr>
          <tr>
            <th>09:25</th>
          </tr>
          <tr>
            <th>09:30</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Predicting conformational landscapes of known and putative fold-switching proteins using AlphaFold2<br/>
                <em>Hannah Wayment-Steele · Sergey Ovchinnikov · Lucy Colwell · Dorothee Kern</em></p></td>
          </tr>
          <tr>
            <th>09:35</th>
          </tr>
          <tr>
            <th>09:40</th>
          </tr>
          <tr>
            <th>09:45</th>
            <td colspan="4" rowspan="4" class="stage-earth">Break</td>
          </tr>
          <tr>
            <th>09:50</th>
          </tr>
          <tr>
            <th>09:55</th>
          </tr>
          <tr>
            <th>10:00</th>
          </tr>
          <tr>
            <th>10:05</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Kathryn Tunyasuvunakool</td>
          </tr>
          <tr>
            <th>10:10</th>
          </tr>
          <tr>
            <th>10:15</th>
          </tr>
          <tr>
            <th>10:20</th>
          </tr>
          <tr>
            <th>10:25</th>
          </tr>
          <tr>
            <th>10:30</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">SWAMPNN: End-to-end protein structures alignment<br/>
                <em>Jeanne Trinquier · Samantha Petti · Shihao Feng · Johannes Soeding · Martin Steinegger · Sergey Ovchinnikov</em></p></td>
          </tr>
          <tr>
            <th>10:35</th>
          </tr>
          <tr>
            <th>10:40</th>
          </tr>
          <tr>
            <th>10:45</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking<br/>
                <em>Gabriele Corso · Hannes Stärk · Bowen Jing · Regina Barzilay · Tommi Jaakkola</em></p></td>
          </tr>
          <tr>
            <th>10:50</th>
          </tr>
          <tr>
            <th>10:55</th>
          </tr>
          <tr>
            <th>11:00</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Dynamic-backbone protein-ligand structure prediction with multiscale generative diffusion models<br/>
                <em>Zhuoran Qiao · Weili Nie · Arash Vahdat · Thomas Miller · Anima Anandkumar</em></p></td>
          </tr>
          <tr>
            <th>11:05</th>
          </tr>
          <tr>
            <th>11:10</th>
          </tr>
          <tr>
            <th>11:15</th>
            <td colspan="4" rowspan="12" class="stage-saturn">Poster Session</td>
          </tr>
          <tr>
            <th>11:20</th>
          </tr>
          <tr>
            <th>11:25</th>
          </tr>
          <tr>
            <th>11:30</th>
          </tr>
          <tr>
            <th>11:35</th>
          </tr>
          <tr>
            <th>11:40</th>
          </tr>
          <tr>
            <th>11:45</th>
          </tr>
          <tr>
            <th>11:50</th>
          </tr>
          <tr>
            <th>11:55</th>
          </tr>
          <tr>
            <th>12:00</th>
          </tr>
          <tr>
            <th>12:05</th>
          </tr>
          <tr>
            <th>12:10</th>
          </tr>
          <tr>
            <th>12:15</th>
            <td colspan="4" rowspan="9" class="stage-earth">Lunch</td>
          </tr>
          <tr>
            <th>12:20</th>
          </tr>
          <tr>
            <th>12:25</th>
          </tr>
          <tr>
            <th>12:30</th>
          </tr>
          <tr>
            <th>12:35</th>
          </tr>
          <tr>
            <th>12:40</th>
          </tr>
          <tr>
            <th>12:45</th>
          </tr>
          <tr>
            <th>12:50</th>
          </tr>
          <tr>
            <th>12:55</th>
          </tr>
        </table>
      </div>
      <div class="col-md-6">
        <table class="schedule">
          <tr>
            <th>01:00</th>
            <td colspan="4" rowspan="5" class="stage-saturn">Invited Speaker - Max Welling</td>
          </tr>
          <tr>
            <th>01:05</th>
          </tr>
          <tr>
            <th>01:10</th>
          </tr>
          <tr>
            <th>01:15</th>
          </tr>
          <tr>
            <th>01:20</th>
          </tr>
          <tr>
            <th>01:25</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">EquiFold: Protein Structure Prediction with a Novel Coarse-Grained Structure Representation<br/>
                <em>Jae Hyeon Lee · Payman Yadollahpour · Andrew Watkins · Nathan Frey · Andrew Leaver-Fay · Stephen Ra · Vladimir Gligorijevic · Kyunghyun Cho · Aviv Regev · Richard Bonneau</em></p></td>
          </tr>
          <tr>
            <th>01:30</th>
          </tr>
          <tr>
            <th>01:35</th>
          </tr>
          <tr>
            <th>01:40</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Predicting Ligand – RNA Binding Using E3-Equivariant Network and Pretraining<br/>
                <em>Zhenfeng Deng · Ruichu Gu · Hangrui Bi · Xinyan Wang · Zhaolei Zhang · Han Wen</em></p></td>
          </tr>
          <tr>
            <th>01:45</th>
          </tr>
          <tr>
            <th>01:50</th>
          </tr>
          <tr>
            <th>01:55</th>
            <td colspan="4" rowspan="5" class="stage-earth">Invited Speaker - Alexander Rives</td>
          </tr>
          <tr>
            <th>02:00</th>
          </tr>
          <tr>
            <th>02:05</th>
          </tr>
          <tr>
            <th>02:10</th>
          </tr>
          <tr>
            <th>02:15</th>
          </tr>
          <tr>
            <th>02:20</th>
            <td colspan="4" rowspan="3" class="stage-saturn">Contributed Talk
              <p style="font-weight:normal">Seq2MSA: A Language Model for Protein Sequence Diversification<br/>
                <em>Pascal Sturmfels · Roshan Rao · Robert Verkuil · Zeming Lin · Tom Sercu · Adam Lerer · Alex Rives</em></p></td>
          </tr>
          <tr>
            <th>02:25</th>
          </tr>
          <tr>
            <th>02:30</th>
          </tr>
          <tr>
            <th>02:35</th>
            <td colspan="4" rowspan="3" class="stage-earth">Contributed Talk
              <p style="font-weight:normal">Metal3D: Accurate prediction of transition metal ion location via deep learning<br/>
                <em>Simon Dürr</em></p></td>
          </tr>
          <tr>
            <th>02:40</th>
          </tr>
          <tr>
            <th>02:45</th>
          </tr>
          <tr>
            <th>02:50</th>
            <td colspan="4" rowspan="2" class="stage-saturn">Break</td>
          </tr>
          <tr>
            <th>02:55</th>
          </tr>
          <tr>
            <th>03:00</th>
          </tr>
          <tr>
            <th>03:05</th>
            <td colspan="4" rowspan="9" class="stage-earth">Panel Discussion</td>
          </tr>
          <tr>
            <th>03:10</th>
          </tr>
          <tr>
            <th>03:15</th>
          </tr>
          <tr>
            <th>03:20</th>
          </tr>
          <tr>
            <th>03:25</th>
          </tr>
          <tr>
            <th>03:30</th>
          </tr>
          <tr>
            <th>03:35</th>
          </tr>
          <tr>
            <th>03:40</th>
          </tr>
          <tr>
            <th>03:45</th>
          </tr>
          <tr>
            <th>03:50</th>
            <td colspan="4" rowspan="1" class="stage-saturn">Closing Remarks</td>
          </tr>
          <tr>
            <th>03:55</th>
            <td colspan="4" rowspan="13" class="stage-earth">Poster Session / Happy Hour</td>
          </tr>
          <tr>
            <th>04:00</th>
          </tr>
          <tr>
            <th>04:05</th>
          </tr>
          <tr>
            <th>04:10</th>
          </tr>
          <tr>
            <th>04:15</th>
          </tr>
          <tr>
            <th>04:20</th>
          </tr>
          <tr>
            <th>04:25</th>
          </tr>
          <tr>
            <th>04:30</th>
          </tr>
          <tr>
            <th>04:35</th>
          </tr>
          <tr>
            <th>04:40</th>
          </tr>
          <tr>
            <th>04:45</th>
          </tr>
          <tr>
            <th>04:50</th>
          </tr>
          <tr>
            <th>04:55</th>
          </tr>
        </table>
      </div>
    </div>
  </div>
  <div id="accepted" class="content-section-b" style="border-top: 0">
    <div class="container">
      <h2 class="section-heading">Accepted Papers</h2>
      <div class="row wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-12 wow fadeInRightBig"  data-animation-delay="200">
          <ul class="paperlist">
            <li>
              <p class="title">3D alignment of cryogenic electron microscopy density maps by minimizing their Wasserstein distance</p>
              <p class="authors">Aryan Tajmir Riahi, Geoffrey Woollard, Frederic Poitevin, Anne Condon, Khanh Dao Duc</p>
              <p class="preprints"><a href="/papers_2022/3D_Reconstruction_of_Protein_Complex_Structures_Using_Synthesized_Multi_View_AFM_Images.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">3D Reconstruction of Protein Complex Structures Using Synthesized Multi-View AFM Images</p>
              <p class="authors">Jaydeep Rade, Soumik Sarkar, Anwesha Sarkar, Adarsh Krishnamurthy</p>
              <p class="preprints"><a href="/papers_2022/3D_alignment_of_cryogenic_electron_microscopy_density_maps_by_minimizing_their_Wasserstein_distance.pdf">[paper]</a> <a href="https://arxiv.org/abs/2211.14662">[preprint]</a></p>
            </li>
            <li>
              <p class="title">A Benchmark Framework for Evaluating Structure-to-Sequence Models for Protein Design</p>
              <p class="authors">Jeffrey Chan, Seyone Chithrananda, David Brookes, Sam Sinai</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">A Federated Learning benchmark for Drug-Target Interaction</p>
              <p class="authors">Filip Svoboda, Gianluca Mittone, Nicholas Lane, Pietro Lió</p>
              <p class="preprints"><a href="/papers_2022/A_Federated_Learning_benchmark_for_Drug_Target_Interaction.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Adversarial Attacks on Protein Language Models</p>
              <p class="authors">Ginevra Carbone, Francesca Cuturello, Luca Bortolussi, Alberto Cazzaniga</p>
              <p class="preprints"><a href="/papers_2022/Adversarial_Attacks_on_Protein_Language_Models.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.24.513465v2">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Agile Language Transformers for Recombinant Protein Expression Optimization</p>
              <p class="authors">Jeliazko Jeliazkov, Maxim Shapovalov, Diego del Alamo, Matt Sternke, Joel Karpiak</p>
              <p class="preprints"><a href="/papers_2022/Agile_Language_Transformers_for_Recombinant_Protein_Expression_Optimization.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Allele-conditional attention mechanism for HLA-peptide complex binding affinity prediction</p>
              <p class="authors">Rodrigo Hormazabal, Doyeong Hwang, Kiyoung Kim, Sehui Han, Kyunghoon Bae, Honglak Lee</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">Antibody optimization enabled by artificial intelligence predictions of binding affinity and naturalness</p>
              <p class="authors">Sharrol Bachas, Goran Rakocevic, David Spencer, Anand Sastry, Robel Haile, John Sutton, George Kasun, Andrew Stachyra, Jahir Gutierrez, Edriss Yassine, Borka Medjo, Vincent Blay, Christa Kohnert, Jennifer Stanton, Alexander Brown, Nebojsa Tijanic, Cailen McCloskey, Rebecca Viazzo, Rebecca Consbruck, Hayley Carter, Simon Gottreich-Levine, Shaheed Abdulhaqq, Jacob Shaul, Abigail Ventura, Randal Olson, Engin Yapici, Joshua Meier, Sean McClain, Matthew Weinstock, Gregory Hannum, Ariel Schwartz, Miles Gander, Roberto Spreafico</p>
              <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2022.08.16.504181v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">APPRAISE: ranking engineered proteins by target binding propensity using pair-wise, competitive structure modeling and physics-informed analysis</p>
              <p class="authors">Xiaozhe Ding, Xinhong Chen, Erin Sullivan, Tim Miles, Viviana Gradinaru</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">ChemSpacE: Interpretable and Interactive Chemical Space Exploration</p>
              <p class="authors">Yuanqi Du, Xian Liu, Nilay Shah, Shengchao Liu, Jieyu Zhang, Bolei Zhou</p>
              <p class="preprints"><a href="https://chemrxiv.org/engage/chemrxiv/article-details/633ba619ea6a22b0ba07e841">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Conditional Invariances for Conformer Invariant Protein Representations</p>
              <p class="authors">Balasubramaniam Srinivasan, Vassilis Ioannidis, Soji Adeshina, Mayank Kakodkar, George Karypis, Bruno Ribeiro</p>
              <p class="preprints"><a href="/papers_2022/Conditional_Invariances_for_Conformer_Invariant_Protein_Representations.pdf">[paper]</a> <a href="https://openreview.net/forum?id=xOFD5BMwsB">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Conditional neural processes for molecules</p>
              <p class="authors">Miguel Garcia-Ortegon, Andreas Bender, Sergio Bacallado</p>
              <p class="preprints"><a href="/papers_2022/Conditional_neural_processes_for_molecules.pdf">[paper]</a> <a href="https://arxiv.org/abs/2210.09211">[preprint]</a></p>
            </li>
            <li>
              <p class="title">ContactNet: Geometric-Based Deep Learning Model for Predicting Protein-Protein Interactions</p>
              <p class="authors">Matan Halfon, Dina Schneidman, Tomer Cohen, raanan fattal</p>
              <p class="preprints"><a href="/papers_2022/ContactNet_Geometric_Based_Deep_Learning_Model_for_Predicting_Protein_Protein_Interactions.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Contrasting drugs from decoys</p>
              <p class="authors">Samuel Sledzieski, Rohit Singh, Lenore J Cowen, Bonnie Berger</p>
              <p class="preprints"><a href="/papers_2022/Contrasting_drugs_from_decoys.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.11.03.515086v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Deep Local Analysis estimates effects of mutations on protein-protein interactions</p>
              <p class="authors">Yasser Mohseni Behbahani, Elodie Laine, Alessandra Carbone</p>
              <p class="preprints"><a href="/papers_2022/Deep_Local_Analysis_estimates_effects_of_mutations_on_protein_protein_interactions.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.09.511484v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Designing Biological Sequences via Meta-Reinforcement Learning and Bayesian Optimization</p>
              <p class="authors">Leo Feng, Padideh Nouri, Aneri Muni, Yoshua Bengio, Pierre-Luc Bacon</p>
              <p class="preprints"><a href="/papers_2022/Designing_Biological_Sequences_via_Meta_Reinforcement_Learning_and_Bayesian_Optimization.pdf">[paper]</a> <a href="https://arxiv.org/abs/2209.06259">[preprint]</a></p>
            </li>
            <li>
              <p class="title">DiffDock: Diffusion Steps, Twists, and Turns for Molecular Docking</p>
              <p class="authors">Gabriele Corso, Hannes Stärk, Bowen Jing, Regina Barzilay, Tommi Jaakkola</p>
              <p class="preprints"><a href="/papers_2022/DiffDock_Diffusion_Steps_Twists_and_Turns_for_Molecular_Docking.pdf">[paper]</a> <a href="https://arxiv.org/abs/2210.01776">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Diffusion probabilistic modeling of protein backbones in 3D for the motif-scaffolding problem</p>
              <p class="authors">Jason Yim, Brian L Trippe, Doug Tischer, David Baker, Tamara Broderick, Regina Barzilay, Tommi Jaakkola</p>
              <p class="preprints"><a href="/papers_2022/Diffusion_probabilistic_modeling_of_protein_backbones_in_3D_for_the_motif_scaffolding_problem.pdf">[paper]</a> <a href="https://arxiv.org/abs/2206.04119">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Does Inter-Protein Contact Prediction Benefit from Multi-Modal Data and Auxiliary Tasks?</p>
              <p class="authors">Arghamitra Talukder, Rujie Yin, Yang Shen, Yuning You</p>
              <p class="preprints"><a href="/papers_2022/Does_Inter_Protein_Contact_Prediction_Benefit_from_Multi_Modal_Data_and_Auxiliary_Tasks.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Dynamic-backbone protein-ligand structure prediction with multiscale generative diffusion models</p>
              <p class="authors">Zhuoran Qiao, Weili Nie, Arash Vahdat, Thomas Miller, Anima Anandkumar</p>
              <p class="preprints"><a href="https://arxiv.org/abs/2209.15171">[preprint]</a></p>
            </li>
            <li>
              <p class="title">End-to-end accurate and high-throughput modeling of antibody-antigen complexes</p>
              <p class="authors">Tomer Cohen, Dina Schneidman</p>
              <p class="preprints"><a href="/papers_2022/End_to_end_accurate_and_high_throughput_modeling_of_antibody_antigen_complexes.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">EquiFold: Protein Structure Prediction with a Novel Coarse-Grained Structure Representation</p>
              <p class="authors">Jae Hyeon Lee, Payman Yadollahpour, Andrew Watkins, Nathan Frey, Andrew Leaver-Fay, Stephen Ra, Vladimir Gligorijevic, Kyunghyun Cho, Aviv Regev, Richard Bonneau</p>
              <p class="preprints"><a href="/papers_2022/EquiFold_Protein_Structure_Prediction_with_a_Novel_Coarse_Grained_Structure_Representation.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.07.511322v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">EvoOpt: an MSA-guided, fully unsupervised sequence optimization pipeline for protein design</p>
              <p class="authors">Hideki Yamaguchi, Yutaka Saito</p>
              <p class="preprints"><a href="/papers_2022/EvoOpt_an_MSA_guided_fully_unsupervised_sequence_optimization_pipeline_for_protein_design.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Explainable Deep Generative Models, Ancestral Fragments, and Murky Regions of the Protein Structure Universe</p>
              <p class="authors">Eli Draizen, Cameron Mura, Philip Bourne</p>
              <p class="preprints"><a href="/papers_2022/Explainable_Deep_Generative_Models_Ancestral_Fragments_and_Murky_Regions_of_the_Protein_Structure_Universe.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.11.16.516787v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">ExpressUrself: A spatial model for predicting recombinant expression from mRNA sequence</p>
              <p class="authors">Michael P Dunne, Javier Caceres-Delpiano</p>
              <p class="preprints"><a href="/papers_2022/ExpressUrself_A_spatial_model_for_predicting_recombinant_expression_from_mRNA_sequence.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Fast and Accurate Antibody Structure Prediction without Sequence Homologs</p>
              <p class="authors">Jiaxiang Wu, Fandi Wu, Biaobin Jiang, Wei Liu, Peilin Zhao</p>
              <p class="preprints"><a href="/papers_2022/Fast_and_Accurate_Antibody_Structure_Prediction_without_Sequence_Homologs.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.11.10.515918v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Fast protein structure searching using structure graph embeddings</p>
              <p class="authors">Joe Greener, Kiarash Jamali</p>
              <p class="preprints"><a href="/papers_2022/Fast_protein_structure_searching_using_structure_graph_embeddings.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.11.28.518224v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Heterogeneous reconstruction of deformable atomic models in Cryo-EM</p>
              <p class="authors">Youssef Nashed, Ariana Peck, Julien Martel, Axel Levy, Bongjin Koo, Gordon Wetzstein, Nina Miolane, Daniel Ratner, Frederic Poitevin</p>
              <p class="preprints"><a href="/papers_2022/Heterogeneous_reconstruction_of_deformable_atomic_models_in_Cryo_EM.pdf">[paper]</a> <a href="https://arxiv.org/abs/2209.15121">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Holographic-(V)AE: an end-to-end SO(3)-Equivariant (Variational) Autoencoder in Fourier Space</p>
              <p class="authors">Gian Marco Visani, Michael Pun, Armita Nourmohammad</p>
              <p class="preprints"><a href="/papers_2022/Holographic_V_AE_an_end_to_end_SO_3_Equivariant_Variational_Autoencoder_in_Fourier_Space.pdf">[paper]</a> <a href="https://arxiv.org/abs/2209.15567">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Identifying endogenous peptide receptors by combining structure and transmembrane topology prediction</p>
              <p class="authors">Felix Teufel, Jan Christian Refsgaard, Christian Toft Madsen, Carsten Stahlhut, Mads Grønborg, Dennis Madsen, Ole Winther</p>
              <p class="preprints"><a href="/papers_2022/Identifying_endogenous_peptide_receptors_by_combining_structure_and_transmembrane_topology_prediction.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.28.514036v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Improving Molecular Pretraining with Complementary Featurizations</p>
              <p class="authors">Yanqiao Zhu, Dingshuo Chen, Yuanqi Du, Yingze Wang, Qiang Liu, Shu Wu</p>
              <p class="preprints"><a href="/papers_2022/Improving_Molecule_Properties_Through_2_Stage_VAE.pdf">[paper]</a> <a href="https://arxiv.org/abs/2209.15101">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Improving Molecule Properties Through 2-Stage VAE</p>
              <p class="authors">Chenghui Zhou, Barnabas Poczos</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">Improving Protein Subcellular Localization Prediction with Structural Prediction &amp; Graph Neural Networks</p>
              <p class="authors">Geoffroy Dubourg-Felonneau, Arash Abbasi, Eyal Akiva, Lawrence Lee</p>
              <p class="preprints"><a href="/papers_2022/Improving_Protein_Subcellular_Localization_Prediction_with_Structural_Prediction_Graph_Neural_Networks.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Investigating graph neural network for RNA structural embedding</p>
              <p class="authors">vaitea opuu, Helene Bret</p>
              <p class="preprints"><a href="/papers_2022/Investigating_graph_neural_network_for_RNA_structural_embedding.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Investigating the conformational landscape of AlphaFold2-predicted protein kinase structures</p>
              <p class="authors">Carmen Al Masri, Francesco Trozzi, Marcel Patek, Anna Cichonska, Balaguru Ravikumar, Rayees Rahman</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">Large-scale self-supervised pre-training on protein three-dimensional structures</p>
              <p class="authors">Ilya Senatorov</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">Latent Space Diffusion Models of Cryo-EM Structures</p>
              <p class="authors">Karsten Kreis, Tim Dockhorn, Zihao Li, Ellen Zhong</p>
              <p class="preprints"><a href="https://arxiv.org/abs/2211.14169">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Learning Free Energy Pathways through Reinforcement Learning of Adaptive Steered Molecular Dynamics</p>
              <p class="authors">Nicholas Ho, John Kevin Cava, John Vant, Ankita Shukla, Jacob Miratsky, Pavan Turaga, Ross Maciejewski, Abhishek Singharoy</p>
              <p class="preprints"><a href="/papers_2022/Learning_Free_Energy_Pathways_through_Reinforcement_Learning_of_Adaptive_Steered_Molecular_Dynamics.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.04.510845v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Learning from physics-based features improves protein property prediction</p>
              <p class="authors">Amy Wang, Ava Soleimany, Alex X Lu, Kevin Yang</p>
              <p class="preprints"><a href="/papers_2022/Learning_from_physics_based_features_improves_protein_property_prediction.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Ligand-aware protein sequence design using protein self contacts</p>
              <p class="authors">Jody Mou, Benjamin Fry, Chun-Chen Yao, Nicholas Polizzi</p>
              <p class="preprints"><a href="https://www.dropbox.com/s/98ri2f9gverljcw/Ligand-aware_protein_sequence_design_using_protein_self_contacts.pdf?dl=0">[paper]</a></p>
            </li>
            <li>
              <p class="title">Lightweight Equivariant Graph Representation Learning for Protein Engineering</p>
              <p class="authors">Bingxin Zhou, · Kai Yi, Xinye Xiong, Pan Tan, Liang Hong, Yuguang Wang</p>
              <p class="preprints"><a href="/papers_2022/Lightweight_Equivariant_Graph_Representation_Learning_for_Protein_Engineering.pdf">[paper]</a> <a href="https://openreview.net/forum?id=IWoHx6bY4Zm">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Masked inverse folding with sequence transfer for protein representation learning</p>
              <p class="authors">Kevin Yang, Niccoló Zanichelli, Hugh Yeh</p>
              <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2022.05.25.493516v2">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Membrane and microtubule rapid instance segmentation with dimensionless instance segmentation by learning graph representations of point clouds</p>
              <p class="authors">Robert Kiewisz, Tristan Bepler</p>
              <p class="preprints"><a href="/papers_2022/Membrane_and_microtubule_rapid_instance_segmentation_with_dimensionless_instance_segmentation_by_learning_graph_representations_of_point_clouds.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Metal3D: Accurate prediction of transition metal ion location via deep learning</p>
              <p class="authors">Simon Dürr</p>
              <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2022.08.22.504853v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">MLPfold: Identification of transition state ensembles in molecular dynamics simulations using machine learning</p>
              <p class="authors">Preetham Venkatesh</p>
              <p class="preprints"><a href="/papers_2022/MLPfold_Identification_of_transition_state_ensembles_in_molecular_dynamics_simulations_using_machine_learning.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">ModelAngelo: Automated Model Building in Cryo-EM Maps</p>
              <p class="authors">Kiarash Jamali, Dari Kimanius, Sjors Scheres</p>
              <p class="preprints"><a href="/papers_2022/ModelAngelo_Automated_Model_Building_in_Cryo_EM_Maps.pdf">[paper]</a> <a href="https://openreview.net/forum?id=65XDF_nwI61">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Online Inference of Structure Factor Amplitudes for Serial X-ray Crystallography</p>
              <p class="authors">Kevin Dalton, Doeke Hekstra</p>
              <p class="preprints"><a href="/papers_2022/Online_Inference_of_Structure_Factor_Amplitudes_for_Serial_X_ray_Crystallography.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Peptide-MHC Structure Prediction With Mixed Residue and Atom Graph Neural Network</p>
              <p class="authors">Antoine Delaunay, Yunguan Fu, Alberto Bégué, Robert McHardy, Bachir Djermani, Liviu Copoiu, Michael Rooney, Andrey Tovchigrechko, Marcin Skwark, Nicolas Lopez Carranza, Maren Lang, Karim Beguir, Ugur Sahin</p>
              <p class="preprints"><a href="/papers_2022/Peptide_MHC_Structure_Prediction_With_Mixed_Residue_and_Atom_Graph_Neural_Network.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.11.23.517618v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Physics aware inference for the cryo-EM inverse problem: anisotropic network model heterogeneity, global pose and microscope defocus</p>
              <p class="authors">Geoffrey Woollard, Shayan Shekarforoush, Frank Wood, Marcus Brubaker, Khanh Dao Duc</p>
              <p class="preprints"><a href="/papers_2022/Physics_aware_inference_for_the_cryo_EM_inverse_problem_anisotropic_network_model_heterogeneity_global_pose_and_microscope_defocus.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Physics-aware Graph Neural Network for Accurate RNA 3D Structure Prediction</p>
              <p class="authors">Shuo Zhang, Lei Xie, Yang Liu</p>
              <p class="preprints"><a href="/papers_2022/Physics_aware_Graph_Neural_Network_for_Accurate_RNA_3D_Structure_Prediction.pdf">[paper]</a> <a href="https://arxiv.org/abs/2210.16392">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Plug &amp; Play Directed Evolution of Proteins with Gradient-based Discrete MCMC</p>
              <p class="authors">Patrick Emami, Aidan Perreault, Jeffrey Law, David Biagioni, Peter St. John</p>
              <p class="preprints"><a href="/papers_2022/Plug_Play_Directed_Evolution_of_Proteins_with_Gradient_based_Discrete_MCMC.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Predicting conformational landscapes of known and putative fold-switching proteins using AlphaFold2</p>
              <p class="authors">Hannah Wayment-Steele, Sergey Ovchinnikov, Lucy Colwell, Dorothee Kern</p>
              <p class="preprints"><a href="/papers_2022/Predicting_conformational_landscapes_of_known_and_putative_fold_switching_proteins_using_AlphaFold2.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.17.512570v1.full">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Predicting Immune Escape with Pretrained Protein Language Model Embeddings</p>
              <p class="authors">Kyle Swanson, Howard Chang, James Zou</p>
              <p class="preprints"><a href="/papers_2022/Predicting_Immune_Escape_with_Pretrained_Protein_Language_Model_Embeddings.pdf">[paper]</a> <a href="https://openreview.net/pdf?id=eA9ysFU2QyH">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Predicting interaction partners using masked language modeling</p>
              <p class="authors">Damiano Sgarbossa, Umberto Lupo, Anne-Florence Bitbol</p>
              <p class="preprints"><a href="/papers_2022/Predicting_interaction_partners_using_masked_language_modeling.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Predicting Ligand – RNA Binding Using E3-Equivariant Network and Pretraining</p>
              <p class="authors">Zhenfeng Deng, Ruichu Gu, Hangrui Bi, Xinyan Wang, Zhaolei Zhang, Han Wen</p>
              <p class="preprints"><a href="/papers_2022/Predicting_Ligand_RNA_Binding_Using_E3_Equivariant_Network_and_Pretraining.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Pretrained protein language model transfer learning: is the final layer representation what we want?</p>
              <p class="authors">Francesca-Zhoufan Li, Ava Soleimany, Kevin Yang, Alex X Lu</p>
              <p class="preprints"><a href="/papers_2022/Pretrained_protein_language_model_transfer_learning_is_the_final_layer_representation_what_we_want.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Protein Sequence Design in a Latent Space via Model-based Reinforcement Learning</p>
              <p class="authors">Minji Lee, Luiz Felipe Vecchietti, Hyunkyu Jung, Hyunjoo Ro, Ho Min Kim, Meeyoung Cha</p>
              <p class="preprints"><a href="/papers_2022/Protein_Sequence_Design_in_a_Latent_Space_via_Model_based_Reinforcement_Learning.pdf">[paper]</a> <a href="https://openreview.net/forum?id=OhjGzRE5N6o">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Protein Structure and Sequence Generation with Equivariant Denoising Diffusion Probabilistic Models</p>
              <p class="authors">Namrata Anand, Tudor Achim</p>
              <p class="preprints"><a href="https://arxiv.org/abs/2205.15019">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Protein structure generation via folding diffusion</p>
              <p class="authors">Kevin Wu, Kevin Yang, Rianne van den Berg, James Zou, Alex X Lu, Ava Soleimany</p>
              <p class="preprints"><a href="https://arxiv.org/abs/2209.15611">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Protein-Protein Docking with Iterative Transformer</p>
              <p class="authors">Lee-Shin Chu, Jeffrey Ruffolo, Jeffrey Gray</p>
              <p class="preprints"><a href="/papers_2022/Protein_Protein_Docking_with_Iterative_Transformer.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Reconstruction of polymer structures from contact maps with Deep Learning</p>
              <p class="authors">Atreya Dey</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">Representation Learning on Biomolecular Structures using Equivariant Graph Attention</p>
              <p class="authors">Tuan Le, Frank Noe, Djork-Arné Clevert</p>
              <p class="preprints"><a href="/papers_2022/Representation_Learning_on_Biomolecular_Structures_using_Equivariant_Graph_Attention.pdf">[paper]</a> <a href="https://openreview.net/forum?id=kv4xUo5Pu6">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Representation of missense variants for predicting modes of action</p>
              <p class="authors">Guojie Zhong, Yufeng Shen</p>
              <p class="preprints"><a href="/papers_2022/Representation_of_missense_variants_for_predicting_modes_of_action.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">RL Boltzmann Generators for Conformer Generation in Data-Sparse Environments</p>
              <p class="authors">Yash Patel, Ambuj Tewari</p>
              <p class="preprints"><a href="/papers_2022/RL_Boltzmann_Generators_for_Conformer_Generation_in_Data_Sparse_Environments.pdf">[paper]</a> <a href="https://arxiv.org/abs/2211.10771">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Seq2MSA: A Language Model for Protein Sequence Diversification</p>
              <p class="authors">Pascal Sturmfels, Roshan Rao, Robert Verkuil, Zeming Lin, Tom Sercu, Adam Lerer, Alex Rives</p>
              <p class="preprints"><a href="/papers_2022/Seq2MSA_A_Language_Model_for_Protein_Sequence_Diversification.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">So ManyFolds, So Little Time: Efficient Protein Structure Prediction With pLMs and MSAs</p>
              <p class="authors">Thomas D Barrett, Amelia Villegas-Morcillo, Louis Robinson, Benoit Gaujac, Karim Beguir, Arthur Flajolet</p>
              <p class="preprints"><a href="/papers_2022/So_ManyFolds_So_Little_Time_Efficient_Protein_Structure_Prediction_With_pLMs_and_MSAs.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.15.511553v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Structure-based Drug Design with Equivariant Diffusion Models</p>
              <p class="authors">Arne Schneuing, Yuanqi Du, Charles Harris, Arian Jamasb, Ilia Igashov, weitao Du, Tom Blundell, Pietro Lió, Carla Gomes, Max Welling, Michael Bronstein, Bruno Correia</p>
              <p class="preprints"><a href="/papers_2022/Structure_based_Drug_Design_with_Equivariant_Diffusion_Models.pdf">[paper]</a> <a href="https://arxiv.org/abs/2210.13695">[preprint]</a></p>
            </li>
            <li>
              <p class="title">SWAMPNN: End-to-end protein structures alignment</p>
              <p class="authors">Jeanne Trinquier, Samantha Petti, Shihao Feng, Johannes Soeding, Martin Steinegger, Sergey Ovchinnikov</p>
              <p class="preprints"><a href="/papers_2022/SWAMPNN_End_to_end_protein_structures_alignment.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">T-cell receptor specific protein language model for prediction and interpretation of epitope binding (ProtLM.TCR)</p>
              <p class="authors">Ahmed Essaghir</p>
              <p class="preprints"></p>
            </li>
            <li>
              <p class="title">The geometry of hidden representations of protein language models</p>
              <p class="authors">Lucrezia Valeriani, Francesca Cuturello, Alessio Ansuini, Alberto Cazzaniga</p>
              <p class="preprints"><a href="/papers_2022/The_geometry_of_hidden_representations_of_protein_language_models.pdf">[paper]</a> <a href="https://www.biorxiv.org/content/10.1101/2022.10.24.513504v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Towards automated crystallographic structure refinement with a differentiable pipeline</p>
              <p class="authors">Minhuan Li, Doeke Hekstra</p>
              <p class="preprints"><a href="/papers_2022/Towards_automated_crystallographic_structure_refinement_with_a_differentiable_pipeline.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Training self-supervised peptide sequence models on artificially chopped proteins</p>
              <p class="authors">Gil Sadeh, Zichen Wang, Jasleen Grewal, Huzefa Rangwala, Layne Price</p>
              <p class="preprints"><a href="/papers_2022/Training_self_supervised_peptide_sequence_models_on_artificially_chopped_proteins.pdf">[paper]</a> <a href="https://arxiv.org/abs/2211.06428">[preprint]</a></p>
            </li>
            <li>
              <p class="title">Unsupervised language models for disease variant prediction</p>
              <p class="authors">Allan Zhou, Nicholas C. Landolfi, Daniel ONeill</p>
              <p class="preprints"><a href="/papers_2022/Unsupervised_language_models_for_disease_variant_prediction.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Using domain-domain interactions to probe the limitations of MSA pairing strategies</p>
              <p class="authors">Alex Hawkins-Hooker, David Jones, Brooks Paige</p>
              <p class="preprints"><a href="/papers_2022/Using_domain_domain_interactions_to_probe_the_limitations_of_MSA_pairing_strategies.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">Visualizing DNA reaction trajectories with deep graph embedding approaches</p>
              <p class="authors">Chenwei Zhang, Anne Condon, Khanh Dao Duc</p>
              <p class="preprints"><a href="/papers_2022/Visualizing_DNA_reaction_trajectories_with_deep_graph_embedding_approaches.pdf">[paper]</a></p>
            </li>
            <li>
              <p class="title">What is hidden in the darkness? Characterization of AlphaFold structural space</p>
              <p class="authors">Janani Durairaj, Joana Maria Soa Pereira, Mehmet Akdel, Torsten Schwede</p>
              <p class="preprints"><a href="https://www.biorxiv.org/content/10.1101/2022.10.11.511548v1">[preprint]</a></p>
            </li>
            <li>
              <p class="title">ZymCTRL: a conditional language model for the controllable generation of artificial enzymes</p>
              <p class="authors">Noelia Ferruz</p>
              <p class="preprints"><a href="/papers_2022/ZymCTRL_a_conditional_language_model_for_the_controllable_generation_of_artificial_enzymes.pdf">[paper]</a></p>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div id="organizers" class="content-section-a" style="border-top: 0">
    <div class="container">
      <h2 class="section-heading">Organizers</h2>
      <div class="row wow fadeInRightBig"  data-animation-delay="200">
        <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
          <h4 class="section-heading">
            <center>
              Jonas <br>
              Adler
            </center>
          </h4>
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/jonas_adler.jpg" alt="Photo of Jonas Adler">
          </center>
          <h5 class="section-heading">
            <center>
              <i>DeepMind</i>
            </center>
          </h5>
        </div>
        <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
          <h4 class="section-heading">
            <center>
              Namrata <br>
              Anand
            </center>
          </h4>
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/namrata_anand.png" alt="Photo of Namrata Anand">
          </center>
          <h5 class="section-heading">
            <center>
              <i>Stanford</i>
            </center>
          </h5>
        </div>
        <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
          <h4 class="section-heading">
            <center>
              John <br>
              Ingraham
            </center>
          </h4>
          <center>
            <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/john_ingraham.jpg" alt="Photo of John Ingraham">
          </center>
          <h5 class="section-heading">
            <center>
              <i>Generate Biomedicines</i>
            </center>
          </h5>
        </div>
        <div class="row wow fadeInRightBig"  data-animation-delay="200">
          <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
            <h4 class="section-heading">
              <center>
                Sergey <br>
                Ovchinnikov
              </center>
            </h4>
            <center>
              <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/sergey_ovchinnikov.jpg" alt="Photo of Sergey Ovchinnikov">
            </center>
            <h5 class="section-heading">
              <center>
                <i>Harvard University</i>
              </center>
            </h5>
          </div>
          <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
            <h4 class="section-heading">
              <center>
                Roshan <br>
                Rao
              </center>
            </h4>
            <center>
              <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/roshan_rao.jpg" alt="Photo of Roshan Rao">
            </center>
            <h5 class="section-heading">
              <center>
                <i>Meta AI</i>
              </center>
            </h5>
          </div>
          <div class="col-sm-2 wow fadeInRightBig"  data-animation-delay="200">
            <h4 class="section-heading">
              <center>
                Ellen <br>
                Zhong
              </center>
            </h4>
            <center>
              <img  class="magnify img-responsive img-rounded" style="border-radius: 50%" height="150" width="150" src="img/organizer/ellen_zhong.jpg" alt="Photo of Ellen Zhong">
            </center>
            <h5 class="section-heading">
              <center>
                <i>Princeton</i>
              </center>
            </h5>
          </div>
        </div>
      </div>
    </div>
  </div>
  
  <!--
	<div id="related" class="content-section-b" style="border-top: 0">
        <div class="container" id="related">
		    <h2 class="section-heading">Related Workshops</h2>
            <div class="row">
                <div class="col-md-12 wow fadeInRightBig" data-animation-delay="200">
                    <p class="lead"> Here are some <a href="https://neurips.cc/Conferences/2021/Schedule?type=Workshop">related workshops at NeurIPS</a> covering structure, biology, and healthcare this year:
                    <ul class="bulleted">
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21870">Learning Meaningful Representations for Life</a></li>
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21862">Machine Learning and the Physical Sciences</a> </li>
                        <li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21834">MLPH: Machine Learning in Public Health</a></li>
						<li><a href="https://neurips.cc/Conferences/2021/Schedule?showEvent=21849">AI for Science: Mind the Gaps</a></li>
						<li><a href="https://neurips.cc/Conferences/2020/Schedule?showEvent=21882">Differentiable Programming</a></li>
                    </ul>
				</div>
            </div>
        </div>
        <br/>
		<br/>
		<div class="container">
			<div class="row wow fadeInLeftBig"  data-animation-delay="200">
				<div class="col-md-12 wow fadeInLeftBig"  data-animation-delay="200">
					<div class="center">
						<img  class="center" style="" height="200" src="./img/letters.png" alt="Letters">
					</div>
				</div>
			</div>
		</div>
	</div>
    -->
  
  <div id="sponsors" class="content-section-a" style="border-top: 0">
    <div class="container">
      <h2 class="section-heading">Sponsors</h2>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/ATOMICAI_logo+name_Yellow_Red_black.png" alt="Atomic Logo"> </div>
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/third-rock.jpeg" alt="TRV Logo"> </div>
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/novo-nordisk.jpeg" alt="Novo Nordisk Logo"> </div>
      </div>
      <br>
      <br>
      <div class="row align-items-center justify-content-center wow fadeInLeftBig"  data-animation-delay="200">
        <div class="col-md-4 wow fadeInLeftBig"  data-animation-delay="200"> <img  class="center" style="" height="150" src="./img/prescient.png" alt="Prescient Logo"> </div>
        <div class="col-md-4"  data-animation-delay="200"> <img  class="center" style="" height="100" src="./img/generate-bio-afpc-logo-cmyk-black.png" alt="Generate Bio Logo"> </div>
      </div>
    </div>
  </div>
</div>
<footer>
  <div class="container">
    <div class="row">
      <div class="col-md-12" id="contact">
        <h3 class="footer-title">Contact</h3>
        <p> Please contact <a href = "mailto: workshopmlsb@gmail.com">workshopmlsb@gmail.com</a> with any questions. </p>
      </div>
      <!-- /col-xs-7 --> 
    </div>
  </div>
</footer>

<!-- JavaScript --> 
<script src="js/jquery-1.10.2.js"></script> 
<script src="js/bootstrap.js"></script> 
<script src="js/owl.carousel.js"></script> 
<script src="js/script.js"></script> 
<!-- StikyMenu --> 
<script src="js/stickUp.min.js"></script> 
<script type="text/javascript">
	  jQuery(function($) {
		$(document).ready( function() {
		  $('.navbar-default').stickUp();

		});
	  });

	</script> 
<!-- Smoothscroll --> 
<script type="text/javascript" src="js/jquery.corner.js"></script> 
<script src="js/wow.min.js"></script> 
<script>
	 new WOW().init();
	</script> 
<script src="js/classie.js"></script> 
<!-- Magnific Popup core JS file --> 
<script src="js/jquery.magnific-popup.js"></script>
</body>
</html>
